## Bibtex info with abstract of 239 articles

@inproceedings{andersonCormorantCovariantMolecular2019,
    abstract = {We propose Cormorant, a rotationally covariant neural network architecture for learning the behavior and properties of complex many-body physical systems. We apply these networks to molecular systems with two goals: learning atomic potential energy surfaces for use in Molecular Dynamics simulations, and learning ground state properties of molecules calculated by Density Functional Theory. Some of the key features of our network are that (a) each neuron explicitly corresponds to a subset of atoms; (b) the activation of each neuron is covariant to rotations, ensuring that overall the network is fully rotationally invariant. Furthermore, the non-linearity in our network is based upon tensor products and the Clebsch-Gordan decomposition, allowing the network to operate entirely in Fourier space. Cormorant significantly outperforms competing algorithms in learning molecular Potential Energy Surfaces from conformational geometries in the MD-17 dataset, and is competitive with other methods at learning geometric, energetic, electronic, and thermodynamic properties of molecules on the GDB-9 dataset.},
    annotation = {WOS:000535866906022},
    author = {Anderson, Brandon and Hy, Truong-Son and Kondor, Risi},
    booktitle = {Advances in {{Neural Information Processing Systems}} 32 (Nips 2019)},
    date = {2019},
    editor = {Wallach, H. and Larochelle, H. and Beygelzimer, A. and d'Alche- Buc, F. and Fox, E. and Garnett, R.},
    issn = {1049-5258},
    langid = {english},
    location = {{La Jolla}},
    options = {useprefix=true},
    publisher = {{Neural Information Processing Systems (nips)}},
    shorttitle = {Cormorant},
    title = {Cormorant: {{Covariant Molecular Neural Networks}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {32}
}

@article{andradeStructureDisorderedTiO22020,
    abstract = {Amorphous TiO2 (a-TiO2 ) is widely used in many fields, ranging from photoelectrochemistry to bioengineering, hence detailed knowledge of its atomic structure is of scientific and technological interest. Here we use an ab initio-based deep neural network potential (DP) to simulate large scale atomic models of crystalline and disordered TiO(2 )with molecular dynamics. Our DP reproduces the structural properties of all 11 TiO2 crystalline phases, predicts the densities and structure factors of molten and amorphous TiO2 with only a few percent deviation from experiments, and describes the pressure dependence of the amorphous structure in agreement with recent observations. It can be extended to model additional structures and compositions, and can be thus of great value in the study of TiO2-based nanomaterials.},
    annotation = {WOS:000587775100002},
    author = {Andrade, Marcos F. Calegari and Selloni, Annabella},
    date = {2020-11-05},
    doi = {10.1103/PhysRevMaterials.4.113803},
    issn = {2475-9953},
    journaltitle = {Physical Review Materials},
    langid = {english},
    location = {{College Pk}},
    number = {11},
    pages = {113803},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. Mater.},
    title = {Structure of Disordered {{TiO2}} Phases from Ab Initio Based Deep Neural Network Simulations},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {4}
}

@article{andreaniHydrogenDynamicsSupercritical2020,
    abstract = {In this work, an investigation of supercritical water is presented combining inelastic and deep inelastic neutron scattering experiments and molecular dynamics simulations based on a machine-learned potential of ab initio quality. The local hydrogen dynamics is investigated at 250 bar and in the temperature range of 553-823 K, covering the evolution from subcritical liquid to supercritical gas-like water. The evolution of libration, bending, and stretching motions in the vibrational density of states is studied, analyzing the spectral features by a mode decomposition. Moreover, the hydrogen nuclear momentum distribution is measured, and its anisotropy is probed experimentally. It is shown that hydrogen bonds survive up to the higher temperatures investigated, and we discuss our results in the framework of the coupling between intramolecular modes and intermolecular librations. Results show that the local potential affecting hydrogen becomes less anisotropic within the molecular plane in the supercritical phase, and we attribute this result to the presence of more distorted hydrogen bonds.},
    annotation = {WOS:000589920000077},
    author = {Andreani, Carla and Romanelli, Giovanni and Parmentier, Alexandra and Senesi, Roberto and Kolesnikov, Alexander and Ko, Hsin-Yu and Andrade, Marcos F. Calegari and Car, Roberto},
    date = {2020-11-05},
    doi = {10.1021/acs.jpclett.0c02547},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {21},
    pages = {9461--9467},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    title = {Hydrogen {{Dynamics}} in {{Supercritical Water Probed}} by {{Neutron Scattering}} and {{Computer Simulations}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {11}
}

@article{angActiveLearningAccelerates2021a,
    abstract = {Modeling dynamical effects in chemical reactions typically requires ab initio molecular dynamics (AIMD) simulations due to the breakdown of transition state theory (TST). Reactive AIMD simulations are limited to lower-accuracy electronic structure methods and weak statistics because quantum mechanical energies and forces must be evaluated at femtosecond time resolution over many replicas. We report a data-driven pipeline that allows for the treatment of dynamical effects with the same level of theory and overall cost as that of TST approaches. High-throughput ab initio calculations and autonomous data acquisition are coupled to graph convolutional neural-network interatomic potentials, allowing for inexpensive reactive AIMD simulations at quantum mechanical accuracy. We demonstrate the approach by accurately simulating post-TS dynamical effects in three distinct pericyclic reactions, including a challenging trispericyclic reaction with a complex bifurcating potential energy surface. This approach is broadly applicable to understanding dynamical effects and predicting reaction outcomes in large, previously intractable systems.},
    annotation = {WOS:000628786100017},
    author = {Ang, Shi Jun and Wang, Wujie and Schwalbe-Koda, Daniel and Axelrod, Simon and Gomez-Bombarelli, Rafael},
    date = {2021-03-11},
    doi = {10.1016/j.chempr.2020.12.009},
    issn = {2451-9294},
    journaltitle = {Chem},
    langid = {english},
    location = {{Cambridge}},
    number = {3},
    pages = {738--751},
    publisher = {{Cell Press}},
    shortjournal = {Chem},
    title = {Active Learning Accelerates Ab Initio Molecular Dynamics on Reactive Energy Surfaces},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {7}
}

@article{atlasEmbeddingQuantumStatistical2021a,
    abstract = {Quantum-mechanically driven charge polarization and charge transfer are ubiquitous in biomolecular systems, controlling reaction rates, allosteric interactions, ligand-protein binding, membrane transport, and dynamically driven structural transformations. Molecular dynamics (MD) simulations of these processes require quantum mechanical (QM) information in order to accurately describe their reactive dynamics. However, current techniques-empirical force fields, subsystem approaches, ab initio MD, and machine learning-vary in their ability to achieve a consistent chemical description across multiple atom types, and at scale. Here we present a physics-based, atomistic force field, the ensemble DFT charge-transfer embedded-atom method, in which QM forces are described at a uniform level of theory across all atoms, avoiding the need for explicit solution of the Schrodinger equation or large, precomputed training data sets. Coupling between the electronic and atomistic length scales is effected through an ensemble density functional theory formulation of the embedded-atom method originally developed for elemental materials. Charge transfer is expressed in terms of ensembles of ionic state basis densities of individual atoms, and charge polarization, in terms of atomic excited-state basis densities. This provides a highly compact yet general representation of the force field, encompassing both local and system-wide effects. Charge rearrangement is realized through the evolution of ensemble weights, adjusted at each dynamical time step via chemical potential equalization.},
    annotation = {WOS:000648873600024},
    author = {Atlas, Susan R.},
    date = {2021-05-06},
    doi = {10.1021/acs.jpca.1c00164},
    issn = {1089-5639},
    journaltitle = {Journal of Physical Chemistry A},
    langid = {english},
    location = {{Washington}},
    number = {17},
    pages = {3760--3775},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. A},
    title = {Embedding {{Quantum Statistical Excitations}} in a {{Classical Force Field}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {125}
}

@article{balyakinDeepMachineLearning2020,
    abstract = {The use of machine learning to develop neural network potentials (NNP) representing the interatomic potential energy surface allows us to achieve an optimal balance between accuracy and efficiency in computer simulation of materials. A key point in developing such potentials is the preparation of a training dataset of ab initio trajectories. Here we apply a deep potential molecular dynamics (DeePMD) approach to develop NNP for silica, which is the representative glassformer widely used as a model system for simulating network-forming liquids and glasses. We show that the use of a relatively small training dataset of high-temperature ab initio configurations is enough to fabricate NNP, which describes well both structural and dynamical properties of liquid silica. In particular, we calculate the pair correlation functions, angular distribution function, velocity autocorrelation functions, vibrational density of states, and mean-square displacement and reveal a close agreement with ab initio data. We show that NNP allows us to expand significantly the time-space scales achievable in simulations and thus calculating dynamical and transport properties with more accuracy than that for ab initio methods. We find that developed NNP allows us to describe the structure of the glassy silica with satisfactory accuracy even though no low-temperature configurations were included in the training procedure. The results obtained open up prospects for simulating structural and dynamical properties of liquids and glasses via NNP.},
    annotation = {WOS:000592521200001},
    author = {Balyakin, I. A. and Rempel, S. and Ryltsev, R. E. and Rempel, A. A.},
    date = {2020-11-23},
    doi = {10.1103/PhysRevE.102.052125},
    issn = {2470-0045},
    journaltitle = {Physical Review E},
    langid = {english},
    location = {{College Pk}},
    number = {5},
    pages = {052125},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. E},
    title = {Deep Machine Learning Interatomic Potential for Liquid Silica},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {102}
}

@article{banaeiMachinelearningbasedInteratomicPotential2019,
    abstract = {We report that the single interatomic potential, developed using Gaussian regression of data from density functional theory calculations, has high accuracy and flexibility to describe phonon transport with ab initio accuracy in two different atomistic configurations: perfect crystalline Si and crystalline Si with vacancies. The high accuracies of second- and third-order force constants from the Gaussian approximation potential (GAP) are demonstrated with phonon dispersion, Gruneisen parameter, three-phonon scattering rate, phonon-vacancy scattering rate, and thermal conductivity, all of which are very close to the results from density functional theory calculations. We also show that the widely used empirical potentials (Stillinger-Weber and Tersoff) produce much larger errors compared to the GAP. The computational cost of GAP is higher than the two empirical potentials, but five orders of magnitude lower than density functional theory calculations. Our work shows that GAP can provide a new opportunity for studying phonon transport in partially disordered crystalline phases with the high predictive power of ab initio calculation but at a feasible computational cost.},
    annotation = {WOS:000477921300002},
    author = {Banaei, Hasan and Guo, Ruiqiang and Hashemi, Amirreza and Lee, Sangyeop},
    date = {2019-07-26},
    doi = {10.1103/PhysRevMaterials.3.074603},
    issn = {2475-9953},
    journaltitle = {Physical Review Materials},
    langid = {english},
    location = {{College Pk}},
    number = {7},
    pages = {074603},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. Mater.},
    title = {Machine-Learning-Based Interatomic Potential for Phonon Transport in Perfect Crystalline {{Si}} and Crystalline {{Si}} with Vacancies},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {3}
}

@article{banjadeStructureMotifcentricLearning2021,
    abstract = {Incorporation of physical principles in a machine learning (ML) architecture is a fundamental step toward the continued development of artificial intelligence for inorganic materials. As inspired by the Pauling's rule, we propose that structure motifs in inorganic crystals can serve as a central input to a machine learning framework. We demonstrated that the presence of structure motifs and their connections in a large set of crystalline compounds can be converted into unique vector representations using an unsupervised learning algorithm. To demonstrate the use of structure motif information, a motif-centric learning framework is created by combining motif information with the atom-based graph neural networks to form an atom-motif dual graph network (AMDNet), which is more accurate in predicting the electronic structures of metal oxides such as bandgaps. The work illustrates the route toward fundamental design of graph neural network learning architecture for complex materials by incorporating beyond-atom physical principles.},
    annotation = {WOS:000642456200011},
    author = {Banjade, Huta R. and Hauri, Sandro and Zhang, Shanshan and Ricci, Francesco and Gong, Weiyi and Hautier, Geoffroy and Vucetic, Slobodan and Yan, Qimin},
    date = {2021-04},
    doi = {10.1126/sciadv.abf1754},
    issn = {2375-2548},
    journaltitle = {Science Advances},
    langid = {english},
    location = {{Washington}},
    number = {17},
    pages = {eabf1754},
    publisher = {{Amer Assoc Advancement Science}},
    shortjournal = {Sci. Adv.},
    title = {Structure Motif-Centric Learning Framework for Inorganic Crystalline Systems},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {7}
}

@article{barryVoxelizedAtomicStructure2020a,
    abstract = {This paper introduces voxelized atomic structure (VASt) potentials as a machine learning (ML) framework for developing interatomic potentials. The VASt framework utilizes a voxelized representation of the atomic structure directly as the input to a convolutional neural network (CNN). This allows for high-fidelity representations of highly complex and diverse spatial arrangements of the atomic environments of interest. The CNN implicitly establishes the low-dimensional features needed to correlate each atomic neighborhood to its net atomic force. The selection of the salient features of the atomic structure (i.e., feature engineering) in the VASt framework is implicit, comprehensive, automated, scalable, and highly efficient. The calibrated convolutional layers learn the complex spatial relationships and multibody interactions that govern the physics of atomic systems with remarkable fidelity. We show that VASt potentials predict highly accurate forces on two phases of silicon carbide and the thermal conductivity of silicon over a range of isotropic strain.},
    annotation = {WOS:000589920000025},
    author = {Barry, Matthew C. and Wise, Kristopher E. and Kalidindi, Surya R. and Kumar, Satish},
    date = {2020-11-05},
    doi = {10.1021/acs.jpclett.0c02271},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {21},
    pages = {9093--9099},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    shorttitle = {Voxelized {{Atomic Structure Potentials}}},
    title = {Voxelized {{Atomic Structure Potentials}}: {{Predicting Atomic Forces}} with the {{Accuracy}} of {{Quantum Mechanics Using Convolutional Neural Networks}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {11}
}

@article{Behler2007,
    abstract = {The accurate description of chemical processes often requires the use of computationally demanding methods like density-functional theory (DFT), making long simulations of large systems unfeasible. In this Letter we introduce a new kind of neural-network representation of DFT potential-energy surfaces, which provides the energy and forces as a function of all atomic positions in systems of arbitrary size and is several orders of magnitude faster than DFT. The high accuracy of the method is demonstrated for bulk silicon and compared with empirical potentials and DFT. The method is general and can be applied to all types of periodic and nonperiodic systems. © 2007 The American Physical Society.},
    author = {Behler, Jörg and Parrinello, Michele},
    date = {2007-04},
    doi = {10.1103/physrevlett.98.146401},
    journaltitle = {Physical Review Letters},
    number = {14},
    title = {Generalized Neural-Network Representation of High-Dimensional Potential-Energy Surfaces},
    url = {https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.98.146401},
    volume = {98}
}

@article{bernsteinNovoExplorationSelfguided2019,
    abstract = {Interatomic potential models based on machine learning (ML) are rapidly developing as tools for material simulations. However, because of their flexibility, they require large fitting databases that are normally created with substantial manual selection and tuning of reference configurations. Here, we show that ML potentials can be built in a largely automated fashion, exploring and fitting potential-energy surfaces from the beginning (de novo) within one and the same protocol. The key enabling step is the use of a configuration-averaged kernel metric that allows one to select the few most relevant and diverse structures at each step. The resulting potentials are accurate and robust for the wide range of configurations that occur during structure searching, despite only requiring a relatively small number of single-point DFT calculations on small unit cells. We apply the method to materials with diverse chemical nature and coordination environments, marking an important step toward the more routine application of ML potentials in physics, chemistry, and materials science.},
    annotation = {WOS:000489649400001},
    author = {Bernstein, Noam and Csanyi, Gabor and Deringer, Volker L.},
    date = {2019-10-11},
    doi = {10.1038/s41524-019-0236-6},
    issn = {2057-3960},
    journaltitle = {Npj Computational Materials},
    langid = {english},
    location = {{London}},
    pages = {99},
    publisher = {{Springernature}},
    shortjournal = {npj Comput. Mater.},
    title = {De Novo Exploration and Self-Guided Learning of Potential-Energy Surfaces},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {5}
}

@article{bhowmikPerspectiveInverseDesign2019a,
    abstract = {Understanding and controlling the complex and dynamic processes at battery interfaces holds the key to developing more durable and ultra high performance secondary batteries. Interfacial processes like dendrite and Solid Electrolyte Interphase (SEI) formation span numerous time-and length scales, and despite decades of research, their formation, composition, structure and function still pose a conundrum. Consequently, "inverse design" of high-performance interfaces and interphases like the SEI, remains an elusive dream. Here, we present a perspective and possible blueprint for a future battery research strategy to reach this ambitious goal. Semi-supervised generative deep learning models trained on all sources of available data, i.e., extensive multifidelity datasets from multi-scale computer simulations and databases, operando characterization from largescale research facilities, high-throughput synthesis and laboratory testing, need to work closely together to unlock this dream. We show how understanding and tracking different types of uncertainties in the experimental and simulation methods, as well as the machine learning framework for the generative model, is crucial for controlling and improving the fidelity in the predictive design of battery interfaces and interphases. We argue that simultaneous utilization of data from multiple domains, including data from failed experiments, will play a critical role in accelerating the development of reliable generative models to enable accelerated discovery and inverse design of durable ultra high performance batteries based on novel materials, structures and designs.},
    annotation = {WOS:000484341600041},
    author = {Bhowmik, Arghya and Castelli, Ivano E. and Garcia-Lastra, Juan Maria and Jorgensen, Peter Bjorn and Winther, Ole and Vegge, Tejs},
    date = {2019-09},
    doi = {10.1016/j.ensm.2019.06.011},
    issn = {2405-8297},
    journaltitle = {Energy Storage Materials},
    langid = {english},
    location = {{Amsterdam}},
    pages = {446--456},
    publisher = {{Elsevier}},
    shortjournal = {Energy Storage Mater.},
    title = {A Perspective on Inverse Design of Battery Interphases Using Multi-Scale Modelling, Experiments and Generative Deep Learning},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {21}
}

@article{bogojeskiQuantumChemicalAccuracy2020a,
    abstract = {Kohn-Sham density functional theory (DFT) is a standard tool in most branches of chemistry, but accuracies for many molecules are limited to 2-3 kcal . mol(-1) with presently-available functionals. Ab initio methods, such as coupled-cluster, routinely produce much higher accuracy, but computational costs limit their application to small molecules. In this paper, we leverage machine learning to calculate coupled-cluster energies from DFT densities, reaching quantum chemical accuracy (errors below 1 kcal . mol(-1)) on test data. Moreover, density-based Delta -learning (learning only the correction to a standard DFT calculation, termed Delta -DFT ) significantly reduces the amount of training data required, particularly when molecular symmetries are included. The robustness of Delta -DFT is highlighted by correcting "on the fly" DFT-based molecular dynamics (MD) simulations of resorcinol (C6H4(OH)(2)) to obtain MD trajectories with coupled-cluster accuracy. We conclude, therefore, that Delta -DFT facilitates running gas-phase MD simulations with quantum chemical accuracy, even for strained geometries and conformer changes where standard DFT fails. High-level ab initio quantum chemical methods carry a high computational burden, thus limiting their applicability. Here, the authors employ machine learning to generate coupled-cluster energies and forces at chemical accuracy for geometry optimization and molecular dynamics from DFT densities.},
    annotation = {WOS:000582054700002},
    author = {Bogojeski, Mihail and Vogt-Maranto, Leslie and Tuckerman, Mark E. and Mueller, Klaus-Robert and Burke, Kieron},
    date = {2020-10-16},
    doi = {10.1038/s41467-020-19093-1},
    issn = {2041-1723},
    journaltitle = {Nature Communications},
    langid = {english},
    location = {{Berlin}},
    number = {1},
    pages = {5223},
    publisher = {{Nature Research}},
    shortjournal = {Nat. Commun.},
    title = {Quantum Chemical Accuracy from Density Functional Approximations via Machine Learning},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {11}
}

@article{bonatiNeuralNetworksbasedVariationally2019a,
    abstract = {Sampling complex free-energy surfaces is one of the main challenges of modern atomistic simulation methods. The presence of kinetic bottlenecks in such surfaces often renders a direct approach useless. A popular strategy is to identify a small number of key collective variables and to introduce a bias potential that is able to favor their fluctuations in order to accelerate sampling. Here, we propose to use machine-learning techniques in conjunction with the recent variationally enhanced sampling method [O. Valsson, M. Parrinello, Phys. Rev. Lett. 113, 090601 (2014)] in order to determine such potential. This is achieved by expressing the bias as a neural network. The parameters are determined in a variational learning scheme aimed at minimizing an appropriate functional. This required the development of a more efficient minimization technique. The expressivity of neural networks allows representing rapidly varying free-energy surfaces, removes boundary effects artifacts, and allows several collective variables to be handled.},
    annotation = {WOS:000485140300014},
    author = {Bonati, Luigi and Zhang, Yue-Yu and Parrinello, Michele},
    date = {2019-09-03},
    doi = {10.1073/pnas.1907975116},
    issn = {0027-8424},
    journaltitle = {Proceedings of the National Academy of Sciences of the United States of America},
    langid = {english},
    location = {{Washington}},
    number = {36},
    pages = {17641--17647},
    publisher = {{Natl Acad Sciences}},
    shortjournal = {Proc. Natl. Acad. Sci. U. S. A.},
    title = {Neural Networks-Based Variationally Enhanced Sampling},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {116}
}

@article{bourgeoisTransformingSolidstatePrecipitates2020,
    abstract = {Many phase transformations associated with solid-state precipitation look structurally simple, yet, inexplicably, take place with great difficulty. A classic case of difficult phase transformations is the nucleation of strengthening precipitates in high-strength lightweight aluminium alloys. Here, using a combination of atomic-scale imaging, simulations and classical nucleation theory calculations, we investigate the nucleation of the strengthening phase theta' onto a template structure in the aluminium-copper alloy system. We show that this transformation can be promoted in samples exhibiting at least one nanoscale dimension, with extremely high nucleation rates for the strengthening phase as well as for an unexpected phase. This template-directed solid-state nucleation pathway is enabled by the large influx of surface vacancies that results from heating a nanoscale solid. Template-directed nucleation is replicated in a bulk alloy as well as under electron irradiation, implying that this difficult transformation can be facilitated under the general condition of sustained excess vacancy concentrations.},
    annotation = {WOS:000549162600025},
    author = {Bourgeois, Laure and Zhang, Yong and Zhang, Zezhong and Chen, Yiqiang and Medhekar, Nikhil},
    date = {2020-03-06},
    doi = {10.1038/s41467-020-15087-1},
    issn = {2041-1723},
    journaltitle = {Nature Communications},
    langid = {english},
    location = {{Berlin}},
    number = {1},
    pages = {1248},
    publisher = {{Nature Research}},
    shortjournal = {Nat. Commun.},
    title = {Transforming Solid-State Precipitates via Excess Vacancies},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {11}
}

@article{burkleDeepLearningApproachFirstPrinciples2021a,
    abstract = {Large-scale first-principles transport calculations, while essential for device modeling, remain computationally demanding. To overcome this bottle neck, we combine first-principles transport calculations with machine learning-based nonlinear regression. We calculate the electronic conductance through first-principles based nonequilibrium Green's function techniques for small systems and map the transport properties onto local properties using local descriptors. We show that using the local descriptor as input features for deep learning-based nonlinear regression allows us to build a robust neural network that can predict the conductance of large systems beyond that of the current state-of-the-art first-principles calculation algorithms. Our protocol is applied to alkali metal nanowires, i.e., potassium, which have unique geometrical and electronic properties and hence nontrivial transport properties. We demonstrate that within our approach we can achieve qualitative agreement with experiment at a fraction of the computational effort as compared to the direct calculation of the transport properties using conventional first-principles methods.},
    annotation = {WOS:000652836400018},
    author = {Burkle, Marius and Perera, Umesha and Gimbert, Florian and Nakamura, Hisao and Kawata, Masaaki and Asai, Yoshihiro},
    date = {2021-04-27},
    doi = {10.1103/PhysRevLett.126.177701},
    issn = {0031-9007},
    journaltitle = {Physical Review Letters},
    langid = {english},
    location = {{College Pk}},
    number = {17},
    pages = {177701},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. Lett.},
    title = {Deep-{{Learning Approach}} to {{First}}-{{Principles Transport Simulations}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {126}
}

@article{byggmastarGaussianApproximationPotentials2020,
    abstract = {We develop a set of machine-learning interatomic potentials for elemental V, Nb, Mo, Ta, and W using the Gaussian approximation potential framework. The potentials show good accuracy and transferability for elastic, thermal, liquid, defect, and surface properties. All potentials are augmented with accurate repulsive potentials, making them applicable to radiation damage simulations involving high-energy collisions. We study melting and liquid properties in detail and use the potentials to provide melting curves up to 400 GPa for all five elements.},
    annotation = {WOS:000576705400008},
    author = {Byggmastar, J. and Nordlund, K. and Djurabekova, F.},
    date = {2020-09-28},
    doi = {10.1103/PhysRevMaterials.4.093802},
    issn = {2475-9953},
    journaltitle = {Physical Review Materials},
    langid = {english},
    location = {{College Pk}},
    number = {9},
    pages = {093802},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. Mater.},
    title = {Gaussian Approximation Potentials for Body-Centered-Cubic Transition Metals},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {4}
}

@article{byggmastarMachinelearningInteratomicPotential2019a,
    abstract = {We introduce a machine-learning interatomic potential for tungsten using the Gaussian approximation potential framework. We specifically focus on properties relevant for simulations of radiation-induced collision cascades and the damage they produce, including a realistic repulsive potential for the short-range many-body cascade dynamics and a good description of the liquid phase. Furthermore, the potential accurately reproduces surface properties and the energetics of vacancy and self-interstitial clusters, which have been longstanding deficiencies of existing potentials. The potential enables molecular dynamics simulations of radiation damage in tungsten with unprecedented accuracy.},
    annotation = {WOS:000490747900001},
    author = {Byggmastar, J. and Hamedani, A. and Nordlund, K. and Djurabekova, F.},
    date = {2019-10-17},
    doi = {10.1103/PhysRevB.100.144105},
    issn = {2469-9950},
    journaltitle = {Physical Review B},
    langid = {english},
    location = {{College Pk}},
    number = {14},
    pages = {144105},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. B},
    title = {Machine-Learning Interatomic Potential for Radiation Damage and Defects in Tungsten},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {100}
}

@article{carboneMachineLearningXRayAbsorption2020,
    abstract = {Simulations of excited state properties, such as spectral functions, are often computationally expensive and therefore not suitable for high-throughput modeling. As a proof of principle, we demonstrate that graph-based neural networks can be used to predict the x-ray absorption near-edge structure spectra of molecules to quantitative accuracy. Specifically, the predicted spectra reproduce nearly all prominent peaks, with 90\% of the predicted peak locations within 1 eVof the ground truth. Besides its own utility in spectral analysis and structure inference, our method can be combined with structure search algorithms to enable high-throughput spectrum sampling of the vast material configuration space, which opens up new pathways to material design and discovery.},
    annotation = {WOS:000526038600011},
    author = {Carbone, Matthew R. and Topsakal, Mehmet and Lu, Deyu and Yoo, Shinjae},
    date = {2020-04-16},
    doi = {10.1103/PhysRevLett.124.156401},
    issn = {0031-9007},
    journaltitle = {Physical Review Letters},
    langid = {english},
    location = {{College Pk}},
    number = {15},
    pages = {156401},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. Lett.},
    title = {Machine-{{Learning X}}-{{Ray Absorption Spectra}} to {{Quantitative Accuracy}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {124}
}

@article{chehaibouComputingRPAAdsorption2019a,
    abstract = {Correlated quantum-chemical methods for condensed matter systems, such as the random phase approximation (RPA), hold the promise of reaching a level of accuracy much higher than that of conventional density functional theory approaches. However, the high computational cost of such methods hinders their broad applicability, in particular for finite- temperature molecular dynamics simulations. We propose a method that couples machine learning techniques with thermodynamic perturbation theory to estimate finite-temperature properties using correlated approximations. We apply this approach to compute the enthalpies of adsorption in zeolites and show that reliable estimates can be obtained by training a machine learning model with as few as 10 RPA energies. This approach paves the way to the broader use of computationally expensive quantum-chemical methods to predict the finite-temperature properties of condensed matter systems.},
    annotation = {WOS:000497260300043},
    author = {Chehaibou, Bilal and Badawi, Michael and Bucko, Tomas and Bazhirov, Timur and Rocca, Dario},
    date = {2019-11},
    doi = {10.1021/acs.jctc.9b00782},
    issn = {1549-9618},
    journaltitle = {Journal of Chemical Theory and Computation},
    langid = {english},
    location = {{Washington}},
    number = {11},
    pages = {6333--6342},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem. Theory Comput.},
    title = {Computing {{RPA Adsorption Enthalpies}} by {{Machine Learning Thermodynamic Perturbation Theory}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {15}
}

@article{chenAtomicEnergiesConvolutional2018,
    abstract = {Understanding interactions and structural properties at the atomic level is often a prerequisite to the design of novel materials. Theoretical studies based on quantum-mechanical first-principles calculations can provide this knowledge but at an immense computational cost. In recent years, machine learning has been successful in predicting structural properties at a much lower cost. Here we propose a simplified structure descriptor with no empirical parameters, "k-Bags", together with a scalable and comprehensive machine learning framework that can deepen our understanding of atomic properties of structures. This model can readily predict structure-energy relations that can provide results close to the accuracy of ab initio methods. The model provides chemically meaningful atomic energies enabling theoretical analysis of organic and inorganic molecular structures. Utilization of the local information provided by the atomic energies significantly improves upon the stochastic steps in our evolutionary global structure optimization, resulting in a much faster global minimum search of molecules, clusters, and surfaced supported species.},
    annotation = {WOS:000438654500047},
    author = {Chen, Xin and Jorgensen, Mathias S. and Li, Jun and Hammer, Bjork},
    date = {2018-07},
    doi = {10.1021/acs.jctc.8b00149},
    issn = {1549-9618},
    journaltitle = {Journal of Chemical Theory and Computation},
    langid = {english},
    location = {{Washington}},
    number = {7},
    pages = {3933--3942},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem. Theory Comput.},
    title = {Atomic {{Energies}} from a {{Convolutional Neural Network}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/6},
    urldate = {2021-08-06},
    volume = {14}
}

@article{chenCriticalReviewMachine2020a,
    abstract = {Machine learning (ML) is rapidly revolutionizing many fields and is starting to change landscapes for physics and chemistry. With its ability to solve complex tasks autonomously, ML is being exploited as a radically new way to help find material correlations, understand materials chemistry, and accelerate the discovery of materials. Here, an in-depth review of the application of ML to energy materials, including rechargeable alkali-ion batteries, photovoltaics, catalysts, thermoelectrics, piezoelectrics, and superconductors, is presented. A conceptual framework is first provided for ML in materials science, with a broad overview of different ML techniques as well as best practices. This is followed by a critical discussion of how ML is applied in energy materials. This review is concluded with the perspectives on major challenges and opportunities in this exciting field.},
    annotation = {WOS:000509828200001},
    author = {Chen, Chi and Zuo, Yunxing and Ye, Weike and Li, Xiangguo and Deng, Zhi and Ong, Shyue Ping},
    date = {2020-02},
    doi = {10.1002/aenm.201903242},
    issn = {1614-6832},
    journaltitle = {Advanced Energy Materials},
    langid = {english},
    location = {{Weinheim}},
    number = {8},
    pages = {1903242},
    publisher = {{Wiley-V C H Verlag Gmbh}},
    shortjournal = {Adv. Energy Mater.},
    title = {A {{Critical Review}} of {{Machine Learning}} of {{Energy Materials}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {10}
}

@article{chenEfficientConstructionExcitedState2020,
    abstract = {Recently, we have developed a multilayer energy-based fragment (MLEBF) method to describe excited states of large systems in which photochemically active and inert regions are separately treated with multiconfigurational and single-reference electronic structure method and their mutual polarization effects are naturally described within the many-body expansion framework. This MLEBF method has been demonstrated to provide highly accurate energies and gradients. In this work, we have further derived the MLEBF method with which highly accurate excited-state Hessian matrices of large systems are efficiently constructed. Moreover, in combination with recently proposed embedded atom neural network (EANN) model we have developed a machine learning (ML) accelerated MLEBF method (i.e., ML-MLEBF) in which photochemically inert region is entirely replaced with trained ML models. ML-MLEBF is found to improve computational efficiency of Hessian matrices in particular for large systems. Furthermore, both MLEBF and ML-MLEBF methods are highly parallel and exhibit low-scaling computational cost with multiple CPUs. The present developments could motivate combining various ML techniques with fragment-based electronic structure methods to explore Hessian-matrix-based excited-state properties of large systems.},
    annotation = {WOS:000550758200020},
    author = {Chen, Wen-Kai and Zhang, Yaolong and Jiang, Bin and Fang, Wei-Hai and Cui, Ganglong},
    date = {2020-07-09},
    doi = {10.1021/acs.jpca.0c04117},
    issn = {1089-5639},
    journaltitle = {Journal of Physical Chemistry A},
    langid = {english},
    location = {{Washington}},
    number = {27},
    pages = {5684--5695},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. A},
    title = {Efficient {{Construction}} of {{Excited}}-{{State Hessian Matrices}} with {{Machine Learning Accelerated Multilayer Energy}}-{{Based Fragment Method}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {124}
}

@article{chenExploitingMachineLearning2020a,
    abstract = {The excited-state dynamics of chromophores in complex environments determine a range of vital biological and energy capture processes. Time-resolved, multidimensional optical spectroscopies provide a key tool to investigate these processes. Although theory has the potential to decode these spectra in terms of the electronic and atomistic dynamics, the need for large numbers of excited-state electronic structure calculations severely limits first-principles predictions of multidimensional optical spectra for chromophores in the condensed phase. Here, we leverage the locality of chromophore excitations to develop machine learning models to predict the excited-state energy gap of chromophores in complex environments for efficiently constructing linear and multidimensional optical spectra. By analyzing the performance of these models, which span a hierarchy of physical approximations, across a range of chromophore-environment interaction strengths, we provide strategies for the construction of machine learning models that greatly accelerate the calculation of multidimensional optical spectra from first principles.},
    annotation = {WOS:000574906500014},
    author = {Chen, Michael S. and Zuehlsdorff, Tim J. and Morawietz, Tobias and Isborn, Christine M. and Markland, Thomas E.},
    date = {2020-09-17},
    doi = {10.1021/acs.jpclett.0c02168},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {18},
    pages = {7559--7568},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    title = {Exploiting {{Machine Learning}} to {{Efficiently Predict Multidimensional Optical Spectra}} in {{Complex Environments}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {11}
}

@article{chengCosegregationMgZn2020,
    abstract = {Using aberration-corrected scanning transmission electron microscopy, the interface between Thprecipitates and Al matrix in the aged Al-2.1Zn-1.7Mg (at.\%) alloy has been investigated. Atomic-resolution X-ray dispersive spectroscopy mapping reveals the occurrence of co-segregation of Mg and Zn atoms in the \{002\}(alpha) habit plane of the eta(1)(MgZn2) precipitates. Site-specific Mg atoms segregate outside the concave of an outermost zig-zag Zn-rich layer of the eta(1) habit plane. Zn segregation surround the segregated Mg atomic columns was also observed. Hybrid molecular dynamics (MD) and Monte Carlo (MC) atomistic simulations with the accuracy of density functional theory indicate the site-specific segregation is energetically favourable. (C) 2020 Acta Materialia Inc. Published by Elsevier Ltd. All rights reserved.},
    annotation = {WOS:000539115800011},
    author = {Cheng, Bingqing and Zhao, Xiaojun and Zhang, Yong and Chen, Houwen and Polmear, Ian and Nie, Jian-Feng},
    date = {2020-08},
    doi = {10.1016/j.scriptamat.2020.04.004},
    issn = {1359-6462},
    journaltitle = {Scripta Materialia},
    langid = {english},
    location = {{Oxford}},
    pages = {51--55},
    publisher = {{Pergamon-Elsevier Science Ltd}},
    shortjournal = {Scr. Mater.},
    title = {Co-Segregation of {{Mg}} and {{Zn}} Atoms at the Planar Eta(1)-Precipitate/{{Al}} Matrix Interface in an Aged {{Al}}-{{Zn}}-{{Mg}} Alloy},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {185}
}

@article{chengRegressionClusteringImproved2019a,
    abstract = {Machine learning (ML) in the representation of molecular-orbital-based (MOB) features has been shown to be an accurate and transferable approach to the prediction of post-Hartree-Fock correlation energies. Previous applications of MOB-ML employed Gaussian Process Regression (GPR), which provides good prediction accuracy with small training sets; however, the cost of GPR training scales cubically with the amount of data and becomes a computational bottleneck for large training sets. In the current work, we address this problem by introducing a clustering/regression/classification implementation of MOB-ML. In the first step, regression clustering (RC) is used to partition the training data to best fit an ensemble of linear regression (LR) models; in the second step, each cluster is regressed independently, using either LR or GPR; and in the third step, a random forest classifier (RFC) is trained for the prediction of cluster assignments based on MOB feature values. Upon inspection, RC is found to recapitulate chemically intuitive groupings of the frontier molecular orbitals, and the combined RC/LR/RFC and RC/GPR/RFC implementations of MOB-ML are found to provide good prediction accuracy with greatly reduced wall-clock training times. For a data set of thermalized (350 K) geometries of 7211 organic molecules of up to seven heavy atoms (QM7b-T), both RC/LR/RFC and RC/GPR/RFC reach chemical accuracy (1 kcal/mol prediction error) with only 300 training molecules, while providing 35000-fold and 4500-fold reductions in the wall-clock training time, respectively, compared to MOB-ML without clustering. The resulting models are also demonstrated to retain transferability for the prediction of large-molecule energies with only small-molecule training data. Finally, it is shown that capping the number of training data points per cluster leads to further improvements in prediction accuracy with negligible increases in wall-clock training time.},
    annotation = {WOS:000502688500012},
    author = {Cheng, Lixue and Kovachki, Nikola B. and Welborn, Matthew and Miller, Thomas F.},
    date = {2019-12},
    doi = {10.1021/acs.jctc.9b00884},
    issn = {1549-9618},
    journaltitle = {Journal of Chemical Theory and Computation},
    langid = {english},
    location = {{Washington}},
    number = {12},
    pages = {6668--6677},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem. Theory Comput.},
    title = {Regression {{Clustering}} for {{Improved Accuracy}} and {{Training Costs}} with {{Molecular}}-{{Orbital}}-{{Based Machine Learning}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {15}
}

@article{chengUniversalDensityMatrix2019a,
    abstract = {We address the degree to which machine learning (ML) can be used to accurately and transferably predict post-Hartree-Fock correlation energies. Refined strategies for feature design and selection are presented, and the molecular-orbital-based machine learning (MOB-ML) method is applied to several test systems. Strikingly, for the second-order Moller-Plessett perturbation theory, coupled cluster with singles and doubles (CCSD), and CCSD with perturbative triples levels of theory, it is shown that the thermally accessible (350 K) potential energy surface for a single water molecule can be described to within 1 mhartree using a model that is trained from only a single reference calculation at a randomized geometry. To explore the breadth of chemical diversity that can be described, MOB-ML is also applied to a new dataset of thermalized (350 K) geometries of 7211 organic models with up to seven heavy atoms. In comparison with the previously reported Delta-ML method, MOB-ML is shown to reach chemical accuracy with threefold fewer training geometries. Finally, a transferability test in which models trained for seven-heavy-atom systems are used to predict energies for thirteen-heavy-atom systems reveals that MOB-ML reaches chemical accuracy with 36-fold fewer training calculations than Delta-ML (140 vs 5000 training calculations). Published under license by AIP Publishing.},
    annotation = {WOS:000463658900003},
    author = {Cheng, Lixue and Welborn, Matthew and Christensen, Anders S. and Miller, Thomas F.},
    date = {2019-04-07},
    doi = {10.1063/1.5088393},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {13},
    pages = {131103},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    shorttitle = {A Universal Density Matrix Functional from Molecular Orbital-Based Machine Learning},
    title = {A Universal Density Matrix Functional from Molecular Orbital-Based Machine Learning: {{Transferability}} across Organic Molecules},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {150}
}

@article{chenIntegratingMachineLearning2019a,
    abstract = {In this work we have combined machine learning techniques with our recently developed multilayer energy-based fragment method for studying excited states of large systems. The photochemically active and inert regions are separately treated with the complete active space self-consistent field method and the trained models. This method is demonstrated to provide accurate energies and gradients leading to essentially the same excited-state potential energy surfaces and nonadiabatic dynamics compared with full ab initio results. Furthermore, in conjunction with the use of machine learning models, this method is highly parallel and exhibits low-scaling computational cost. Finally, the present work could encourage the marriage of machine learning with fragment-based electronic structure methods to explore photochemistry of large systems.},
    annotation = {WOS:000503919300043},
    author = {Chen, Wen-Kai and Fang, Wei-Hai and Cui, Ganglong},
    date = {2019-12-19},
    doi = {10.1021/acs.jpclett.9b03113},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {24},
    pages = {7836--7841},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    title = {Integrating {{Machine Learning}} with the {{Multilayer Energy}}-{{Based Fragment Method}} for {{Excited States}} of {{Large Systems}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {10}
}

@article{chenTensorAlloyAutomaticAtomistic2020a,
    abstract = {Atomistic modeling is important for studying physical and chemical properties of materials. Recently, machine learning interaction potentials have gained much more attentions as they can provide density functional theory level predictions within negligible time. The symmetry function descriptor based atomistic neural network is the most widely used model for modeling alloys. To precisely describe complex potential energy surfaces, integrating advanced metrics, such as force or virial stress, into training can be of great help. In this work, we propose a virtual-atom approach to model the total energy of symmetry function descriptors based atomistic neural network. Our approach creates the computation graph directly from atomic positions. Thus, the derivations of forces and virial can be handled by TensorFlow automatically and efficiently. The virtual atom approach with AutoGrad within TensorFlow allows for efficient training to not just energies and forces, but also virial stress. This new approach is implemented in our open-source program TensorAlloy, which supports constructing machine learning interaction potentials for both molecules and solids. The QM7 and SNAP/Ni-Mo datasets are used to demonstrate the performances of our program. (C) 2019 Elsevier B.V. All rights reserved.},
    annotation = {WOS:000528001100011},
    author = {Chen, Xin and Gao, Xing-Yu and Zhao, Ya-Fan and Lin, De-Ye and Chu, Wei-Dong and Song, Hai-Feng},
    date = {2020-05},
    doi = {10.1016/j.cpc.2019.107057},
    issn = {0010-4655},
    journaltitle = {Computer Physics Communications},
    langid = {english},
    location = {{Amsterdam}},
    pages = {107057},
    publisher = {{Elsevier}},
    shortjournal = {Comput. Phys. Commun.},
    shorttitle = {{{TensorAlloy}}},
    title = {{{TensorAlloy}}: {{An}} Automatic Atomistic Neural Network Program for Alloys},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {250}
}

@article{chirikiConstructingConvexEnergy2019a,
    abstract = {We propose a global optimization strategy for atomistic structure determination based on two new concepts: a few-atom complementary energy landscape and atomic role models. Global optimization of costly energy expressions may be aided by performing some of the optimization on model energy landscapes. These are often based on a sum-of-atomic-contributions form that accurately reproduces every local energy minimum of the true energy expression. However, we propose that, by not including all atomic contributions, the resulting energy landscapes may become more convex, making the search for the global optimum more facile. A role model is someone we aspire to be more like; in the same vein we define the role model of an atom to be another atom whose local environment the first atom seeks to obtain itself. Basing a complementary energy landscape on the distance of some atoms from their role models in a feature space, we arrive at a useful few-atom complementary energy landscape. We show that relaxation in this landscape is an effective mutation when employed in an evolutionary algorithm used to identify the bulk cristobalite structure of SiO2 and the (1 x 4) surface reconstruction of anatase TiO2 (001).},
    annotation = {WOS:000503387500005},
    author = {Chiriki, Siva and Christiansen, Mads-Peter and Hammer, B.},
    date = {2019-12-19},
    doi = {10.1103/PhysRevB.100.235436},
    issn = {2469-9950},
    journaltitle = {Physical Review B},
    langid = {english},
    location = {{College Pk}},
    number = {23},
    pages = {235436},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. B},
    title = {Constructing Convex Energy Landscapes for Atomistic Structure Optimization},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {100}
}

@article{chmielaExactMolecularDynamics2018,
    abstract = {Molecular dynamics (MD) simulations employing classical force fields constitute the cornerstone of contemporary atomistic modeling in chemistry, biology, and materials science. However, the predictive power of these simulations is only as good as the underlying interatomic potential. Classical potentials often fail to faithfully capture key quantum effects in molecules and materials. Here we enable the direct construction of flexible molecular force fields from high-level ab initio calculations by incorporating spatial and temporal physical symmetries into a gradient-domain machine learning (sGDML) model in an automatic data-driven way. The developed sGDML approach faithfully reproduces global force fields at quantum-chemical CCSD(T) level of accuracy and allows converged molecular dynamics simulations with fully quantized electrons and nuclei. We present MD simulations, for flexible molecules with up to a few dozen atoms and provide insights into the dynamical behavior of these molecules. Our approach provides the key missing ingredient for achieving spectroscopic accuracy in molecular simulations.},
    annotation = {WOS:000445329000022},
    author = {Chmiela, Stefan and Sauceda, Huziel E. and Mueller, Klaus-Robert and Tkatchenko, Alexandre},
    date = {2018-09-24},
    doi = {10.1038/s41467-018-06169-2},
    issn = {2041-1723},
    journaltitle = {Nature Communications},
    langid = {english},
    location = {{London}},
    pages = {3887},
    publisher = {{Nature Publishing Group}},
    shortjournal = {Nat. Commun.},
    title = {Towards Exact Molecular Dynamics Simulations with Machine-Learned Force Fields},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/6},
    urldate = {2021-08-06},
    volume = {9}
}

@article{chmielaSGDMLConstructingAccurate2019a,
    abstract = {We present an optimized implementation of the recently proposed symmetric gradient domain machine learning (sGDML) model. The sGDML model is able to faithfully reproduce global potential energy surfaces (PES) for molecules with a few dozen atoms from a limited number of user-provided reference molecular conformations and the associated atomic forces. Here, we introduce a Python software package to reconstruct and evaluate custom sGDML force fields (FFs), without requiring in-depth knowledge about the details of the model. A user-friendly command-line interface offers assistance through the complete process of model creation, in an effort to make this novel machine learning approach accessible to broad practitioners. Our paper serves as a documentation, but also includes a practical application example of how to reconstruct and use a PBEO+MBD FF for paracetamol. Finally, we show how to interface 5GDML with the FF simulation engines ASE (Larsen et al., 2017) and i-PI (Kapil et al., 2019) to run numerical experiments, including structure optimization, classical and path integral molecular dynamics and nudged elastic band calculations. (C) 2019 The Author(s). Published by Elsevier B.V.},
    annotation = {WOS:000474312900005},
    author = {Chmiela, Stefan and Sauceda, Huziel E. and Poltavsky, Igor and Mueller, Klaus-Robert and Tkatchenko, Alexandre},
    date = {2019-07},
    doi = {10.1016/j.cpc.2019.02.007},
    issn = {0010-4655},
    journaltitle = {Computer Physics Communications},
    langid = {english},
    location = {{Amsterdam}},
    pages = {38--45},
    publisher = {{Elsevier Science Bv}},
    shortjournal = {Comput. Phys. Commun.},
    shorttitle = {{{sGDML}}},
    title = {{{sGDML}}: {{Constructing}} Accurate and Data Efficient Molecular Force Fields Using Machine Learning},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {240}
}

@article{choiEfficientTrainingMachine2020a,
    abstract = {Machine learning potentials provide an efficient and comprehensive tool to simulate large-scale systems inaccessible by conventional first-principles methods still in a similar level of accuracy. One critical issue in constructing machine learning potentials is to build training data sets cost-effectively that can represent the potential energy surface in a wide range of configurations. We develop a scheme named randomized atomic-system generator (RAG) to produce the training sets that widely cover the potential energy surface by combining the random sampling and structural optimization. We apply the scheme to construct the machine learning potentials for simulation of chalcogen-based phase change materials. Constructed machine learning potentials successfully simulate the dynamics of melting and crystallization processes of binary GeTe at a level comparable to first-principles simulations. The visual analysis shows that the RAG-generated training set represents the crystallization process including the amorphous phases. From the velocity autocorrelation function obtained from the molecular dynamics simulations, we calculate the phonon density of states to analyze the vibrational properties during crystallization.},
    annotation = {WOS:000577150000022},
    author = {Choi, Young-Jae and Jhi, Seung-Hoon},
    date = {2020-10-01},
    doi = {10.1021/acs.jpcb.0c05075},
    issn = {1520-6106},
    journaltitle = {Journal of Physical Chemistry B},
    langid = {english},
    location = {{Washington}},
    number = {39},
    pages = {8704--8710},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. B},
    title = {Efficient {{Training}} of {{Machine Learning Potentials}} by a {{Randomized Atomic}}-{{System Generator}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {124}
}

@article{christensenFCHLRevisitedFaster2020a,
    abstract = {We introduce the FCHL19 representation for atomic environments in molecules or condensed-phase systems. Machine learning models based on FCHL19 are able to yield predictions of atomic forces and energies of query compounds with chemical accuracy on the scale of milliseconds. FCHL19 is a revision of our previous work [F. A. Faber et al., J. Chem. Phys. 148, 241717 (2018)] where the representation is discretized and the individual features are rigorously optimized using Monte Carlo optimization. Combined with a Gaussian kernel function that incorporates elemental screening, chemical accuracy is reached for energy learning on the QM7b and QM9 datasets after training for minutes and hours, respectively. The model also shows good performance for non-bonded interactions in the condensed phase for a set of water clusters with a mean absolute error (MAE) binding energy error of less than 0.1 kcal/mol/molecule after training on 3200 samples. For force learning on the MD 17 dataset, our optimized model similarly displays state-of-the-art accuracy with a regressor based on Gaussian process regression. When the revised FCHL19 representation is combined with the operator quantum machine learning regressor, forces and energies can be predicted in only a few milliseconds per atom. The model presented herein is fast and lightweight enough for use in general chemistry problems as well as molecular dynamics simulations. (C) 2020 Author(s).Y},
    annotation = {WOS:000519967800007},
    author = {Christensen, Anders S. and Bratholm, Lars A. and Faber, Felix A. and von Lilienfeld, O. Anatole},
    date = {2020-01-31},
    doi = {10.1063/1.5126701},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {4},
    options = {useprefix=true},
    pages = {044107},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    shorttitle = {{{FCHL}} Revisited},
    title = {{{FCHL}} Revisited: {{Faster}} and More Accurate Quantum Machine Learning},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {152}
}

@article{coleyAutonomousDiscoveryChemical2020a,
    abstract = {This two-part Review examines how automation has contributed to different aspects of discovery in the chemical sciences. In this first part, we describe a classification for discoveries of physical matter (molecules, materials, devices), processes, and models and how they are unified as search problems. We then introduce a set of questions and considerations relevant to assessing the extent of autonomy. Finally, we describe many case studies of discoveries accelerated by or resulting from computer assistance and automation from the domains of synthetic chemistry, drug discovery, inorganic chemistry, and materials science. These illustrate how rapid advancements in hardware automation and machine learning continue to transform the nature of experimentation and modeling. Part two reflects on these case studies and identifies a set of open challenges for the field.},
    annotation = {WOS:000538529000001},
    author = {Coley, Connor W. and Eyke, Natalie S. and Jensen, Klavs F.},
    date = {2020-12-14},
    doi = {10.1002/anie.201909987},
    issn = {1433-7851},
    journaltitle = {Angewandte Chemie-International Edition},
    langid = {english},
    location = {{Weinheim}},
    number = {51},
    pages = {22858--22893},
    publisher = {{Wiley-V C H Verlag Gmbh}},
    shortjournal = {Angew. Chem.-Int. Edit.},
    shorttitle = {Autonomous {{Discovery}} in the {{Chemical Sciences Part I}}},
    title = {Autonomous {{Discovery}} in the {{Chemical Sciences Part I}}: {{Progress}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {59}
}

@article{cruzeiroHighlyAccurateManyBody2021,
    abstract = {Dinitrogen pentoxide (N2O5) is an important intermediate in the atmospheric chemistry of nitrogen oxides. Although there has been much research, the processes that govern the physical interactions between N2O5 and water are still not fully understood at a molecular level. Gaining a quantitative insight from computer simulations requires going beyond the accuracy of classical force fields while accessing length scales and time scales that are out of reach for high-level quantum-chemical approaches. To this end, we present the development of MB-nrg many-body potential energy functions for nonreactive simulations of N2O5 in water. This MB-nrg model is based on electronic structure calculations at the coupled cluster level of theory and is compatible with the successful MB-pol model for water. It provides a physically correct description of long-range many-body interactions in combination with an explicit representation of up to threebody short-range interactions in terms of multidimensional permutationally invariant polynomials. In order to further investigate the importance of the underlying interactions in the model, a TTM-nrg model was also devised. TTM-nrg is a more simplistic representation that contains only two-body short-range interactions represented through Born-Mayer functions. In this work, an active learning approach was employed to efficiently build representative training sets of monomer, dimer, and trimer structures, and benchmarks are presented to determine the accuracy of our new models in comparison to a range of density functional theory methods. By assessing the binding curves, distortion energies of N2O5, and interaction energies in clusters of N2O5 and water, we evaluate the importance of two-body and three-body short-range potentials. The results demonstrate that our MB-nrg model has high accuracy with respect to the coupled cluster reference, outperforms current density functional theory models, and thus enables highly accurate simulations of N2O5 in aqueous environments.},
    annotation = {WOS:000674289800011},
    author = {Cruzeiro, Vinicius Wilian D. and Lambros, Eleftherios and Riera, Marc and Roy, Ronak and Paesani, Francesco and Gotz, Andreas W.},
    date = {2021-07-13},
    doi = {10.1021/acs.jctc.1c00069},
    issn = {1549-9618},
    journaltitle = {Journal of Chemical Theory and Computation},
    langid = {english},
    location = {{Washington}},
    number = {7},
    pages = {3931--3945},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem. Theory Comput.},
    shorttitle = {Highly {{Accurate Many}}-{{Body Potentials}} for {{Simulations}} of {{N2O5}} in {{Water}}},
    title = {Highly {{Accurate Many}}-{{Body Potentials}} for {{Simulations}} of {{N2O5}} in {{Water}}: {{Benchmarks}}, {{Development}}, and {{Validation}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {17}
}

@article{cuevas-zuviriaAnalyticalModelElectron2020a,
    abstract = {We present an analytical model representation of the electron density rho(r) in molecules in the form of expansions of a few functions (exponentials and a Gaussian) per atom. Based on a former analytical model of rho(r) in atoms, we devised its molecular implementation by introducing the anisotropy inherent in the electron distribution of atoms in molecules by means of proper anisotropic functions. The resulting model named A2MD (anisotropic analytical model of density) takes an analytical form highly suitable for obtaining the electron density in large biomolecules as its computational cost scales linearly with the number of atoms. To obtain the parameters of the model, we first devised a fitting procedure to reference electron densities obtained in ab initio correlated quantum calculations. Second, in order to skip costly ab initio calculations, we also developed a machine learning (ML)-based predictor that used neural networks trained on broad molecular datasets to determine the parameters of the model. The resulting ML methodology that we named A2MDnet (A2MD network-trained) was able to provide reliable electron densities as a basis to predict molecular features without requiring quantum calculations. The results presented together with the low computational scaling associated to the A2MD representation of rho(r) suggest potential applications to obtain reliable electron densities and rho(r)-based molecular properties in biomacromolecules.},
    annotation = {WOS:000563791600015},
    author = {Cuevas-Zuviria, Bruno and Pacios, Luis F.},
    date = {2020-08-24},
    doi = {10.1021/acs.jcim.0c00197},
    issn = {1549-9596},
    journaltitle = {Journal of Chemical Information and Modeling},
    langid = {english},
    location = {{Washington}},
    number = {8},
    pages = {3831--3842},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem Inf. Model.},
    title = {Analytical {{Model}} of {{Electron Density}} and {{Its Machine Learning Inference}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {60}
}

@article{cuiBiomolecularQMMM2021,
    abstract = {QM/MM simulations have become an indispensable tool in many chemical and biochemical investigations. Considering the tremendous degree of success, including recognition by a 2013 Nobel Prize in Chemistry, are there still "burning challenges" in QM/MM methods, especially for biomolecular systems? In this short Perspective, we discuss several issues that we believe greatly impact the robustness and quantitative applicability of QM/MM simulations to many, if not all, biomolecules. We highlight these issues with observations and relevant advances from recent studies in our group and others in the field. Despite such limited scope, we hope the discussions are of general interest and will stimulate additional developments that help push the field forward in meaningful directions.},
    annotation = {WOS:000614308000001},
    author = {Cui, Qiang and Pal, Tanmoy and Xie, Luke},
    date = {2021-01-28},
    doi = {10.1021/acs.jpcb.0c09898},
    issn = {1520-6106},
    journaltitle = {Journal of Physical Chemistry B},
    langid = {english},
    location = {{Washington}},
    number = {3},
    pages = {689--702},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. B},
    shorttitle = {Biomolecular {{QM}}/{{MM Simulations}}},
    title = {Biomolecular {{QM}}/{{MM Simulations}}: {{What Are Some}} of the "{{Burning Issues}}"?},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {125}
}

@article{daiGrainBoundaryStrengthening2020,
    abstract = {Interaction between grain boundaries and impurities usually leads to significant altering of material properties. Understanding the composition-structure-property relationship of grain boundaries is a key avenue for tailoring and designing high performance materials. In this work, we studied segregation of W into ZrB2 grain boundaries by a hybrid method combining Monte Carlo (MC) and molecular dynamics (MD), and examined the effects of segregation on grain boundary strengths by MD tensile testing with a fitted machine learning potential. It is found that W prefers grain boundary sites with local compression strains due to its smaller size compared to Zr. Rich segregation patterns (including monolayer, off-center bilayer, and other complex patterns); segregation induced grain boundary structure reconstruction; and order-disorder like segregation pattern transformation are discovered. Strong segregation tendency of W into ZrB2 grain boundaries and significant improvements on grain boundary strengths are certified, which guarantees outstanding high temperature performance of ZrB2-based UHTCs.},
    annotation = {WOS:000564257400003},
    author = {Dai, Fu-Zhi and Wen, Bo and Xiang, Huimin and Zhou, Yanchun},
    date = {2020-12},
    doi = {10.1016/j.jeurceramsoc.2020.06.007},
    issn = {0955-2219},
    journaltitle = {Journal of the European Ceramic Society},
    langid = {english},
    location = {{Oxford}},
    number = {15},
    pages = {5029--5036},
    publisher = {{Elsevier Sci Ltd}},
    shortjournal = {J. Eur. Ceram. Soc.},
    shorttitle = {Grain Boundary Strengthening in {{ZrB2}} by Segregation of {{W}}},
    title = {Grain Boundary Strengthening in {{ZrB2}} by Segregation of {{W}}: {{Atomistic}} Simulations with Deep Learning Potential},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {40}
}

@article{daiTemperatureDependentThermal2021,
    abstract = {High entropy diborides are new categories of ultra-high temperature ceramics, which are believed promising candidates for applications in hypersonic vehicles. However, knowledge on high temperature thermal and mechanical properties of high entropy diborides is still lacking unit now. In this work, variations of thermal and elastic properties of high entropy (Ti0.2Zr0.2Hf0.2Nb0.2Ta0.2)B-2 with respect to temperature were predicted by molecular dynamics simulations. Firstly, a deep learning potential for Ti-Zr-Hf-Nb-Ta-B diboride system was fitted with its prediction error in energy and force respectively being 9.2 meV/atom and 208 meV/A, in comparison with first-principles calculations. Then, temperature dependent lattice constants, anisotropic thermal expansions, anisotropic phonon thermal conductivities, and elastic properties of high entropy (Ti0.2Zr0.2Hf0.2Nb0.2Ta0.2)B-2 from 0 degrees C to 2400 degrees C were evaluated, where the predicted room temperature values agree well with experimental measurements. In addition, intrinsic lattice distortions of (Ti0.2Zr0.2Hf0.2Nb0.2Ta0.2)B-2 were analyzed by displacements of atoms from their ideal positions, which are in an order of 10(-3) A and one order of magnitude smaller than those in (Ti0.2Zr0.2Hf0.2Nb0.2Ta0.2)C. It indicates that lattice distortions in (Ti0.2Zr0.2Hf0.2Nb0.2Ta0.2)B-2 is not so severe as expected. With the new paradigm of machine learning potential, deep insight into high entropy materials can be achieved in the future, since the chemical and structural complexly in high entropy materials can be well handled by machine learning potential. (C) 2021 Published by Elsevier Ltd on behalf of The editorial office of Journal of Materials Science \& Technology.},
    annotation = {WOS:000636043600002},
    author = {Dai, Fu-Zhi and Sun, Yinjie and Wen, Bo and Xiang, Huimin and Zhou, Yanchun},
    date = {2021-05-10},
    doi = {10.1016/j.jmst.2020.07.014},
    issn = {1005-0302},
    journaltitle = {Journal of Materials Science \& Technology},
    langid = {english},
    location = {{Shenyang}},
    pages = {8--15},
    publisher = {{Journal Mater Sci Technol}},
    shortjournal = {J. Mater. Sci. Technol.},
    shorttitle = {Temperature {{Dependent Thermal}} and {{Elastic Properties}} of {{High Entropy}} ({{Ti0}}.{{2Zr0}}.{{2Hf0}}.{{2Nb0}}.{{2Ta0}}.2){{B}}-2},
    title = {Temperature {{Dependent Thermal}} and {{Elastic Properties}} of {{High Entropy}} ({{Ti0}}.{{2Zr0}}.{{2Hf0}}.{{2Nb0}}.{{2Ta0}}.2){{B}}-2: {{Molecular Dynamics Simulation}} by {{Deep Learning Potential}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {72}
}

@article{daiTheoreticalPredictionThermal2020,
    abstract = {High entropy materials (HEMs, e.g. high entropy alloys, high entropy ceramics) have gained increasing interests due to the possibility that they can provide challenge properties unattainable by traditional materials. Though a large number of HEMs have emerged, there is still in lack of theoretical predictions and simulations on HEMs, which is probably caused by the chemical complexity of HEMs. In this work, we demonstrate that the machine learning potentials developed in recent years can overcome the complexity of HEMs, and serve as powerful theoretical tools to simulate HEMs. A deep learning potential (DLP) for high entropy (Zr0.2Hf0.2Ti0.2Nb0.2Ta0.2)C is fitted with the prediction error in energy and force being 9.4 meV/atom and 217 meV/A, respectively. The reliability and generality of the DLP are affirmed, since it can accurately predict lattice parameters and elastic constants of mono-phase carbides TMC (TM = Ti, Zr, Hf, Nb and Ta). Lattice constants (increase from 4.5707 A to 4.6727 A), thermal expansion coefficients (increase from 7.85x10(-6) K-1 to 10.58 x 10(-6) K-1), phonon thermal conductivities (decrease from 2.02 W m(-1) K-1 to 0.95 W m(-1) K-1), and elastic properties of high entropy (Zr0.2Hf0.2Ti0.2Nb0.2Ta0.2)C in temperature ranging from 0 degrees C to 2400 degrees C are predicted by molecular dynamics simulations. The predicted room temperature properties agree well with experimental measurements, indicating the high accuracy of the DLP. With introducing of machine learning potentials, many problems that are intractable by traditional methods can be handled now. It is hopeful that deep insight into HEMs can be obtained in the future by such powerful methods. (C) 2020 Published by Elsevier Ltd on behalf of The editorial office of Journal of Materials Science \& Technology.},
    annotation = {WOS:000522855000019},
    author = {Dai, Fu-Zhi and Wen, Bo and Sun, Yinjie and Xiang, Huimin and Zhou, Yanchun},
    date = {2020-04-15},
    doi = {10.1016/j.jmst.2020.01.005},
    issn = {1005-0302},
    journaltitle = {Journal of Materials Science \& Technology},
    langid = {english},
    location = {{Shenyang}},
    pages = {168--174},
    publisher = {{Journal Mater Sci Technol}},
    shortjournal = {J. Mater. Sci. Technol.},
    title = {Theoretical Prediction on Thermal and Mechanical Properties of High Entropy ({{Zr0}}.{{2Hf0}}.{{2Ti0}}.{{2Nb0}}.{{2Ta0}}.2){{C}} by Deep Learning Potential},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {43}
}

@article{deringerGeneralpurposeMachinelearningForce2020a,
    abstract = {Elemental phosphorus is attracting growing interest across fundamental and applied fields of research. However, atomistic simulations of phosphorus have remained an outstanding challenge. Here, we show that a universally applicable force field for phosphorus can be created by machine learning (ML) from a suitably chosen ensemble of quantum-mechanical results. Our model is fitted to density-functional theory plus many-body dispersion (DFT+MBD) data; its accuracy is demonstrated for the exfoliation of black and violet phosphorus (yielding monolayers of "phosphorene" and "hittorfene"); its transferability is shown for the transition between the molecular and network liquid phases. An application to a phosphorene nanoribbon on an experimentally relevant length scale exemplifies the power of accurate and flexible ML-driven force fields for next-generation materials modelling. The methodology promises new insights into phosphorus as well as other structurally complex, e.g., layered solids that are relevant in diverse areas of chemistry, physics, and materials science. Atomistic simulations of phosphorus represent a challenge due to the element's highly diverse allotropic structures. Here the authors propose a general-purpose machine-learning force field for elemental phosphorus, which can describe a broad range of relevant bulk and nanostructured allotropes.},
    annotation = {WOS:000591379900003},
    author = {Deringer, Volker L. and Caro, Miguel A. and Csanyi, Gabor},
    date = {2020-10-29},
    doi = {10.1038/s41467-020-19168-z},
    issn = {2041-1723},
    journaltitle = {Nature Communications},
    langid = {english},
    location = {{Berlin}},
    number = {1},
    pages = {5461},
    publisher = {{Nature Research}},
    shortjournal = {Nat. Commun.},
    title = {A General-Purpose Machine-Learning Force Field for Bulk and Nanostructured Phosphorus},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {11}
}

@article{deringerModellingUnderstandingBattery2020,
    abstract = {The realistic computer modelling of battery materials is an important research goal, with open questions ranging from atomic-scale structure and dynamics to macroscopic phenomena. Quantum-mechanical methods offer high accuracy and predictive power in small-scale atomistic simulations, but they quickly reach their limits when complex electrochemical systems are to be studied-for example, when structural disorder or even fully amorphous phases are present, or when reactions take place at the interface between electrodes and electrolytes. In this Perspective, it is argued that emerging machine learning based interatomic potentials are promising tools for studying battery materials on the atomistic and nanometre length scales, affording quantum-mechanical accuracy yet being many orders of magnitude faster, and thereby extending the capabilities of current battery modelling methodology. Initial applications to solid-state electrolyte and anode materials in lithium-ion batteries are highlighted, and future directions and possible synergies with experiments are discussed.},
    annotation = {WOS:000576549400001},
    author = {Deringer, Volker L.},
    date = {2020-10},
    doi = {10.1088/2515-7655/abb011},
    issn = {2515-7655},
    journaltitle = {Journal of Physics-Energy},
    langid = {english},
    location = {{Bristol}},
    number = {4},
    pages = {041003},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {J. Phys-Energy},
    title = {Modelling and Understanding Battery Materials with Machine-Learning-Driven Atomistic Simulations},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {2}
}

@article{dralHierarchicalMachineLearning2020a,
    abstract = {We present hierarchical machine learning (hML) of highly accurate potential energy surfaces (PESs). Our scheme is based on adding predictions of multiple Delta -machine learning models trained on energies and energy corrections calculated with a hierarchy of quantum chemical methods. Our (semi-)automatic procedure determines the optimal training set size and composition of each constituent machine learning model, simultaneously minimizing the computational effort necessary to achieve the required accuracy of the hML PES. Machine learning models are built using kernel ridge regression, and training points are selected with structure-based sampling. As an illustrative example, hML is applied to a high-level ab initio CH3Cl PES and is shown to significantly reduce the computational cost of generating the PES by a factor of 100 while retaining similar levels of accuracy (errors of similar to 1 cm(-1)).},
    annotation = {WOS:000537898900006},
    author = {Dral, Pavlo O. and Owens, Alec and Dral, Alexey and Csanyi, Gabor},
    date = {2020-05-29},
    doi = {10.1063/5.0006498},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {20},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Hierarchical Machine Learning of Potential Energy Surfaces},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {152}
}

@article{dralMLatomIntegrativePlatform2021a,
    abstract = {Atomistic machine learning (AML) simulations are used in chemistry at an ever-increasing pace. A large number of AML models has been developed, but their implementations are scattered among different packages, each with its own conventions for input and output. Thus, here we give an overview of our MLatom 2 software package, which provides an integrative platform for a wide variety of AML simulations by implementing from scratch and interfacing existing software for a range of state-of-the-art models. These include kernel method-based model types such as KREG (native implementation), sGDML, and GAP-SOAP as well as neural-network-based model types such as ANI, DeepPot-SE, and PhysNet. The theoretical foundations behind these methods are overviewed too. The modular structure of MLatom allows for easy extension to more AML model types. MLatom 2 also has many other capabilities useful for AML simulations, such as the support of custom descriptors, farthest-point and structure-based sampling, hyperparameter optimization, model evaluation, and automatic learning curve generation. It can also be used for such multi-step tasks as Delta-learning, self-correction approaches, and absorption spectrum simulation within the machine-learning nuclear-ensemble approach. Several of these MLatom 2 capabilities are showcased in application examples.},
    annotation = {WOS:000659201700002},
    author = {Dral, Pavlo O. and Ge, Fuchun and Xue, Bao-Xin and Hou, Yi-Fan and Pinheiro, Max and Huang, Jianxing and Barbatti, Mario},
    date = {2021-08},
    doi = {10.1007/s41061-021-00339-5},
    issn = {2365-0869},
    journaltitle = {Topics in Current Chemistry},
    langid = {english},
    location = {{Cham}},
    number = {4},
    pages = {27},
    publisher = {{Springer International Publishing Ag}},
    shortjournal = {Top. Curr. Chem.},
    shorttitle = {{{MLatom}} 2},
    title = {{{MLatom}} 2: {{An Integrative Platform}} for {{Atomistic Machine Learning}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {379}
}

@article{dralQuantumChemistryAge2020a,
    abstract = {As the quantum chemistry (QC) community embraces machine learning (ML), the number of new methods and applications based on the combination of QC and ML is surging. In this Perspective, a view of the current state of affairs in this new and exciting research field is offered, challenges of using machine learning in quantum chemistry applications are described, and potential future developments are outlined. Specifically, examples of how machine learning is used to improve the accuracy and accelerate quantum chemical research are shown. Generalization and classification of existing techniques are provided to ease the navigation in the sea of literature and to guide researchers entering the field. The emphasis of this Perspective is on supervised machine learning.},
    annotation = {WOS:000526339000051},
    author = {Dral, Pavlo O.},
    date = {2020-03-19},
    doi = {10.1021/acs.jpclett.9b03664},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {6},
    pages = {2336--2347},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    title = {Quantum {{Chemistry}} in the {{Age}} of {{Machine Learning}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {11}
}

@article{druchokEfficientGenerationCorrection2021a,
    abstract = {Efficient design and screening of the novel molecules is a major challenge in drug and material design. This paper focuses on a multi-stage pipeline, in which several deep neural network models are combined to map discrete molecular representations into continuous vector space to later generate from it new molecular structures with desired properties. Here, the Attention-based Sequence-to-Sequence model is added to "spellcheck" and correct generated structures, while the oversampling in the continuous space allows generating candidate structures with desired distribution for properties and molecular descriptors, even for a small reference datasets. We further use computer simulation to validate the desired properties in the numerical experiment. With the focus on the drug design, such a pipeline allows generating novel structures with a control of Synthetic Accessibility Score and a series of metrics that assess the drug-likeliness. Our code is available at .},
    annotation = {WOS:000617743400001},
    author = {Druchok, Maksym and Yarish, Dzvenymyra and Gurbych, Oleksandr and Maksymenko, Mykola},
    date = {2021-04-30},
    doi = {10.1002/jcc.26494},
    issn = {0192-8651},
    journaltitle = {Journal of Computational Chemistry},
    langid = {english},
    location = {{Hoboken}},
    number = {11},
    pages = {746--760},
    publisher = {{Wiley}},
    shortjournal = {J. Comput. Chem.},
    title = {Toward Efficient Generation, Correction, and Properties Control of Unique Drug-like Structures},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {42}
}

@article{druzbickiDynamicsSpectroscopyNeutronsRecent2021,
    abstract = {This work provides an up-to-date overview of recent developments in neutron spectroscopic techniques and associated computational tools to interrogate the structural properties and dynamical behavior of complex and disordered materials, with a focus on those of a soft and polymeric nature. These have and continue to pave the way for new scientific opportunities simply thought unthinkable not so long ago, and have particularly benefited from advances in high-resolution, broadband techniques spanning energy transfers from the meV to the eV. Topical areas include the identification and robust assignment of low-energy modes underpinning functionality in soft solids and supramolecular frameworks, or the quantification in the laboratory of hitherto unexplored nuclear quantum effects dictating thermodynamic properties. In addition to novel classes of materials, we also discuss recent discoveries around water and its phase diagram, which continue to surprise us. All throughout, emphasis is placed on linking these ongoing and exciting experimental and computational developments to specific scientific questions in the context of the discovery of new materials for sustainable technologies.},
    annotation = {WOS:000650724800001},
    author = {Druzbicki, Kacper and Gaboardi, Mattia and Fernandez-Alonso, Felix},
    date = {2021-05},
    doi = {10.3390/polym13091440},
    journaltitle = {Polymers},
    langid = {english},
    location = {{Basel}},
    number = {9},
    pages = {1440},
    publisher = {{Mdpi}},
    shortjournal = {Polymers},
    title = {Dynamics \& {{Spectroscopy}} with {{Neutrons}}-{{Recent Developments}} \& {{Emerging Opportunities}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {13}
}

@article{duanDataDrivenApproachesCan2020,
    abstract = {High-throughput computational screening typically employs methods (i.e., density functional theory or DFT) that can fail to describe challenging molecules, such as those with strongly correlated electronic structure. In such cases, multireference (MR) correlated wavefunction theory (WFT) would be the appropriate choice but remains more challenging to carry out and automate than single-reference (SR) WFT or DFT. Numerous diagnostics have been proposed for identifying when MR character is likely to have an effect on the predictive power of SR calculations, but conflicting conclusions about diagnostic performance have been reached on small data sets. We compute 15 MR diagnostics, ranging from affordable DFT-based to more costly MR-WFT-based diagnostics, on a set of 3165 equilibrium and distorted small organic molecules containing up to six heavy atoms. Conflicting MR character assignments and low pairwise linear correlations among diagnostics are also observed over this set. We evaluate the ability of existing diagnostics to predict the percent recovery of the correlation energy, \%E-corr. None of the DFT-based diagnostics are nearly as predictive of \%E-corr as the best WFT-based diagnostics. To overcome the limitation of this cost-accuracy trade-off, we develop machine learning (ML, i.e., kernel ridge regression) models to predict WFT-based diagnostics from a combination of DFT-based diagnostics and a new, size-independent 3D geometric representation. The ML-predicted diagnostics correlate as well with MR effects as their computed (i.e., with WFT) values, significantly improving over the DFT-based diagnostics on which the models were trained. These ML models thus provide a promising approach to improve upon DFT-based diagnostic accuracy while remaining suitably low cost for high-throughput screening.},
    annotation = {WOS:000607532300032},
    author = {Duan, Chenru and Liu, Fang and Nandy, Aditya and Kulik, Heather J.},
    date = {2020-07-14},
    doi = {10.1021/acs.jctc.0c00358},
    issn = {1549-9618},
    journaltitle = {Journal of Chemical Theory and Computation},
    langid = {english},
    location = {{Washington}},
    number = {7},
    pages = {4373--4387},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem. Theory Comput.},
    title = {Data-{{Driven Approaches Can Overcome}} the {{Cost}}-{{Accuracy Trade}}-{{Off}} in {{Multireference Diagnostics}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {16}
}

@article{duanLearningFailurePredicting2019a,
    abstract = {High-throughput computational screening for chemical discovery mandates the automated and unsupervised simulation of thousands of new molecules and materials. In challenging materials spaces, such as open shell transition metal chemistry, characterization requires time-consuming first-principles simulation that often necessitates human intervention. These calculations can frequently lead to a null result, e.g., the calculation does not converge or the molecule does not stay intact during a geometry optimization. To overcome this challenge toward realizing fully automated chemical discovery in transition metal chemistry, we have developed the first machine learning models that predict the likelihood of successful simulation outcomes. We train support vector machine and artificial neural network classifiers to predict simulation outcomes (i.e., geometry optimization result and degree of {$<$} S-2 {$>$} deviation) for a chosen electronic structure method based on chemical composition. For these static models, we achieve an area under the curve of at least 0.95, minimizing computational time spent on nonproductive simulations and therefore enabling efficient chemical space exploration. We introduce a metric of model uncertainty based on the distribution of points in the latent space to systematically improve model prediction confidence. In a complementary approach, we train a convolutional neural network classification model on simulation output electronic and geometric structure time series data. This dynamic model generalizes more readily than the static classifier by becoming more predictive as input simulation length increases. Finally, we describe approaches for using these models to enable autonomous job control in transition metal complex discovery.},
    annotation = {WOS:000464475500020},
    author = {Duan, Chenru and Janet, Jon Paul and Liu, Fang and Nandy, Aditya and Kulik, Heather J.},
    date = {2019-04},
    doi = {10.1021/acs.jctc.9b00057},
    issn = {1549-9618},
    journaltitle = {Journal of Chemical Theory and Computation},
    langid = {english},
    location = {{Washington}},
    number = {4},
    pages = {2331--2345},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem. Theory Comput.},
    shorttitle = {Learning from {{Failure}}},
    title = {Learning from {{Failure}}: {{Predicting Electronic Structure Calculation Outcomes}} with {{Machine Learning Models}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {15}
}

@article{ellisAcceleratingFinitetemperatureKohnSham2021a,
    abstract = {We present a numerical modeling workflow based on machine learning which reproduces the total energies produced by Kohn-Sham density functional theory (DFT) at finite electronic temperature to within chemical accuracy at negligible computational cost. Based on deep neural networks, our workflow yields the local density of states (LDOS) for a given atomic configuration. From the LDOS, spatially resolved, energy-resolved, and integrated quantities can be calculated, including the DFT total free energy, which serves as the Born-Oppenheimer potential energy surface for the atoms. We demonstrate the efficacy of this approach for both solid and liquid metals and compare results between independent and unified machine-learning models for solid and liquid aluminum. Our machine-learning density functional theory framework opens up the path towards multiscale materials modeling for matter under ambient and extreme conditions at a computational scale and cost that is unattainable with current algorithms.},
    annotation = {WOS:000671587300005},
    author = {Ellis, J. A. and Fiedler, L. and Popoola, G. A. and Modine, N. A. and Stephens, J. A. and Thompson, A. P. and Cangi, A. and Rajamanickam, S.},
    date = {2021-07-08},
    doi = {10.1103/PhysRevB.104.035120},
    issn = {2469-9950},
    journaltitle = {Physical Review B},
    langid = {english},
    location = {{College Pk}},
    number = {3},
    pages = {035120},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. B},
    title = {Accelerating Finite-Temperature {{Kohn}}-{{Sham}} Density Functional Theory with Deep Neural Networks},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {104}
}

@article{Galib,
    abstract = {Nitrogen oxides are removed from the troposphere through the reactive uptake of N 2 O 5 into aqueous aerosol. This process is thought to occur within the bulk of an aerosol, through solvation and subsequent hydrolysis. However, this perspective is difficult to reconcile with field measurements and cannot be verified directly because of the fast reaction kinetics of N 2 O 5. Here, we use molecular simulations, including reactive potentials and importance sampling, to study the uptake of N 2 O 5 into an aqueous aerosol. Rather than being mediated by the bulk, uptake is dominated by interfacial processes due to facile hydrolysis at the liquid-vapor interface and competitive reevaporation. With this molecular information, we propose an alternative interfacial reactive uptake model consistent with existing experimental observations.},
    author = {Galib, M and Limmer, DT},
    journaltitle = {science.sciencemag.org},
    title = {Reactive Uptake of {{N2O5}} by Atmospheric Aerosol Is Dominated by Interfacial Processes},
    url = {https://science.sciencemag.org/content/371/6532/921.abstract}
}

@article{gaoDeepLearningProtein2020a,
    abstract = {Deep learning is catalyzing a scientific revolution fueled by big data, accessible toolkits, and powerful computational resources, impacting many fields, including protein structural modeling. Protein structural modeling, such as predicting structure from amino acid sequence and evolutionary information, designing proteins toward desirable functionality, or predicting properties or behavior of a protein, is critical to understand and engineer biological systems at the molecular level. In this review, we summarize the recent advances in applying deep learning techniques to tackle problems in protein structural modeling and design. We dissect the emerging approaches using deep learning techniques for protein structural modeling and discuss advances and challenges that must be addressed. We argue for the central importance of structure, following the "sequence/structure/function'' paradigm. This review is directed to help both computational biologists to gain familiarity with the deep learning methods applied in protein modeling, and computer scientists to gain perspective on the biologically meaningful problems that may benefit from deep learning techniques.},
    annotation = {WOS:000653831500007},
    author = {Gao, Wenhao and Mahajan, Sai Pooja and Sulam, Jeremias and Gray, Jeffrey J.},
    date = {2020-12-11},
    doi = {10.1016/j.patter.2020.100142},
    issn = {2666-3899},
    journaltitle = {Patterns},
    langid = {english},
    location = {{Amsterdam}},
    number = {9},
    pages = {100142},
    publisher = {{Elsevier}},
    shortjournal = {Patterns},
    title = {Deep {{Learning}} in {{Protein Structural Modeling}} and {{Design}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {1}
}

@article{gaoShortSolventModel2020a,
    abstract = {Coulomb interactions play a major role in determining the thermodynamics, structure, and dynamics of condensed-phase systems, but often present significant challenges. Computer simulations usually use periodic boundary conditions to minimize corrections from finite cell boundaries but the long range of the Coulomb interactions generates significant contributions from distant periodic images of the simulation cell, usually calculated by Ewald sum techniques. This can add significant overhead to computer simulations and hampers the development of intuitive local pictures and simple analytic theory. In this paper, we present a general framework based on local molecular field theory to accurately determine the contributions from long-ranged Coulomb interactions to the potential of mean force between ionic or apolar hydrophobic solutes in dilute aqueous solutions described by standard classical point charge water models. The simplest approximation leads to a short solvent (SS) model, with truncated solvent-solvent and solute-solvent Coulomb interactions and long-ranged but screened Coulomb interactions only between charged solutes. The SS model accurately describes the interplay between strong short-ranged solute core interactions, local hydrogen-bond configurations, and long-ranged dielectric screening of distant charges, competing effects that are difficult to capture in standard implicit solvent models.},
    annotation = {WOS:000508977600017},
    author = {Gao, Ang and Remsing, Richard C. and Weeks, John D.},
    date = {2020-01-21},
    doi = {10.1073/pnas.1918981117},
    issn = {0027-8424},
    journaltitle = {Proceedings of the National Academy of Sciences of the United States of America},
    langid = {english},
    location = {{Washington}},
    number = {3},
    pages = {1293--1302},
    publisher = {{Natl Acad Sciences}},
    shortjournal = {Proc. Natl. Acad. Sci. U. S. A.},
    title = {Short Solvent Model for Ion Correlations and Hydrophobic Association},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {117}
}

@article{gaoTorchANIFreeOpen2020a,
    abstract = {This paper presents TorchANI, a PyTorch-based program for training/inference of ANI (ANAKIN-ME) deep learning models to obtain potential energy surfaces and other physical properties of molecular systems. ANI is an accurate neural network potential originally implemented using C++/CUDA in a program called NeuroChem. Compared with NeuroChem, TorchANI has a design emphasis on being lightweight, user friendly, cross platform, and easy to read and modify for fast prototyping, while allowing acceptable sacrifice on running performance. Because the computation of atomic environmental vectors and atomic neural networks are all implemented using PyTorch operators, TorchANI is able to use PyTorch's autograd engine to automatically compute analytical forces and Hessian matrices, as well as do force training without requiring any additional codes. TorchANI is open-source and freely available on GitHub: https://github.com/aiqm/torchani.},
    annotation = {WOS:000557375300011},
    author = {Gao, Xiang and Ramezanghorbani, Farhad and Isayev, Olexandr and Smith, Justin S. and Roitberg, Adrian E.},
    date = {2020-07-27},
    doi = {10.1021/acs.jcim.0c00451},
    issn = {1549-9596},
    journaltitle = {Journal of Chemical Information and Modeling},
    langid = {english},
    location = {{Washington}},
    number = {7},
    pages = {3408--3415},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem Inf. Model.},
    shorttitle = {{{TorchANI}}},
    title = {{{TorchANI}}: {{A Free}} and {{Open Source PyTorch}}-{{Based Deep Learning Implementation}} of the {{ANI Neural Network Potentials}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {60}
}

@article{georgeCombiningPhononAccuracy2020,
    abstract = {Machine learning driven interatomic potentials, including Gaussian approximation potential (GAP) models, are emerging tools for atomistic simulations. Here, we address the methodological question of how one can fit GAP models that accurately predict vibrational properties in specific regions of configuration space while retaining flexibility and transferability to others. We use an adaptive regularization of the GAP fit that scales with the absolute force magnitude on any given atom, thereby exploring the Bayesian interpretation of GAP regularization as an "expected error" and its impact on the prediction of physical properties for a material of interest. The approach enables excellent predictions of phonon modes (to within 0.1 THz-0.2 THz) for structurally diverse silicon allotropes, and it can be coupled with existing fitting databases for high transferability across different regions of configuration space, which we demonstrate for liquid and amorphous silicon. These findings and workflows are expected to be useful for GAP-driven materials modeling more generally.},
    annotation = {WOS:000556313400002},
    author = {George, Janine and Hautier, Geoffroy and Bartok, Albert P. and Csanyi, Gabor and Deringer, Volker L.},
    date = {2020-07-28},
    doi = {10.1063/5.0013826},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {4},
    pages = {044104},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Combining Phonon Accuracy with High Transferability in {{Gaussian}} Approximation Potential Models},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {153}
}

@article{goscinskiRoleFeatureSpace2021a,
    abstract = {Efficient, physically-inspired descriptors of the structure and composition of molecules and materials play a key role in the application of machine-learning techniques to atomistic simulations. The proliferation of approaches, as well as the fact that each choice of features can lead to very different behavior depending on how they are used, e.g. by introducing non-linear kernels and non-Euclidean metrics to manipulate them, makes it difficult to objectively compare different methods, and to address fundamental questions on how one feature space is related to another. In this work we introduce a framework to compare different sets of descriptors, and different ways of transforming them by means of metrics and kernels, in terms of the structure of the feature space that they induce. We define diagnostic tools to determine whether alternative feature spaces contain equivalent amounts of information, and whether the common information is substantially distorted when going from one feature space to another. We compare, in particular, representations that are built in terms of n-body correlations of the atom density, quantitatively assessing the information loss associated with the use of low-order features. We also investigate the impact of different choices of basis functions and hyperparameters of the widely used SOAP and Behler-Parrinello features, and investigate how the use of non-linear kernels, and of a Wasserstein-type metric, change the structure of the feature space in comparison to a simpler linear feature space.},
    annotation = {WOS:000660866700001},
    author = {Goscinski, Alexander and Fraux, Guillaume and Imbalzano, Giulio and Ceriotti, Michele},
    date = {2021-06},
    doi = {10.1088/2632-2153/abdaf7},
    journaltitle = {Machine Learning-Science and Technology},
    langid = {english},
    location = {{Bristol}},
    number = {2},
    pages = {025028},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {Mach. Learn.-Sci. Technol.},
    title = {The Role of Feature Space in Atomistic Learning},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {2}
}

@article{govoniCodeInteroperabilityExtends2021a,
    abstract = {The functionality of many materials is critically dependent on the integration of dissimilar components and on the interfaces that arise between them. The description of such heterogeneous components requires the development and deployment of first principles methods, coupled to appropriate dynamical descriptions of matter and advanced sampling techniques, in order to capture all the relevant length and time scales of importance to the materials' performance. It is thus essential to build simple, streamlined computational schemes for the prediction and design of multiple properties of broad classes of materials, by developing interoperable codes which can be efficiently coupled to each other to perform complex tasks. We discuss the use of interoperable codes to simulate the structural and spectroscopic characterization of materials, including chemical reactions for catalysis, the description of defects for quantum information science, and heat and charge transport.},
    annotation = {WOS:000621097500001},
    author = {Govoni, Marco and Whitmer, Jonathan and de Pablo, Juan and Gygi, Francois and Galli, Giulia},
    date = {2021-02-19},
    doi = {10.1038/s41524-021-00501-z},
    journaltitle = {Npj Computational Materials},
    langid = {english},
    location = {{Berlin}},
    number = {1},
    options = {useprefix=true},
    pages = {32},
    publisher = {{Nature Research}},
    shortjournal = {npj Comput. Mater.},
    title = {Code Interoperability Extends the Scope of Quantum Simulations},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {7}
}

@article{grisafiIncorporatingLongrangePhysics2019a,
    abstract = {The most successful and popular machine learning models of atomic-scale properties derive their transferability from a locality ansatz. The properties of a large molecule or a bulk material are written as a sum over contributions that depend on the configurations within finite atom-centered environments. The obvious downside of this approach is that it cannot capture nonlocal, nonadditive effects such as those arising due to long-range electrostatics or quantum interference. We propose a solution to this problem by introducing nonlocal representations of the system, which are remapped as feature vectors that are defined locally and are equivariant in O(3). We consider, in particular, one form that has the same asymptotic behavior as the electrostatic potential. We demonstrate that this framework can capture nonlocal, long-range physics by building a model for the electrostatic energy of randomly distributed point-charges, for the unrelaxed binding curves of charged organic molecular dimers, and for the electronic dielectric response of liquid water. By combining a representation of the system that is sensitive to long-range correlations with the transferability of an atom-centered additive model, this method outperforms current state-of-the-art machine-learning schemes and provides a conceptual framework to incorporate nonlocal physics into atomistic machine learning. Published under license by AIP Publishing.},
    annotation = {WOS:000504063200009},
    author = {Grisafi, Andrea and Ceriotti, Michele},
    date = {2019-11-28},
    doi = {10.1063/1.5128375},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {20},
    pages = {204105},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Incorporating Long-Range Physics in Atomic-Scale Machine Learning},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {151}
}

@article{grisafiMultiscaleApproachPrediction2021a,
    abstract = {Electronic nearsightedness is one of the fundamental principles that governs the behavior of condensed matter and supports its description in terms of local entities such as chemical bonds. Locality also underlies the tremendous success of machine-learning schemes that predict quantum mechanical observables - such as the cohesive energy, the electron density, or a variety of response properties - as a sum of atom-centred contributions, based on a short-range representation of atomic environments. One of the main shortcomings of these approaches is their inability to capture physical effects ranging from electrostatic interactions to quantum delocalization, which have a long-range nature. Here we show how to build a multi-scale scheme that combines in the same framework local and non-local information, overcoming such limitations. We show that the simplest version of such features can be put in formal correspondence with a multipole expansion of permanent electrostatics. The data-driven nature of the model construction, however, makes this simple form suitable to tackle also different types of delocalized and collective effects. We present several examples that range from molecular physics to surface science and biophysics, demonstrating the ability of this multi-scale approach to model interactions driven by electrostatics, polarization and dispersion, as well as the cooperative behavior of dielectric response functions.},
    annotation = {WOS:000619216100010},
    author = {Grisafi, Andrea and Nigam, Jigyasa and Ceriotti, Michele},
    date = {2021-02-14},
    doi = {10.1039/d0sc04934d},
    issn = {2041-6520},
    journaltitle = {Chemical Science},
    langid = {english},
    location = {{Cambridge}},
    number = {6},
    pages = {2078--2090},
    publisher = {{Royal Soc Chemistry}},
    shortjournal = {Chem. Sci.},
    title = {Multi-Scale Approach for the Prediction of Atomic Scale Properties},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {12}
}

@article{Grubisic2021,
    abstract = {We present PyXtal\textsubscript{F}F-a package based on Python programming language-for developing machine learning potentials (MLPs). The aim of PyXtal\textsubscript{F}F is to promote the application of atomistic simulations through providing several choices of atom-centered descriptors and machine learning regressions in one platform. Based on the given choice of descriptors (including the atom-centered symmetry functions, embedded atom density, SO4 bispectrum, and smooth SO3 power spectrum), PyXtal\textsubscript{F}F can train MLPs with either generalized linear regression or neural network models, by simultaneously minimizing the errors of energy/forces/stress tensors in comparison with the data from ab-initio simulations. The trained MLP model from PyXtal\textsubscript{F}F is interfaced with the Atomic Simulation Environment (ASE) package, which allows different types of lightweight simulations such as geometry optimization, molecular dynamics simulation, and physical properties prediction. Finally, we will illustrate the performance of PyXtal\textsubscript{F}F by applying it to investigate several material systems, including the bulk SiO 2 , high entropy alloy NbMoTaW, and elemental Pt for general purposes. Full documentation of PyXtal\textsubscript{F}F is available at https://pyxtal-ff.readthedocs.io.},
    author = {Grubišić, L and Hajba, M and Entropy, D Lacmanović -},
    date = {2021},
    doi = {10.1088/2632-2153/abc940},
    journaltitle = {mdpi.com},
    pages = {27001},
    title = {Deep Neural Network Model for Approximating Eigenmodes Localized by a Confining Potential},
    url = {https://www.mdpi.com/956466},
    volume = {2}
}

@article{guAdaptiveIronbasedMagnetic,
    abstract = {With unique physicochemical properties and biological effects, magnetic nanomaterials (MNMs) play a crucial role in the biomedical field. In particular, magnetic iron oxide nanoparticles (MIONPs) are approved by the United States Food and Drug Administration (FDA) for clinical applications at present due to their low toxicity, biocompatibility, and biodegradability. Despite the unarguable effectiveness, massive space for improving such materials' performance still needs to be filled. Recently, many efforts have been devoted to improving the preparation methods based on the materials' biosafety. Besides, researchers have successfully regulated the performance of magnetic nanoparticles (MNPs) by changing their sizes, morphologies, compositions; or by aggregating as-synthesized MNPs in an orderly arrangement to meet various clinical requirements. The rise of cloud computing and artificial intelligence techniques provides novel ways for fast material characterization, automated data analysis, and mechanism demonstration. In this review, we summarized the studies that focused on the preparation routes and performance regulations of high-quality MNPs, and their special properties applied in biomedical detection, diagnosis, and treatment. At the same time, the future development of MNMs was also discussed.},
    annotation = {WOS:000663236300002},
    author = {Gu, Ning and Zhang, Zuoheng and Li, Yan},
    doi = {10.1007/s12274-021-3546-1},
    issn = {1998-0124},
    journaltitle = {Nano Research},
    langid = {english},
    location = {{Beijing}},
    publisher = {{Tsinghua Univ Press}},
    shortjournal = {Nano Res.},
    title = {Adaptive Iron-Based Magnetic Nanomaterials of High Performance for Biomedical Applications},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06}
}

@article{gubaevFinitetemperatureInterplayStructural2021,
    abstract = {An active learning approach to train machine-learning interatomic potentials (moment tensor potentials) for multicomponent alloys to ab initio data is presented. Employing this approach, the disordered body-centered cubic (bcc) TiZrHfTax system with varying Ta concentration is investigated via molecular dynamics simulations. Our results show a strong interplay between elastic properties and the structural. phase stability, strongly affecting the mechanical properties. Based on these insights we systematically screen composition space for regimes where elastic constants show little or no temperature dependence (elinvar effect).},
    annotation = {WOS:000671591000001},
    author = {Gubaev, Konstantin and Ikeda, Yuji and Tasnadi, Ferenc and Neugebauer, Joerg and Shapeev, Alexander and Grabowski, Blazej and Koermann, Fritz},
    date = {2021-07-08},
    doi = {10.1103/PhysRevMaterials.5.073801},
    issn = {2475-9953},
    journaltitle = {Physical Review Materials},
    langid = {english},
    location = {{College Pk}},
    number = {7},
    pages = {073801},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. Mater.},
    title = {Finite-Temperature Interplay of Structural Stability, Chemical Complexity, and Elastic Properties of Bcc Multicomponent Alloys from Ab Initio Trained Machine-Learning Potentials},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {5}
}

@article{guglerEnumerationNovoInorganic2020a,
    abstract = {Despite being attractive targets for functional materials, the discovery of transition metal complexes with high-throughput computational screening is challenged by the amount of feasible coordination numbers, spin states, or oxidation states and the potentially large sizes of ligands. To overcome these limitations, we take inspiration from organic chemistry where full enumeration of neutral, closed-shell molecules under the constraint of size has enriched discovery efforts. We design monodentate and bidentate ligands from scratch for the construction of mononuclear, octahedral transition metal complexes with up to 13 heavy atoms (i.e., metal, C, N, O, P, or S). From {$>$}11 000 theoretical ligands, we develop a heuristic score for ranking a chemically feasible 2500 ligand subset, only 71 of which were previously included in common organic molecule databases. We characterize the top 20\% of scored ligands with density functional theory (DFT) in an octahedral homoleptic ligand database (OHLDB). The OHLDB contains i) the geometry optimized structures of 1250 homoleptic octahedral complexes obtained from the enumerated pool of ligands and an open-shell transition metal (M(ii)/M(iii), M = Cr, Mn, Fe, or Co) and ii) the resulting high-spin/low-spin adiabatic electronic energy differences (Delta EH-L) obtained with hybrid DFT. Over the OHLDB, we observe structure-property (i.e., Delta EH-L) relationships different from those expected on the basis of ligand field arguments or from our prior data sets. Finally, we demonstrate how incorporating OHLDB data into artificial neural network (ANN) training improves ANN out-of-sample performance on much larger transition metal complexes.},
    annotation = {WOS:000508398900010},
    author = {Gugler, Stefan and Janet, Jon Paul and Kulik, Heather J.},
    date = {2020-01-01},
    doi = {10.1039/c9me00069k},
    issn = {2058-9689},
    journaltitle = {Molecular Systems Design \& Engineering},
    langid = {english},
    location = {{Cambridge}},
    number = {1},
    pages = {139--152},
    publisher = {{Royal Soc Chemistry}},
    shortjournal = {Mol. Syst. Des. Eng.},
    title = {Enumeration of de Novo Inorganic Complexes for Chemical Discovery and Machine Learning},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {5}
}

@article{guHighrepetitionrateFemtosecondMidinfrared2019a,
    abstract = {We demonstrate an effective method to obtain high-repetition-rate femtosecond mid-infrared (mid-IR) pulses by nonlinear optical modulation of mid-IR continuous-wave (CW) quantum and interband cascade lasers (ICLs and QCLs). In the experiment, a high-repetition-rate femtosecond ytterbium-doped fiber laser with nanojoule-level pulse energy was used as the pump source of optical parametric amplifiers to modulate and amplify the mid-IR CW laser. Near transform-limited 84 fs duration (7.3 cycles) mid-IR pulses were generated with above 200 mW average power and a repetition rate of 160 MHz. Moreover, the spectral tunability was demonstrated using CW QCL at different wavelengths. The scheme offered a simple method to produce high-repetition-rate ultrashort pulses and that can be flexibly adopted in other mid-IR regions. (C) 2019 Optical Society of America},
    annotation = {WOS:000499141000052},
    author = {Gu, Chenglin and Zuo, Zhong and Luo, Daping and Peng, Daowang and Di, Yuanfeng and Zou, Xing and Yang, Liu and Li, Wenxue},
    date = {2019-12-01},
    doi = {10.1364/OL.44.005848},
    issn = {0146-9592},
    journaltitle = {Optics Letters},
    langid = {english},
    location = {{Washington}},
    number = {23},
    pages = {5848--5851},
    publisher = {{Optical Soc Amer}},
    shortjournal = {Opt. Lett.},
    title = {High-Repetition-Rate Femtosecond Mid-Infrared Pulses Generated by Nonlinear Optical Modulation of Continuous-Wave {{QCLs}} and {{ICLs}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {44}
}

@article{guoBergmantypeMediumRange2019a,
    abstract = {In recent years, some arguments about the existence of medium-range order (MRO) in the Zr-Rh system have been put forward. However, research on the structural features of the Zr-Rh binary alloy at the atomic level is still lacking. This study uses ab initio molecular dynamics simulations to systematically study the local structures of Zr77Rh23 from the liquid to the glassy states. Pair correlation function (PCF), coordination number (CN), Honeycutt-Anderson(HA) index, bond-angle distribution functions, and the Voronoi tessellation method are used to reveal a clear icosahedral-like configuration in the amorphous Zr77Rh23 alloy. It is noteworthy that the splitting in the second peak of the partial PDF implies the existence of a medium range order (MRO) in the Zr77Rh23 system. We obtain the local order in three-dimensional atomic density distributions by using a new atomistic cluster alignment (ACA) method. Interestingly, a Bergman-type MRO is observed in the glassy Zr77Rh23. Furthermore, the spatial distribution and connections of icosahedral-like clusters are shown to further demonstrate the MRO network. Our findings shed light on the nature of atomic local structures of amorphous Zr77Rh23 alloy and have important implications to understanding the formation of various MROs in metallic glasses. (C) 2019 Elsevier B.V. All rights reserved.},
    annotation = {WOS:000464663600078},
    author = {Guo, Y. R. and Qiao, Chong and Wang, J. J. and Shen, H. and Wang, S. Y. and Zheng, Y. X. and Zhang, R. J. and Chen, L. Y. and Su, Wan-Sheng and Wang, C. Z. and Ho, K. M.},
    date = {2019-06-25},
    doi = {10.1016/j.jallcom.2019.03.197},
    issn = {0925-8388},
    journaltitle = {Journal of Alloys and Compounds},
    langid = {english},
    location = {{Lausanne}},
    pages = {675--682},
    publisher = {{Elsevier Science Sa}},
    shortjournal = {J. Alloy. Compd.},
    title = {Bergman-Type Medium Range Order in Amorphous {{Zr77Rh23}} Alloy Studied by Ab Initio Molecular Dynamics Simulations},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {790}
}

@article{hajibabaeiSparseGaussianProcess2021a,
    abstract = {For machine learning of interatomic potentials a scalable sparse Gaussian process regression formalism is introduced with a data-efficient on-the-fly adaptive sampling algorithm. With this approach, the computational cost is effectively reduced to those of the Bayesian linear regression methods while maintaining the appealing characteristics of the exact Gaussian process regression. As a showcase, experimental melting and glass-crystallization temperatures are reproduced for Li7P3S11, Li diffusivity is simulated, and an unchartered phase is revealed with much lower Li diffusivity which should be circumvented.},
    annotation = {WOS:000657121500002},
    author = {Hajibabaei, Amir and Myung, Chang Woo and Kim, Kwang S.},
    date = {2021-06-01},
    doi = {10.1103/PhysRevB.103.214102},
    issn = {2469-9950},
    journaltitle = {Physical Review B},
    langid = {english},
    location = {{College Pk}},
    number = {21},
    pages = {214102},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. B},
    shorttitle = {Sparse {{Gaussian}} Process Potentials},
    title = {Sparse {{Gaussian}} Process Potentials: {{Application}} to Lithium Diffusivity in Superionic Conducting Solid Electrolytes},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {103}
}

@article{hajinazarMAISEConstructionNeural,
    abstract = {Module for ab initio structure evolution (MAISE) is an open-source package for materials modeling and prediction. The code's main feature is an automated generation of neural network (NN) interatomic potentials for use in global structure searches. The systematic construction of Behler-Parrinello-type NN models approximating ab initio energy and forces relies on two approaches introduced in our recent studies. An evolutionary sampling scheme for generating reference structures improves the NNs' mapping of regions visited in unconstrained searches, while a stratified training approach enables the creation of standardized NN models for multiple elements. A more flexible NN architecture proposed here expands the applicability of the stratified scheme for an arbitrary number of elements. The full workflow in the NN development is managed with a customizable 'MAISE-NET' wrapper written in Python. The global structure optimization capability in MAISE is based on an evolutionary algorithm applicable for nanopar-ticles, films, and bulk crystals. A multitribe extension of the algorithm allows for an efficient simultaneous optimization of nanoparticles in a given size range. Implemented structure analysis functions include fingerprinting with radial distribution functions and finding space groups with the SPGLIB tool. This work overviews MAISE's available features, constructed models, and confirmed predictions.},
    archiveprefix = {arXiv},
    arxivid = {2005.12131v2},
    author = {Hajinazar, S and Thorn, A and Sandoval, ED},
    eprint = {2005.12131v2},
    eprinttype = {arxiv},
    isbn = {2005.12131v2},
    journaltitle = {Elsevier},
    title = {{{MAISE}}: {{Construction}} of Neural Network Interatomic Models and Evolutionary Structure Optimization},
    url = {https://www.sciencedirect.com/science/article/pii/S0010465520303301}
}

@article{haMachineLearningAssistedExcited2021,
    abstract = {We present a machine learning-assisted excited state molecular dynamics (ML-ESMD) based on the ensemble density functional theory framework. Since we represent a diabatic Hamiltonian in terms of generalized valence bond ansatz within the state-interaction state-averaged spin-restricted ensemble-referenced Kohn-Sham (SI-SA-REKS) method, we can avoid singularities near conical intersections, which are crucial in excited state molecular dynamics simulations. We train the diabatic Hamiltonian elements and their analytical gradients with the SchNet architecture to construct machine learning models, while the phase freedom of off-diagonal elements of the Hamiltonian is cured by introducing the phase-less loss function. Our machine learning models show reasonable accuracy with mean absolute errors of similar to 0.1 kcal/mol and similar to 0.5 kcal/mol/A for the diabatic Hamiltonian elements and their gradients, respectively, for penta-2,4-dieniminium cation. Moreover, by exploiting the diabatic representation, our models can predict correct conical intersection structures and their topologies. In addition, our ML-ESMD simulations give almost identical result with a direct dynamics at the same level of theory.},
    annotation = {WOS:000634678200009},
    author = {Ha, Jong-Kwon and Kim, Kicheol and Min, Seung Kyu},
    date = {2021-02-09},
    doi = {10.1021/acs.jctc.0c01261},
    issn = {1549-9618},
    journaltitle = {Journal of Chemical Theory and Computation},
    langid = {english},
    location = {{Washington}},
    number = {2},
    pages = {694--702},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem. Theory Comput.},
    title = {Machine {{Learning}}-{{Assisted Excited State Molecular Dynamics}} with the {{State}}-{{Interaction State}}-{{Averaged Spin}}-{{Restricted Ensemble}}-{{Referenced Kohn}}-{{Sham Approach}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {17}
}

@article{hanDynamicObservationDendritic2020a,
    abstract = {We report the laser-induced solid-state transformation between a periodic "approximant" and quasicrystal in the Al-Cr system during rapid quenching. Dynamic transmission electron microscopy allows us to capture in situ the dendritic growth of the metastable quasicrystals. The formation of dendrites during solid-state transformation is a rare phenomenon, which we attribute to the structural similarity between the two intermetallics. Through ab initio molecular dynamics simulations, we identify the dominant structural motif to be a 13-atom icosahedral cluster transcending the phases of matter.},
    annotation = {WOS:000587289700008},
    author = {Han, Insung and McKeown, Joseph T. and Tang, Ling and Wang, Cai-Zhuang and Parsamehr, Hadi and Xi, Zhucong and Lu, Ying-Rui and Kramer, Matthew J. and Shahani, Ashwin J.},
    date = {2020-11-06},
    doi = {10.1103/PhysRevLett.125.195503},
    issn = {0031-9007},
    journaltitle = {Physical Review Letters},
    langid = {english},
    location = {{College Pk}},
    number = {19},
    pages = {195503},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. Lett.},
    title = {Dynamic {{Observation}} of {{Dendritic Quasicrystal Growth}} upon {{Laser}}-{{Induced Solid}}-{{State Transformation}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {125}
}

@article{hanMachineLearningApproach2021a,
    abstract = {A proper treatment of electron correlation effects is indispensable for accurate simulation of compounds. Various post-Hartree-Fock methods have been adopted to calculate correlation energies of chemical systems, but time complexity usually prevents their usage in a large scale. Here, we propose a density functional approximation, based on machine learning using neural networks, which can be readily employed to produce results comparable to second-order Moller-Plesset perturbation (MP2) ones for organic compounds with reduced computational cost. Various systems have been tested and the transferability across basis sets, structures, and nuclear configurations has been evaluated. Only a small number of molecules at the equilibrium structure has been needed for the training, and generally less than 5\% relative error has been achieved for structures outside the training domain and systems containing about 140 atoms. In addition, this approach has been applied to make predictions on correlation energies of nuclear configurations extracted from density functional theory-based molecular dynamics trajectories with only one or two structures as training data.},
    annotation = {WOS:000634678200016},
    author = {Han, Ruocheng and Rodriguez-Mayorga, Mauricio and Luber, Sandra},
    date = {2021-02-09},
    doi = {10.1021/acs.jctc.0c00898},
    issn = {1549-9618},
    journaltitle = {Journal of Chemical Theory and Computation},
    langid = {english},
    location = {{Washington}},
    number = {2},
    pages = {777--790},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem. Theory Comput.},
    title = {A {{Machine Learning Approach}} for {{MP2 Correlation Energies}} and {{Its Application}} to {{Organic Compounds}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {17}
}

@article{hanTrajectorybasedMachineLearning2020a,
    abstract = {Ab initiomolecular dynamics (AIMD) has become a popular simulation technique but long simulation times are often hampered due to its high computational effort. Alternatively, classical molecular dynamics (MD) based on force fields may be used, which, however, has certain shortcomings compared to AIMD. In order to alleviate that situation, a trajectory-based machine learning (TrajML) approach is introduced for the construction of force fields by learning from AIMD trajectories. Only nuclear trajectories are required, which can be obtained by other methods beyond AIMD as well. We developed an easy-to-use MD machine learning package (TrajML MD) for instant modelling of the force field and system-focussed prediction of molecular configurations for MD trajectories. It consumes similar computational resources as classical MD but can simulate complex systems with a higher accuracy due to the targeted learning on the system of interest.},
    annotation = {WOS:000547403200001},
    author = {Han, R. and Luber, S.},
    date = {2020-10-17},
    doi = {10.1080/00268976.2020.1788189},
    issn = {0026-8976},
    journaltitle = {Molecular Physics},
    langid = {english},
    location = {{Abingdon}},
    number = {19-20},
    publisher = {{Taylor \& Francis Ltd}},
    shortjournal = {Mol. Phys.},
    title = {Trajectory-Based Machine Learning Method and Its Application to Molecular Dynamics},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {118}
}

@article{hanUniformlyAccurateMachine2019,
    abstract = {A framework is introduced for constructing interpretable and truly reliable reduced models for multiscale problems in situations without scale separation. Hydrodynamic approximation to the kinetic equation is used as an example to illustrate the main steps and issues involved. To this end, a set of generalized moments are constructed first to optimally represent the underlying velocity distribution. The well-known closure problem is then solved with the aim of best capturing the associated dynamics of the kinetic equation. The issue of physical constraints such as Galilean invariance is addressed and an active-learning procedure is introduced to help ensure that the dataset used is representative enough. The reduced system takes the form of a conventional moment system and works regardless of the numerical discretization used. Numerical results are presented for the BGK (Bhatnagar-GrossKrook) model and binary collision of Maxwell molecules. We demonstrate that the reduced model achieves a uniform accuracy in a wide range of Knudsen numbers spanning from the hydrodynamic limit to free molecular flow.},
    annotation = {WOS:000493720200012},
    author = {Han, Jiequn and Ma, Chao and Ma, Zheng and E, Weinan},
    date = {2019-10-29},
    doi = {10.1073/pnas.1909854116},
    issn = {0027-8424},
    journaltitle = {Proceedings of the National Academy of Sciences of the United States of America},
    langid = {english},
    location = {{Washington}},
    number = {44},
    pages = {21983--21991},
    publisher = {{Natl Acad Sciences}},
    shortjournal = {Proc. Natl. Acad. Sci. U. S. A.},
    title = {Uniformly Accurate Machine Learning-Based Hydrodynamic Models for Kinetic Equations},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {116}
}

@article{harmonValidatingFirstprinciplesMolecular2020a,
    abstract = {Metal oxide/water interfaces play a crucial role in many electrochemical and photocatalytic processes, such as photoelectrochemical water splitting, the creation of fuel from sunlight, and electrochemical CO2 reduction. First-principles electronic structure calculations can reveal unique insights into these processes, such as the role of the alignment of the oxide electronic energy levels with those of liquid water. An essential prerequisite for the success of such calculations is the ability to predict accurate structural models of these interfaces, which in turn requires careful experimental validation. Here we report a general, quantitative validation protocol for first-principles molecular dynamics simulations of oxide/aqueous interfaces. The approach makes direct comparisons of interfacial x-ray reflectivity (XR) signals from experimental measurements and those obtained from ab initio simulations with semilocal and van der Waals functionals. The protocol is demonstrated here for the case of the Al2O3(001)/water interface, one of the simplest oxide/water interfaces. We discuss the technical requirements needed for validation, including the choice of the density functional, the simulation cell size, and the optimal choice of the thermodynamic ensemble. Our results establish a general paradigm for the validation of structural models and interactions at solid/water interfaces derived from first-principles simulations. While there is qualitative agreement between the simulated structures and the experimental best-fit structure, direct comparisons of simulated and measured XR intensities show quantitative discrepancies that derive from both bulk regions (i.e., alumina and water) as well as the interfacial region, highlighting the need for accurate density functionals to properly describe interfacial interactions. Our results show that XR data are sensitive not only to the atomic structure (i.e., the atom locations) but also to the electron-density distributions in both the substrate and at the interface.},
    annotation = {WOS:000591531700008},
    author = {Harmon, Katherine J. and Letchworth-Weaver, Kendra and Gaiduk, Alex P. and Giberti, Federico and Gygi, Francois and Chan, Maria K. Y. and Fenter, Paul and Galli, Giulia},
    date = {2020-11-16},
    doi = {10.1103/PhysRevMaterials.4.113805},
    issn = {2475-9953},
    journaltitle = {Physical Review Materials},
    langid = {english},
    location = {{College Pk}},
    number = {11},
    pages = {113805},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. Mater.},
    title = {Validating First-Principles Molecular Dynamics Calculations of Oxide/Water Interfaces with x-Ray Reflectivity Data},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {4}
}

@article{hernandezFastAccurateTransferable2019a,
    abstract = {The length and time scales of atomistic simulations are limited by the computational cost of the methods used to predict material properties. In recent years there has been great progress in the use of machine-learning algorithms to develop fast and accurate interatomic potential models, but it remains a challenge to develop models that generalize well and are fast enough to be used at extreme time and length scales. To address this challenge, we have developed a machine-learning algorithm based on symbolic regression in the form of genetic programming that is capable of discovering accurate, computationally efficient many-body potential models. The key to our approach is to explore a hypothesis space of models based on fundamental physical principles and select models within this hypothesis space based on their accuracy, speed, and simplicity. The focus on simplicity reduces the risk of overfitting the training data and increases the chances of discovering a model that generalizes well. Our algorithm was validated by rediscovering an exact Lennard-Jones potential and a Sutton-Chen embedded-atom method potential from training data generated using these models. By using training data generated from density functional theory calculations, we found potential models for elemental copper that are simple, as fast as embedded-atom models, and capable of accurately predicting properties outside of their training set. Our approach requires relatively small sets of training data, making it possible to generate training data using highly accurate methods at a reasonable computational cost. We present our approach, the forms of the discovered models, and assessments of their transferability, accuracy and speed.},
    annotation = {WOS:000496957400002},
    author = {Hernandez, Alberto and Balasubramanian, Adarsh and Yuan, Fenglin and Mason, Simon A. M. and Mueller, Tim},
    date = {2019-11-18},
    doi = {10.1038/s41524-019-0249-1},
    journaltitle = {Npj Computational Materials},
    langid = {english},
    location = {{London}},
    pages = {112},
    publisher = {{Nature Publishing Group}},
    shortjournal = {npj Comput. Mater.},
    title = {Fast, Accurate, and Transferable Many-Body Interatomic Potentials by Symbolic Regression},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {5}
}

@article{herrCompressingPhysicsAutoencoder2019a,
    abstract = {We define a vector quantity which corresponds to atomic species identity by compressing a set of physical properties with an autoencoder. This vector, referred to here as the elemental modes, provides many advantages in downstream machine learning tasks. Using the elemental modes directly as the feature vector, we trained a neural network to predict formation energies of elpasolites with improved accuracy over previous works on the same task. Combining the elemental modes with geometric features used in high-dimensional neural network potentials (HD-NNPs) solves many problems of scaling and efficiency in the development of such neural network potentials. Whereas similar models in the past have been limited to typically four atomic species (H, C, N, and O), our implementation does not scale in cost by adding more atomic species and allows us to train an HD-NNP model which treats molecules containing H, C, N, O, F, P, S, Cl, Se, Br, and I. Finally, we establish that our implementation allows us to define feature vectors for alchemical intermediate states in the HD-NNP model, which opens up new possibilities for performing alchemical free energy calculations on systems where bond breaking/forming is important. Published under license by AIP Publishing.},
    annotation = {WOS:000483889300007},
    author = {Herr, John E. and Koh, Kevin and Yao, Kun and Parkhill, John},
    date = {2019-08-28},
    doi = {10.1063/1.5108803},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {8},
    pages = {084103},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    shorttitle = {Compressing Physics with an Autoencoder},
    title = {Compressing Physics with an Autoencoder: {{Creating}} an Atomic Species Representation to Improve Machine Learning Models in the Chemical Sciences},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {151}
}

@article{houDielectricConstantSupercritical2020,
    abstract = {A huge amount of water at supercritical conditions exists in Earth's interior, where its dielectric properties play a critical role in determining how it stores and transports materials. However, it is very challenging to obtain the static dielectric constant of water, E-0, in a wide pressure-temperature (P-T) range as found in deep Earth either experimentally or by first-principles simulations. Here, we introduce a neural network dipole model, which, combined with molecular dynamics, can be used to compute P-T dependent dielectric properties of water as accurately as first-principles methods but much more efficiently. We found that E-0 may vary by one order of magnitude in Earth's upper mantle, suggesting that the solvation properties of water change dramatically at different depths. Although E-0 and the molecular dipole moment increase with an increase in pressure along an isotherm, the dipolar angular correlation has its maximum at 5 GPa-7 GPa, which may indicate that hydrogen bonds become weaker at high pressure. We also calculated the frequency-dependent dielectric constant of water in the microwave range, which, to the best of our knowledge, has not been calculated from first principles, and found that temperature affects the dielectric absorption more than pressure. Our results are of great use in many areas, e.g., modeling water-rock interactions in geochemistry. The computational approach introduced here can be readily applied to other molecular fluids.},
    annotation = {WOS:000571925300001},
    author = {Hou, Rui and Quan, Yuhui and Pan, Ding},
    date = {2020-09-14},
    doi = {10.1063/5.0020811},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {10},
    pages = {101103},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Dielectric Constant of Supercritical Water in a Large Pressure-Temperature Range},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {153}
}

@article{huangLearningThermodynamicallyStable2021,
    abstract = {In this work, we develop a method for learning interpretable, thermodynamically stable and Galilean invariant partial differential equations (PDEs) based on the conservation-dissipation formalism of irreversible thermodynamics. As governing equations for non-equilibrium flows in one dimension, the learned PDEs are parameterized by fully connected neural networks and satisfy the conservation-dissipation principle automatically. In particular, they are hyperbolic balance laws and Galilean invariant. The training data are generated from a kinetic model with smooth initial data. Numerical results indicate that the learned PDEs can achieve good accuracy in a wide range of Knudsen numbers. Remarkably, the learned dynamics can give satisfactory results with randomly sampled discontinuous initial data and Sod's shock tube problem although it is trained only with smooth initial data.},
    author = {Huang, Juntao and Ma, Zhiting and Zhou, Yizhou and Yong, Wen An},
    date = {2021},
    doi = {10.1515/JNET-2021-0008/HTML},
    journaltitle = {Journal of Non-Equilibrium Thermodynamics},
    publisher = {{De Gruyter Open Ltd}},
    title = {Learning Thermodynamically Stable and Galilean Invariant Partial Differential Equations for Non-Equilibrium Flows},
    url = {https://www.degruyter.com/document/doi/10.1515/jnet-2021-0008/html}
}

@article{huInclusionMachineLearning2018,
    abstract = {We discuss a theoretical approach that employs machine learning potential energy surfaces (ML-PESs) in the nonadiabatic dynamics simulation of polyatomic systems by taking 6-aminopyrimidine as a typical example. The Zhu-Nakamura theory is employed in the surface hopping dynamics, which does not require the calculation of the nonadiabatic coupling vectors. The kernel ridge regression is used in the construction of the adiabatic PESs. In the nonadiabatic dynamics simulation, we use ML-PESs for most geometries and switch back to the electronic structure calculations for a few geometries either near the S-1/S-0 conical intersections or in the out-of-confidence regions. The dynamics results based on ML-PESs are consistent with those based on CASSCF PESs. The ML-PESs are further used to achieve the highly efficient massive dynamics simulations with a large number of trajectories. This work displays the powerful role of ML methods in the nonadiabatic dynamics simulation of polyatomic systems.},
    annotation = {WOS:000435026100002},
    author = {Hu, Deping and Xie, Yu and Li, Xusong and Li, Lingyue and Lan, Zhenggang},
    date = {2018-06-07},
    doi = {10.1021/acs.jpclett.8b00684},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {11},
    pages = {2725--2732},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    title = {Inclusion of {{Machine Learning Kernel Ridge Regression Potential Energy Surfaces}} in {{On}}-the-{{Fly Nonadiabatic Molecular Dynamics Simulation}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/6},
    urldate = {2021-08-06},
    volume = {9}
}

@article{huNeuralNetworkForce2020a,
    abstract = {A method using machine learning (ML) is proposed to describe metal growth for simulations, which retains the accuracy of ab initio density functional theory (DFT) and results in a thousands-fold reduction in the computational time. This method is based on atomic energy decomposition from DFT calculations. Compared with other ML methods, our energy decomposition approach can yield much more information with the same DFT calculations. This approach is employed for the amorphous sodium system, where only 1000 DFT molecular dynamics images are enough for training an accurate model. The DFT and neural network potential (NNP) are compared for the dynamics to show that similar structural properties are generated. Finally, metal growth experiments from liquid to solid in a small and larger system are carried out to demonstrate the ability of using NNP to simulate the real growth process.},
    annotation = {WOS:000515424300026},
    author = {Hu, Qin and Weng, Mouyi and Chen, Xin and Li, Shucheng and Pan, Feng and Wang, Lin-Wang},
    date = {2020-02-20},
    doi = {10.1021/acs.jpclett.9b03780},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {4},
    pages = {364--1369},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    title = {Neural {{Network Force Fields}} for {{Metal Growth Based}} on {{Energy Decompositions}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {11}
}

@article{huPerspectiveMultiscaleSimulation2021a,
    abstract = {Phonon-mediated thermal transport is inherently multi-scale. The wave-length of phonons (considering phonons as waves) is typically at the nanometer scale; the typical size of a phonon wave energy packet is tens of nanometers, while the phonon mean free path (MFP) can be as long as microns. At different length scales, the phonons will interact with structures of different feature sizes, which can be as small as 0D defects (point defects), short to medium range linear defects (dislocations), medium to large range 2D planar defects (stacking faults and twin boundaries), and large scale 3D defects (voids, inclusions, and various microstructures). The nature of multi-scale thermal transport is that there are different heat transfer physics across different length scales and in the meantime the physics crossing the different scales is interdependent and coupled. Since phonon behavior is usually mode dependent, thermal transport in materials with a combined micro-/nano-structure complexity becomes complicated, making modeling this kind of transport process very challenging. In this perspective, we first summarize the advantages and disadvantages of computational methods for mono-scale heat transfer and the state-of-the-art multi-scale thermal transport modeling. We then discuss a few important aspects of the future development of multi-scale modeling, in particular with the aid of modern machine learning and uncertainty quantification techniques. As more sophisticated theoretical and computational methods continue to advance thermal transport predictions, novel heat transfer physics and thermally functional materials will be discovered for the pertaining energy systems and technologies.},
    annotation = {WOS:000612961700001},
    author = {Hu, Ming and Yang, Zhonghua},
    date = {2021-01-21},
    doi = {10.1039/d0cp03372c},
    issn = {1463-9076},
    journaltitle = {Physical Chemistry Chemical Physics},
    langid = {english},
    location = {{Cambridge}},
    number = {3},
    pages = {1785--1801},
    publisher = {{Royal Soc Chemistry}},
    shortjournal = {Phys. Chem. Chem. Phys.},
    title = {Perspective on Multi-Scale Simulation of Thermal Transport in Solids and Interfaces},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {23}
}

@article{husicCoarseGrainingMolecular2020,
    abstract = {Coarse graining enables the investigation of molecular dynamics for larger systems and at longer timescales than is possible at an atomic resolution. However, a coarse graining model must be formulated such that the conclusions we draw from it are consistent with the conclusions we would draw from a model at a finer level of detail. It has been proved that a force matching scheme defines a thermodynamically consistent coarse-grained model for an atomistic system in the variational limit. Wang et al. [ACS Cent. Sci. 5, 755 (2019)] demonstrated that the existence of such a variational limit enables the use of a supervised machine learning framework to generate a coarse-grained force field, which can then be used for simulation in the coarse-grained space. Their framework, however, requires the manual input of molecular features to machine learn the force field. In the present contribution, we build upon the advance of Wang et al. and introduce a hybrid architecture for the machine learning of coarse-grained force fields that learn their own features via a subnetwork that leverages continuous filter convolutions on a graph neural network architecture. We demonstrate that this framework succeeds at reproducing the thermodynamics for small biomolecular systems. Since the learned molecular representations are inherently transferable, the architecture presented here sets the stage for the development of machine-learned, coarse-grained force fields that are transferable across molecular systems.},
    annotation = {WOS:000595274800001},
    author = {Husic, Brooke E. and Charron, Nicholas E. and Lemm, Dominik and Wang, Jiang and Perez, Adria and Majewski, Maciej and Kramer, Andreas and Chen, Yaoyi and Olsson, Simon and de Fabritiis, Gianni and Noe, Frank and Clementi, Cecilia},
    date = {2020-11-21},
    doi = {10.1063/5.0026133},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {19},
    options = {useprefix=true},
    pages = {194101},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Coarse Graining Molecular Dynamics with Graph Neural Networks},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {153}
}

@article{jacksonEfficientMultiscaleOptoelectronic2020,
    abstract = {Conjugated polymers represent a high-potential material class due to the tunability of their optoelectronic properties via straightforward processing protocols. However, to in silico tailor these optoelectronic properties, one must compute the conformationally dependent electronic structure over mesoscopic length and time scales. This task represents a challenging multiscale computational problem in which quantum chemistry and atomistic or coarse-grained molecular dynamics must be integrated. Recently, we introduced a methodology, called electronic coarse graining (ECG), that utilizes artificial neural networks to compute ab initio quality electronic structures using only the reduced degrees of freedom of coarse-grained models. Here, we adapt ECG to variable-molecular weight conjugated oligomers by casting the problem in the framework of sequence detection. A machine-learning (ML) architecture employing one-dimensional convolutional neural networks and long short-term memory networks is utilized to compute ground-state orbital energies, charge density distributions, and optical spectra solely from the coarse-grained model's configurational degrees of freedom. Robust molecular weight transferability for ECG is established via a A-ML approach that leverages model electronic Hamiltonians for ground and excited-state property determination. The accuracy and transferability of the ECG methodology opens the door for scalable optoelectronic property prediction in conjugated polymers directly from coarse-grained degrees of freedom.},
    annotation = {WOS:000507721500051},
    author = {Jackson, Nicholas E. and Bowen, Alec S. and de Pablo, Juan J.},
    date = {2020-01-14},
    doi = {10.1021/acs.macromol.9b02020},
    issn = {0024-9297},
    journaltitle = {Macromolecules},
    langid = {english},
    location = {{Washington}},
    number = {1},
    options = {useprefix=true},
    pages = {482--490},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {Macromolecules},
    title = {Efficient {{Multiscale Optoelectronic Prediction}} for {{Conjugated Polymers}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {53}
}

@article{jacksonElectronicStructureCoarsegrained2019,
    abstract = {Computational studies aimed at understanding conformationally dependent electronic structure in soft materials require a combination of classical and quantum-mechanical simulations, for which the sampling of conformational space can be particularly demanding. Coarse-grained (CG) models provide a means of accessing relevant time scales, but CG configurations must be back-mapped into atomistic representations to perform quantum-chemical calculations, which is computationally intensive and inconsistent with the spatial resolution of the CG models. A machine learning approach, denoted as artificial neural network electronic coarse graining (ANN-ECG), is presented here in which the conformationally dependent electronic structure of a molecule is mapped directly to CG pseudo-atom configurations. By averaging over decimated degrees of freedom, ANN-ECG accelerates simulations by eliminating backmapping and repeated quantum-chemical calculations. The approach is accurate, consistent with the CG spatial resolution, and can be used to identify computationally optimal CG resolutions.},
    annotation = {WOS:000462564300049},
    author = {Jackson, Nicholas E. and Bowen, Alec S. and Antony, Lucas W. and Webb, Michael A. and Vishwanath, Venkatram and de Pablo, Juan J.},
    date = {2019-03},
    doi = {10.1126/sciadv.aav1190},
    issn = {2375-2548},
    journaltitle = {Science Advances},
    langid = {english},
    location = {{Washington}},
    number = {3},
    options = {useprefix=true},
    pages = {eaav1190},
    publisher = {{Amer Assoc Advancement Science}},
    shortjournal = {Sci. Adv.},
    title = {Electronic Structure at Coarse-Grained Resolutions from Supervised Machine Learning},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {5}
}

@article{jacksonRecentAdvancesMachine2019,
    abstract = {The multiscale design of soft materials requires an ensemble of computational techniques spanning quantum-chemistry to molecular dynamics to continuum modeling. The recent emergence of machine-learning (ML) and modern optimization algorithms has accelerated material property prediction, as well as stimulated the development of hybrid ML/molecular modeling methodologies capable of providing physical insights unobtainable from purely physics-based modeling and intuition. Such hybrid techniques also have important ramifications for the ML-enhanced interpretation of results from simulations and experiments alike. Leveraging ML techniques for the design of chemical or morphological structures based on a target property or functionality represents an exciting goal for the general area of soft materials, including polymers, liquid crystals, colloids, or biomolecules, to name a few representative classes of systems. Here, we provide a perspective on recent work using ML techniques of relevance for the multiscale design of soft materials and outline potential future directions of interest to the soft materials community.},
    annotation = {WOS:000472549500014},
    author = {Jackson, Nicholas E. and Webb, Michael A. and de Pablo, Juan J.},
    date = {2019-03},
    doi = {10.1016/j.coche.2019.03.005},
    issn = {2211-3398},
    journaltitle = {Current Opinion in Chemical Engineering},
    langid = {english},
    location = {{Oxford}},
    options = {useprefix=true},
    pages = {106--114},
    publisher = {{Elsevier Sci Ltd}},
    shortjournal = {Curr. Opin. Chem. Eng.},
    title = {Recent Advances in Machine Learning towards Multiscale Soft Materials Design},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {23}
}

@article{janetQuantitativeUncertaintyMetric2019a,
    abstract = {Machine learning (ML) models, such as artificial neural networks, have emerged as a complement to high-throughput screening, enabling characterization of new compounds in seconds instead of hours. The promise of ML models to enable large-scale chemical space exploration can only be realized if it is straightforward to identify when molecules and materials are outside the model's domain of applicability. Established uncertainty metrics for neural network models are either costly to obtain (e.g., ensemble models) or rely on feature engineering (e.g., feature space distances), and each has limitations in estimating prediction errors for chemical space exploration. We introduce the distance to available data in the latent space of a neural network ML model as a low-cost, quantitative uncertainty metric that works for both inorganic and organic chemistry. The calibrated performance of this approach exceeds widely used uncertainty metrics and is readily applied to models of increasing complexity at no additional cost. Tightening latent distance cutoffs systematically drives down predicted model errors below training errors, thus enabling predictive error control in chemical discovery or identification of useful data points for active learning.},
    annotation = {WOS:000482978700005},
    author = {Janet, Jon Paul and Duan, Chenru and Yang, Tzuhsiung and Nandy, Aditya and Kulik, Heather J.},
    date = {2019-09-14},
    doi = {10.1039/c9sc02298h},
    issn = {2041-6520},
    journaltitle = {Chemical Science},
    langid = {english},
    location = {{Cambridge}},
    number = {34},
    pages = {7913--7922},
    publisher = {{Royal Soc Chemistry}},
    shortjournal = {Chem. Sci.},
    title = {A Quantitative Uncertainty Metric Controls Error in Neural Network-Driven Chemical Discovery},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {10}
}

@article{Jiang,
    abstract = {Atmospheric aerosol nucleation contributes to around half of cloud condensation nuclei (CCN) globally and the nucleated particles can grow larger to impact air quality and consequently human health. Despite the decades' efforts, the detailed nucleation mechanism is still poorly understood. The ultimate goal of theoretical understanding aerosol nucleation is to simulate nucleation in ambient condition. However, there is lack of accurate reactive force field. Here for the first time, we propose the reactive force field with good size scalability for nucleation systems based on deep neural network and further bridge the simulation in the limited box with cluster kinetics towards boosting the aerosol simulation to be fully ab initio. We found that the formation rates based on hard sphere collision rate constants tend to be underestimated. Besides, the framework here is transferable to other nucleation systems, potentially revolutionizing the atmospheric aerosol nucleation field. Theoretical understanding of nucleation mechanism largely relies on classical nucleation theory (CNT) 1 which give a general mind map for nucleation thermodynamics and kinetics 2 even though the capillary assumption is extensively criticized 3. The emergence and then broadly employed theoretical model in the past decade, Atmospheric Cluster Dynamics Code (ACDC) 4 , surmounts the drawbacks of CNT through coupled quantum chemical thermodynamics 5 with the birth-death equations 2. Within the model, collision rate constants and evaporation rates are the two most critical parameters, determining the accuracy to predict macro parameters like cluster concentrations and formation rates which can be directly compared with experiments 6. Evaporation rates, derived from detailed balance and ab initio thermodynamics can be very accurate with sophisticated theoretical calculations 7. However, collision rate constants, derived from simple hard sphere model, is still very rough, far from the ab initio based evaporation rate. Here we propose a general framework to potentially boost the aerosol nucleation simulation toward fully ab initio. In the framework, deep neural network based force field (DNN-FF) is trained that can perform robust nucleation molecular dynamics (MD) simulations to derive the collision rate constants. Then static quantum chemical thermodynamics based evaporation rates are coupled into cluster dynamics model to provide ab initio kinetics for atmospheric aerosol nucleation.},
    author = {Jiang, S and Liu, YR and Huang, T and Feng, YJ and Wang, CY},
    journaltitle = {arxiv.org},
    title = {Towards Fully Ab Initio Simulation of Atmospheric Aerosol Nucleation},
    url = {https://arxiv.org/abs/2107.04802}
}

@article{jiangAccurateDeepPotential2021a,
    abstract = {Combining first-principles accuracy and empirical-potential efficiency for the description of the potential energy surface (PES) is the philosopher's stone for unraveling the nature of matter via atomistic simulation. This has been particularly challenging for multi-component alloy systems due to the complex and non-linear nature of the associated PES. In this work, we develop an accurate PES model for the Al-Cu-Mg system by employing deep potential (DP), a neural network based representation of the PES, and DP generator (DP-GEN), a concurrent-learning scheme that generates a compact set of ab initio data for training. The resulting DP model gives predictions consistent with first-principles calculations for various binary and ternary systems on their fundamental energetic and mechanical properties, including formation energy, equilibrium volume, equation of state, interstitial energy, vacancy and surface formation energy, as well as elastic moduli. Extensive benchmark shows that the DP model is ready and will be useful for atomistic modeling of the Al-Cu-Mg system within the full range of concentration.},
    annotation = {WOS:000655293300001},
    author = {Jiang, Wanrun and Zhang, Yuzhi and Zhang, Linfeng and Wang, Han},
    date = {2021-05},
    doi = {10.1088/1674-1056/abf134},
    issn = {1674-1056},
    journaltitle = {Chinese Physics B},
    langid = {english},
    location = {{Bristol}},
    number = {5},
    pages = {050706},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {Chin. Phys. B},
    title = {Accurate {{Deep Potential}} Model for the {{Al}}-{{Cu}}-{{Mg}} Alloy in the Full Concentration Space},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {30}
}

@article{kangLargeScaleAtomicSimulation2020a,
    abstract = {Atomic simulations based on quantum mechanics (QM) calculations have entered into the tool box of chemists over the past few decades, facilitating an understanding of a wide range of chemistry problems, from structure characterization to reactivity determination. Due to the poor scaling and high computational cost intrinsic to QM calculations, one has to either sacrifice accuracy or time when performing large-scale atomic simulations. The battle to find a better compromise between accuracy and speed has been central to the development of new theoretical methods. The recent advances of machine-learning (ML)-based large-scale atomic simulations has shown great promise to the benefit of many branches of chemistry. Instead of solving the Schrodinger equation directly, ML-based simulations rely on a large data set of accurate potential energy surfaces (PESs) and complex numerical models to predict the total energy. These simulations feature both a high speed and a high accuracy for computing large systems. Due to the lack of a physical foundation in numerical models, ML models are often frustrated in their predictivity and robustness, which are key to applications. Focusing on these concerns, here we overview the recent advances in ML methodologies for atomic simulations on three key aspects. Namely, the generation of a representative data set, the extensity of ML models, and the continuity of data representation. While global optimization methods are the natural choice for building a representative data set, the stochastic surface walking method is shown to provide the desired PES sampling for both minima and transition regions on the PES. The current ML models generally utilize local geometrical descriptors as an input and consider the total energy as the sum of atomic energies. There are many flavors of data descriptors and ML models, but the applications for material and reaction predictions are still limited, not least because of the difficulty to train the associated vast global data sets. We show that our recently designed power-type structure descriptors together with a feed-forward neural network (NN) model are compatible with highly complex global PES data, which has led to a large family of global NN (G-NN) potentials. Two recent applications of G-NN potentials in material and reaction simulations are selected to illustrate how ML-based atomic simulations can help the discovery of new materials and reactions.},
    annotation = {WOS:000584415900009},
    author = {Kang, Pei-Lin and Shang, Cheng and Liu, Zhi-Pan},
    date = {2020-10-20},
    doi = {10.1021/acs.accounts.0c00472},
    issn = {0001-4842},
    journaltitle = {Accounts of Chemical Research},
    langid = {english},
    location = {{Washington}},
    number = {10},
    pages = {2119--2129},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {Accounts Chem. Res.},
    title = {Large-{{Scale Atomic Simulation}} via {{Machine Learning Potentials Constructed}} by {{Global Potential Energy Surface Exploration}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {53}
}

@article{koEnablingLargeScaleCondensedPhase2020a,
    abstract = {By including a fraction of exact exchange (EXX), hybrid functionals reduce the self-interaction error in semilocal density functional theory (DFT) and thereby furnish a more accurate and reliable description of the underlying electronic structure in systems throughout biology, chemistry, physics, and materials science. However, the high computational cost associated with the evaluation of all required EXX quantities has limited the applicability of hybrid DFT in the treatment of large molecules and complex condensed-phase materials. To overcome this limitation, we describe a linear-scaling approach that utilizes a local representation of the occupied orbitals (e.g., maximally localized Wannier functions (MLWFs)) to exploit the sparsity in the real-space evaluation of the quantum mechanical exchange interaction in finite-gap systems. In this work, we present a detailed description of the theoretical and algorithmic advances required to perform MLWF-based ab initio molecular dynamics (AIMD) simulations of large-scale condensed-phase systems of interest at the hybrid DFT level. We focus our theoretical discussion on the integration of this approach into the framework of Car-Parrinello AIMD, and highlight the central role played by the MLWF-product potential (i.e., the solution of Poisson's equation for each corresponding MLWF-product density) in the evaluation of the EXX energy and wave function forces. We then provide a comprehensive description of the exx algorithm implemented in the open-source Quantum ESPRESSO program, which employs a hybrid MPI/OpenMP parallelization scheme to efficiently utilize the high-performance computing (HPC) resources available on current- and next-generation supercomputer architectures. This is followed by a critical assessment of the accuracy and parallel performance (e.g., strong and weak scaling) of this approach when AIMD simulations of liquid water are performed in the canonical (NVT) ensemble. With access to HPC resources, we demonstrate that exx enables hybrid DFT-based AIMD simulations of condensed-phase systems containing 500-1000 atoms (e.g., (H2O)(256)) with a wall time cost that is comparable to that of semilocal DFT. In doing so, exx takes us one step closer to routinely performing AIMD simulations of complex and large-scale condensed-phase systems for sufficiently long time scales at the hybrid DFT level of theory.},
    annotation = {WOS:000541503600027},
    author = {Ko, Hsin-Yu and Jia, Junteng and Santra, Biswajit and Wu, Xifan and Car, Roberto and DiStasio, Robert},
    date = {2020-06-09},
    doi = {10.1021/acs.jctc.9b01167},
    issn = {1549-9618},
    journaltitle = {Journal of Chemical Theory and Computation},
    langid = {english},
    location = {{Washington}},
    number = {6},
    pages = {3757--3785},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem. Theory Comput.},
    title = {Enabling {{Large}}-{{Scale Condensed}}-{{Phase Hybrid Density Functional Theory Based Ab Initio Molecular Dynamics}}. 1. {{Theory}}, {{Algorithm}}, and {{Performance}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {16}
}

@article{krishnamoorthyDielectricConstantLiquid2021a,
    abstract = {The static dielectric constant epsilon(0) and its temperature dependence for liquid water is investigated using neural network quantum molecular dynamics (NNQMD). We compute the exact dielectric constant in canonical ensemble from NNQMD trajectories using fluctuations in macroscopic polarization computed from maximally localized Wannier functions (MLWF). Two deep neural networks are constructed. The first, NNQMD, is trained on QMD configurations for liquid water under a variety of temperature and density conditions to learn potential energy surface and forces and then perform molecular dynamics simulations. The second network, NNMLWF, is trained to predict locations of MLWF of individual molecules using the atomic configurations from NNQMD. Training data for both the neural networks is produced using a highly accurate quantum-mechanical method, DFT-SCAN that yields an excellent description of liquid water. We produce 280 x 10(6) configurations of water at 7 temperatures using NNQMD and predict MLWF centers using NNMLWF to compute the polarization fluctuations. The length of trajectories needed for a converged value of the dielectric constant at 0 degrees C is found to be 20 ns (40 x 10(6) configurations with 0.5 fs time step). The computed dielectric constants for 0, 15, 30, 45, 60, 75, and 90 degrees C are in good agreement with experiments. Our scalable scheme to compute dielectric constants with quantum accuracy is also applicable to other polar molecular liquids.},
    annotation = {WOS:000655929800008},
    author = {Krishnamoorthy, Aravind and Nomura, Ken-ichi and Baradwaj, Nitish and Shimamura, Kohei and Rajak, Pankaj and Mishra, Ankit and Fukushima, Shogo and Shimojo, Fuyuki and Kalia, Rajiv and Nakano, Aiichiro and Vashishta, Priya},
    date = {2021-05-25},
    doi = {10.1103/PhysRevLett.126.216403},
    issn = {0031-9007},
    journaltitle = {Physical Review Letters},
    langid = {english},
    location = {{College Pk}},
    number = {21},
    pages = {216403},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. Lett.},
    title = {Dielectric {{Constant}} of {{Liquid Water Determined}} with {{Neural Network Quantum Molecular Dynamics}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {126}
}

@article{kwonEstimationSecondVirial2021a,
    abstract = {We test the accuracy of the neural network interaction potentials against accurate thermodynamic data of He and N2. We perform single point energy calculations at CCSD(T) level, construct the intermolecular interactions by using neural networks, and perform Monte Carlo simulations to calculate virial coefficients. We find that the deviation of virial coefficients from previous studies is small for He while the difference is about 3 cm3mol- 1 for N2. Our results show that when trying to obtain accurate thermodynamic data from the neural network interaction potentials, not only the level of quantum chemical calculations should be an important factor, but also at least thousands of training data points would be required.},
    annotation = {WOS:000662652500011},
    author = {Kwon, Taejin and Song, Han Wook and Woo, Sam Yong and Kim, Jong-Ho and Sung, Bong June},
    date = {2021-08-01},
    doi = {10.1016/j.chemphys.2021.111231},
    issn = {0301-0104},
    journaltitle = {Chemical Physics},
    langid = {english},
    location = {{Amsterdam}},
    pages = {111231},
    publisher = {{Elsevier}},
    shortjournal = {Chem. Phys.},
    title = {The Estimation of the Second Virial Coefficients of {{He}} and {{N2}} Based on Neural Network Potentials with Quantum Mechanical Calculations},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {548}
}

@article{leModelingElectrifiedMetal2021,
    abstract = {The structure and potential distribution of electric double layers (EDLs) are of close relevance to the performance of electrode materials. In the past years, despite tremendous efforts devoted to this topic, an atomistic picture of the EDL is still lacking, let alone understanding on how the EDL structure is related to the dielectric property of interface water. In this article, we briefly review the recent progress in modeling electrified metal/water interfaces using ab initio molecular dynamics (AIMD). The ab initio methods for EDL modeling is firstly summarized, and then we discuss the structures of interface water on metal electrodes at different potential conditions. Moreover, we illustrate the potential-dependent behavior of chemisorbed water on Pt(111) surface and its relationship with the peak of the differential Helmholtz capacitance observed by experiment. At last, we give some perspective for future development in ab initio modeling of electrochemical interfaces.},
    annotation = {WOS:000662265900007},
    author = {Le, Jia-Bo and Cheng, Jun},
    date = {2021-06},
    doi = {10.1016/j.coelec.2021.100693},
    issn = {2451-9103},
    journaltitle = {Current Opinion in Electrochemistry},
    langid = {english},
    location = {{Amsterdam}},
    pages = {100693},
    publisher = {{Elsevier}},
    shortjournal = {Curr. Opin. Electrochem.},
    shorttitle = {Modeling Electrified Metal/Water Interfaces from Ab Initio Molecular Dynamics},
    title = {Modeling Electrified Metal/Water Interfaces from Ab Initio Molecular Dynamics: {{Structure}} and {{Helmholtz}} Capacitance},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {27}
}

@article{leModelingElectrochemicalInterfaces2020a,
    abstract = {The origin of the potential difference between the potential of zero charge of a metal/water interface and the work function of the metal is a recurring issue because it is related to how water interacts with metal surface in the absence of surface charge. Recently ab initio molecular dynamics method has been used to model electrochemical interfaces to study interfacial potential and the structure of interface water. Here, we will first introduce the computational standard hydrogen electrode method, which allows for ab initio determination of electrode potentials that can be directly compared with experiment. Then, we will review the recent progress from ab initio molecular dynamics simulation in understanding the interaction between water and metal and its impact on interfacial potential. Finally, we will give our perspective for future development of ab initio computational electrochemistry.},
    annotation = {WOS:000519095200020},
    author = {Le, Jia-Bo and Cheng, Jun},
    date = {2020-02},
    doi = {10.1016/j.coelec.2019.11.008},
    issn = {2451-9103},
    journaltitle = {Current Opinion in Electrochemistry},
    langid = {english},
    location = {{Amsterdam}},
    pages = {129--136},
    publisher = {{Elsevier}},
    shortjournal = {Curr. Opin. Electrochem.},
    shorttitle = {Modeling Electrochemical Interfaces from Ab Initio Molecular Dynamics},
    title = {Modeling Electrochemical Interfaces from Ab Initio Molecular Dynamics: Water Adsorption on Metal Surfaces at Potential of Zero Charge},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {19}
}

@article{leoniNonclassicalNucleationPathways2021a,
    abstract = {The nucleation of crystals from liquid melt is often characterized by a competition between different crystalline structures or polymorphs and can result in nuclei with heterogeneous compositions. These mixed-phase nuclei can display nontrivial spatial arrangements, such as layered and onionlike structures, whose composition varies according to the radial distance, and which so far have been explained on the basis of bulk and surface free-energy differences between the competing phases. Here we extend the generality of these nonclassical nucleation processes, showing that layered and onionlike structures can emerge solely based on structural fluctuations even in the absence of free-energy differences. We consider two examples of competing crystalline structures, hcp and fcc forming in hard spheres relevant for repulsive colloids and dense liquids, and the cubic and hexagonal diamond forming in water relevant also for other group 14 elements such as carbon and silicon. We introduce a novel structural order parameter that combined with a neural-network classification scheme allows us to study the properties of the growing nucleus from the early stages of nucleation. We find that small nuclei have distinct size fluctuations and compositions from the nuclei that emerge from the growth stage. The transition between these two regimes is characterized by the formation of onionlike structures, in which the composition changes with the distance from the center of the nucleus, similar to what is seen in the two-step nucleation process.},
    annotation = {WOS:000671589300001},
    author = {Leoni, Fabio and Russo, John},
    date = {2021-07-09},
    doi = {10.1103/PhysRevX.11.031006},
    issn = {2160-3308},
    journaltitle = {Physical Review X},
    langid = {english},
    location = {{College Pk}},
    number = {3},
    pages = {031006},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. X},
    title = {Nonclassical {{Nucleation Pathways}} in {{Stacking}}-{{Disordered Crystals}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {11}
}

@article{liAnalysisTrajectorySimilarity2018,
    abstract = {We propose an "automatic" approach to analyze the results of the on-the-fly trajectory surface hopping simulation on the multi-channel nonadiabatic photoisomerization dynamics by considering the trajectory similarity and the configuration similarity. We choose a representative system phytochromobilin (P Phi B) chromophore model to illustrate the analysis protocol. After a large number of trajectories are obtained, it is possible to define the similarity of different trajectories by the Frechet distance and to employ the trajectory clustering analysis to divide all trajectories into several clusters. Each cluster in principle represents a photoinduced isomerization reaction channel. This idea provides an effective approach to understand the branching ratio of the multi-channel photoisomerization dynamics. For each cluster, the dimensionality reduction is employed to understand the configuration similarity in the trajectory propagation, which provides the understanding of the major geometry evolution features in each reaction channel. The results show that this analysis protocol not only assigns all trajectories into different photoisomerization reaction channels but also extracts the major molecular motion without the requirement of the pre-known knowledge of the active photoisomerization site. As a side product of this analysis tool, it is also easy to find the so-called "typical" or "representative" trajectory for each reaction channel. Published by AIP Publishing.},
    annotation = {WOS:000454626000008},
    author = {Li, Xusong and Hu, Deping and Xie, Yu and Lan, Zhenggang},
    date = {2018-12-28},
    doi = {10.1063/1.5048049},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {24},
    pages = {244104},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Analysis of Trajectory Similarity and Configuration Similarity in On-the-Fly Surface-Hopping Simulation on Multi-Channel Nonadiabatic Photoisomerization Dynamics},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/6},
    urldate = {2021-08-06},
    volume = {149}
}

@article{liangMachineLearningDrivenSimulationsMicrostructure2021,
    abstract = {Theoretical studies on the MgCl2-KCl eutectic heavily rely on ab initio calculations based on density functional theory (DFT). However, neither large-scale nor long-time calculations are feasible in the framework of the ab initio method, which makes it challenging to accurately predict some properties. To address this issue, a scheme based on ab initio calculation, deep neural networks, and machine learning is introduced. By training on high-quality data sets generated by ab initio calculations, a deep potential (DP) is constructed to describe the interaction between atoms. This work shows that the DP enables higher efficiency and similar accuracy relative to DFT. By performing molecular dynamics simulations with DP, the microstructure and thermophysical properties of the MgCl2-KCl eutectic (32:68 mol \%) are investigated. The structural evolution with temperature is analyzed through partial radial distribution functions, coordination numbers, angular distribution functions, and structural factors. Meanwhile, the estimated thermophysical properties are discussed, including density, thermal expansion coefficient, shear viscosity, self-diffusion coefficient, and specific heat capacity. It reveals that the Mg2+ ions in this system have a distorted tetrahedral geometry rather than an octahedral one (with vacancies). The microstructure of the MgCl2-KCl eutectic shows the feature of medium-range order, and this feature will be enhanced at a higher temperature. All predicted thermophysical properties are in good agreement with the experimental results. The hydrodynamic radius determined from the shear viscosity and self-diffusion coefficient shows that the Me ions have a strong local structure and diffuse as if with an intact coordination shell. Overall, this work provides a thorough understanding of the microstructure and enriches the data of the thermophysical properties of the MgCl2-KCl eutectic.},
    annotation = {WOS:000614062400050},
    author = {Liang, Wenshuo and Lu, Guimin and Yu, Jianguo},
    date = {2021-01-27},
    doi = {10.1021/acsami.0c20665},
    issn = {1944-8244},
    journaltitle = {Acs Applied Materials \& Interfaces},
    langid = {english},
    location = {{Washington}},
    number = {3},
    pages = {4034--4042},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {ACS Appl. Mater. Interfaces},
    title = {Machine-{{Learning}}-{{Driven Simulations}} on {{Microstructure}} and {{Thermophysical Properties}} of {{MgCl2}}-{{KCl Eutectic}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {13}
}

@article{liangMolecularDynamicsSimulations2020,
    abstract = {In previous work, molten magnesium chloride has been investigated using first-principles molecular dynamics (FPMD) simulations based on density functional theory (DFT). However, such simulations are computationally intensive and therefore are restricted in terms of simulated size and time. In this work, a machine learning-based deep potential (DP) is trained to accelerate the molecular dynamics simulation of molten magnesium chloride. The trained DP can accurately describe the energies and forces with the prediction errors in energy and force being 1.76 x 10(-3) eV/atom and 4.76 x 10(-2) eV angstrom(-1), respectively. Applying the deep potential molecular dynamics (DPMD) approach, simulations can be performed with more than 1000 atoms, which is infeasible for FPMD simulations. Additionally, the partial radial distribution functions, angle distribution functions, densities, and self-diffusion coefficients predicted by DPMD simulations are also in reasonable agreement with FPMD or experimental results. This work shows that the DP enables higher efficiency and similar accuracy relative to DFT, exhibiting a bright application prospect in modeling molten salt systems.},
    annotation = {WOS:000586457200001},
    author = {Liang, Wenshuo and Lu, Guimin and Yu, Jianguo},
    date = {2020-12},
    doi = {10.1002/adts.202000180},
    journaltitle = {Advanced Theory and Simulations},
    langid = {english},
    location = {{Weinheim}},
    number = {12},
    pages = {2000180},
    publisher = {{Wiley-V C H Verlag Gmbh}},
    shortjournal = {Adv. Theory Simul.},
    title = {Molecular {{Dynamics Simulations}} of {{Molten Magnesium Chloride Using Machine}}-{{Learning}}-{{Based Deep Potential}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {3}
}

@article{liangTheoreticalPredictionLocal2021,
    abstract = {In this work, the local structure and transport properties of three typical alkali chlorides (LiCl, NaCl, and KCl) were investigated by our newly trained deep potentials (DPs). We extracted datasets from ab initio molecular dynamics (AIMD) calculations and used these to train and validate the DPs. Large-scale and long-time molecular dynamics simulations were performed over a wider range of temperatures than AIMD to confirm the reliability and generality of the DPs. We demonstrated that the generated DPs can serve as a powerful tool for simulating alkali chlorides; the DPs also provide results with accuracy that is comparable to that of AIMD and efficiency that is similar to that of empirical potentials. The partial radial distribution functions and angle distribution functions predicted using the DPs are in close agreement with those derived from AIMD. The estimated densities, self-diffusion coefficients, shear viscosities, and electrical conductivities also matched well with the AIMD and experimental data. This work provides confidence that DPs can be used to explore other systems, including mixtures of chlorides or entirely different salts. (C) 2021 Elsevier B.V. All rights reserved.},
    annotation = {WOS:000640021600009},
    author = {Liang, Wenshuo and Lu, Guimin and Yu, Jianguo},
    date = {2021-06-10},
    doi = {10.1016/j.jmst.2020.09.040},
    issn = {1005-0302},
    journaltitle = {Journal of Materials Science \& Technology},
    langid = {english},
    location = {{Shenyang}},
    pages = {78--85},
    publisher = {{Journal Mater Sci Technol}},
    shortjournal = {J. Mater. Sci. Technol.},
    title = {Theoretical Prediction on the Local Structure and Transport Properties of Molten Alkali Chlorides by Deep Potentials},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {75}
}

@article{liBetterApproximationsHigh2020,
    abstract = {Deep neural networks with rectified linear units (ReLU) are getting more and more popular due to their universal representation power and successful applications. Some theoretical progress regarding the approximation power of deep ReLU network for functions in Sobolev space and Korobov space have recently been made by [D. Yarotsky, Neural Network, 94:103-114, 2017] and [H. Montanelli and Q. Du, SIAM J Math. Data Sci., 1:78-92, 2019], etc. In this paper, we show that deep networks with rectified power units (RePU) can give better approximations for smooth functions than deep ReLU networks. Our analysis bases on classical polynomial approximation theory and some efficient algorithms proposed in this paper to convert polynomials into deep RePU networks of optimal size with no approximation error. Comparing to the results on ReLU networks, the sizes of RePU networks required to approximate functions in Sobolev space and Korobov space with an error tolerance epsilon, by our constructive proofs, are in general O(log1/epsilon) times smaller than the sizes of corresponding ReLU networks constructed in most of the existing literature. Comparing to the classical results of Mhaskar [Mhaskar, Adv. Comput. Math. 1:61-80, 1993], our constructions use less number of activation functions and numerically more stable, they can be served as good initials of deep RePU networks and further trained to break the limit of linear approximation theory. The functions represented by RePU networks are smooth functions, so they naturally fit in the places where derivatives are involved in the loss function.},
    annotation = {WOS:000501534800002},
    author = {Li, Bo and Tang, Shanshan and Yu, Haijun},
    date = {2020-02},
    doi = {10.4208/cicp.OA-2019-0168},
    issn = {1815-2406},
    journaltitle = {Communications in Computational Physics},
    langid = {english},
    location = {{Wanchai}},
    number = {2},
    pages = {379--411},
    publisher = {{Global Science Press}},
    shortjournal = {Commun. Comput. Phys.},
    title = {Better {{Approximations}} of {{High Dimensional Smooth Functions}} by {{Deep Neural Networks}} with {{Rectified Power Units}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {27}
}

@article{liDevelopmentRobustNeuralnetwork2021a,
    abstract = {Molten salts are a promising class of ionic liquids for clean energy applications, such as nuclear and solar energy. However, efficient and accurate evaluation of salt properties from a fundamental, microscopic perspective remains a challenge. Here, we apply artificial neural networks to atomistic modeling of molten NaCl to accurately reproduce the properties from ab initio quantum mechanical calculations based on density functional theory (DFT). The obtained neural network interatomic potential (NNIP) effectively captures the effects of both long-range and short-range interactions, which are crucial for modeling ionic liquids. Extensive validations suggest that the NNIP is capable of predicting the structural, thermophysical, and transport properties of molten NaCl as well as properties of crystalline NaCl, demonstrating near-DFT accuracy and 10(3) x higher efficiency in atomistic simulations. This application of NNIP suggests a paradigm shift from empirical/semiempirical/ab initio approaches to an efficient and accurate machine learning scheme in molten salt modeling.},
    annotation = {WOS:000658764300009},
    author = {Li, Qing-Jie and Kucukbenli, Emine and Lam, Stephen and Khaykovich, Boris and Kaxiras, Efthimios and Li, Ju},
    date = {2021-03-24},
    doi = {10.1016/j.xcrp.2021.100359},
    issn = {2666-3864},
    journaltitle = {Cell Reports Physical Science},
    langid = {english},
    location = {{Amsterdam}},
    number = {3},
    pages = {100359},
    publisher = {{Elsevier}},
    shortjournal = {Cell Rep. Phys. Sci.},
    title = {Development of Robust Neural-Network Interatomic Potential for Molten Salt},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {2}
}

@article{liIntroducingBlockDesign2021a,
    abstract = {The number of states required for describing a many-body quantum system increases exponentially with the number of particles; thus, it is time- and effort-consuming to exactly calculate molecular properties. Herein, we propose a deep learning algorithm named block-based graph neural network (BGNN) as an approximate solution. The algorithm can be understood as a representation learning process to extract useful interactions between a target atom and its neighboring atomic groups. Compared to other graph model variants, BGNN achieved the smallest mean absolute errors in most tasks on two large molecular datasets, QM9 and Alchemy. Our advanced machine learning method exhibits general applicability and can be readily employed for bioactivity prediction and other tasks relevant to drug discovery and materials design.},
    annotation = {WOS:000641316100001},
    author = {Li, Yuquan and Li, Pengyong and Yang, Xing and Hsieh, Chang-Yu and Zhang, Shengyu and Wang, Xiaorui and Lu, Ruiqiang and Liu, Huanxiang and Yao, Xiaojun},
    date = {2021-06-15},
    doi = {10.1016/j.cej.2021.128817},
    issn = {1385-8947},
    journaltitle = {Chemical Engineering Journal},
    langid = {english},
    location = {{Lausanne}},
    pages = {128817},
    publisher = {{Elsevier Science Sa}},
    shortjournal = {Chem. Eng. J.},
    title = {Introducing Block Design in Graph Neural Networks for Molecular Properties Prediction},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {414}
}

@article{liMultilevelFineTuningClosing2021a,
    abstract = {In scientific machine learning, regression networks have been recently applied to approximate solution maps (e.g., the potential-ground state map of the Schrodinger equation). In this paper, we aim to reduce the generalization error without spending more time on generating training samples. However, to reduce the generalization error, the regression network needs to be fit on a large number of training samples (e.g., a collection of potential-ground state pairs). The training samples can be produced by running numerical solvers, which takes significant time in many applications. In this paper, we aim to reduce the generalization error without spending more time on generating training samples. Inspired by few-shot learning techniques, we develop the multilevel fine-tuning algorithm by introducing levels of training: we first train the regression network on samples generated at the coarsest grid and then successively fine-tune the network on samples generated at finer grids. Within the same amount of time, numerical solvers generate more samples on coarse grids than on fine grids. We demonstrate a significant reduction of generalization error in numerical experiments on challenging problems with oscillations, discontinuities, or rough coefficients. Further analysis can be conducted in the neural tangent kernel regime, and we provide practical estimators to the generalization error. The number of training samples at different levels can be optimized for the smallest estimated generalization error under the constraint of budget for training data. The optimized distribution of budget over levels provides practical guidance with theoretical insight as in the celebrated multilevel Monte Carlo algorithm.},
    annotation = {WOS:000636051200013},
    author = {Li, Zhihan and Fan, Yuwei and Ying, Lexing},
    date = {2021},
    doi = {10.1137/20M1326404},
    issn = {1540-3459},
    journaltitle = {Multiscale Modeling \& Simulation},
    langid = {english},
    location = {{Philadelphia}},
    number = {1},
    pages = {344--373},
    publisher = {{Siam Publications}},
    shortjournal = {Multiscale Model. Simul.},
    shorttitle = {Multilevel {{Fine}}-{{Tuning}}},
    title = {Multilevel {{Fine}}-{{Tuning}}: {{Closing Generalization Gaps}} in {{Approximation}} of {{Solution Maps Under}} a {{Limited Budget}} for {{Training Data}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {19}
}

@article{linNeuralnetworkBasedFramework2021,
    abstract = {Interaction potentials are critical to molecular dynamics simulations on fundamental mechanisms at atomic scales. Combination of well-developed single-element empirical potentials via cross interaction (CI) is an important and effective way to develop alloy embedded-atom method (EAM) potentials. In this work, based on neural-network (NN) models, firstly we proposed a framework to construct CI potential functions via utilizing single-element potentials. The framework contained four steps: (1) extracting characteristic points from single-element potential functions, (2) constructing CI functions by cubic spline interpolation, (3) evaluating the accuracy of CI functions by referring to first-principle (FP) data, and (4) searching for reasonable CI functions via NN models. Then with this framework, we developed a Zr-Nb alloy CI potential utilizing the MA-III (pure Zr potential developed by Mendelev and Ackland in 2007) and the Fellinger, Park and Wilkins (FPW) (pure Nb potential developed by FPW in 2010) potentials as single-element parts. The calculated results with this Zr-Nb alloy potential showed that: (1) the newly developed CI potential functions could simultaneously present the potential-function features of Zr and Nb; (2) the normalized energy-volume curves of L1(2) Zr3Nb, B2 ZrNb and L1(2) ZrNb3 calculated by this CI potential reasonably agreed with FP results; (3) the referred MA-III Zr and FPW Nb potentials can satisfactorily reproduce the priority of prismatic slip in Zr and the tension-compression asymmetry of {$<$} 111 \& rang;\{112\} slip in Nb, while other ab initio developed Zr-Nb alloy potentials cannot. Our study indicates that, this NN based framework can take full advantage of single-element potentials, and is very convenient to develop EAM potentials of alloys; moreover, the new-developed Zr-Nb alloy EAM potential can reasonably describe the complicated deformation behaviors in Zr-Nb systems.},
    annotation = {WOS:000596723200001},
    author = {Lin, Bo and Wang, Jincheng and Li, Junjie and Wang, Zhijun},
    date = {2021-02-24},
    doi = {10.1088/1361-648X/abcb69},
    issn = {0953-8984},
    journaltitle = {Journal of Physics-Condensed Matter},
    langid = {english},
    location = {{Bristol}},
    number = {8},
    pages = {084004},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {J. Phys.-Condes. Matter},
    shorttitle = {A Neural-Network Based Framework of Developing Cross Interaction in Alloy Embedded-Atom Method Potentials},
    title = {A Neural-Network Based Framework of Developing Cross Interaction in Alloy Embedded-Atom Method Potentials: Application to {{Zr}}-{{Nb}} Alloy},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {33}
}

@article{linNumericalMethodsKohnSham2019,
    abstract = {Kohn-Sham density functional theory (DFT) is the most widely used electronic structure theory. Despite significant progress in the past few decades, the numerical solution of Kohn-Sham DFT problems remains challenging, especially for large-scale systems. In this paper we review the basics as well as state-of-the-art numerical methods, and focus on the unique numerical challenges of DFT.},
    annotation = {WOS:000491992100004},
    author = {Lin, Lin and Lu, Jianfeng and Ying, Lexing},
    date = {2019},
    doi = {10.1017/S0962492919000047},
    issn = {0962-4929},
    journaltitle = {Acta Numerica},
    langid = {english},
    location = {{New York}},
    pages = {405--539},
    publisher = {{Cambridge Univ Press}},
    shortjournal = {Acta Numer.},
    title = {Numerical Methods for {{Kohn}}-{{Sham}} Density Functional Theory},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {28}
}

@article{linUnravellingFastAlkaliIon2021a,
    abstract = {Solid-state nuclear magnetic resonance (ssNMR) has received extensive attention in characterizing alkali-ion battery materials because it is highly sensitive for probing the local environment and dynamic information of atoms/ions. However, precise spectral assignment cannot be carried out by conventional DFT for high-rate battery materials at room temperature. Herein, combining DFT calculation of paramagnetic shift and deep potential molecular dynamics (DPMD) simulation to achieve the converged Na+ distribution at hundreds of nanoseconds, we obtain the statistically averaged paramagnetic shift, which is in excellent agreement with ssNMR measurements. Two Na-23 shifts induced by different stacking sequences of transition metal layers are revealed in the fast chemically exchanged NMR spectra of P2-type Na-2/3(Mg1/3Mn2/3)O-2 for the first time. This DPMD simulation auxiliary protocol can be beneficial to a wide range of ssNMR analysis in fast chemically exchanged material systems.},
    annotation = {WOS:000641246200001},
    author = {Lin, Min and Liu, Xiangsi and Xiang, Yuxuan and Wang, Feng and Liu, Yunpei and Fu, Riqiang and Cheng, Jun and Yang, Yong},
    date = {2021-05-25},
    doi = {10.1002/anie.202102740},
    issn = {1433-7851},
    journaltitle = {Angewandte Chemie-International Edition},
    langid = {english},
    location = {{Weinheim}},
    number = {22},
    pages = {12547--12553},
    publisher = {{Wiley-V C H Verlag Gmbh}},
    shortjournal = {Angew. Chem.-Int. Edit.},
    title = {Unravelling the {{Fast Alkali}}-{{Ion Dynamics}} in {{Paramagnetic Battery Materials Combined}} with {{NMR}} and {{Deep}}-{{Potential Molecular Dynamics Simulation}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {60}
}

@article{liPowerNetEfficientRepresentations2020,
    abstract = {Deep neural network with rectified linear units (ReLU) is getting more and more popular recently. However, the derivatives of the function represented by a ReLU network are not continuous, which limit the usage of ReLU network to situations only when smoothness is not required. In this paper, we construct deep neural networks with rectified power units (RePU), which can give better approximations for smooth functions. Optimal algorithms are proposed to explicitly build neural networks with sparsely connected RePUs, which we call PowerNets, to represent polynomials with no approximation error. For general smooth functions, we first project the function to their polynomial approximations, then use the proposed algorithms to construct corresponding PowerNets. Thus, the error of best polynomial approximation provides an upper bound of the best RePU network approximation error. For smooth functions in higher dimensional Sobolev spaces, we use fast spectral transforms for tensor-product grid and sparse grid discretization to get polynomial approximations. Our constructive algorithms show clearly a close connection between spectral methods and deep neural networks: PowerNets with n hidden layers can exactly represent polynomials up to degree s(n), where s is the power of RePUs. The proposed PowerNets have potential applications in the situations where high-accuracy is desired or smoothness is required.},
    annotation = {WOS:000531091900004},
    author = {Li, Bo and Tang, Shanshan and Yu, Haijun},
    date = {2020-06},
    doi = {10.4208/jms.v53n2.20.03},
    issn = {1006-6837},
    journaltitle = {Journal of Mathematical Study},
    langid = {english},
    location = {{Wanchai}},
    number = {2},
    pages = {159--191},
    publisher = {{Global Science Press}},
    shortjournal = {J. Math. Study},
    shorttitle = {{{PowerNet}}},
    title = {{{PowerNet}}: {{Efficient Representations}} of {{Polynomials}} and {{Smooth Functions}} by {{Deep Neural Networks}} with {{Rectified Power Units}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {53}
}

@article{liTheoreticalStudyNa2021a,
    abstract = {Recent discoveries in the antiperovskite-class sodium superionic conductors call for a thorough molecular dynamics (MD) study of sodium ion mobility, but the practical use of MD is often hindered by the accuracy-vs.-efficiency dilemma. Here we applied the recently developed deep potential molecular dynamics (DeePMD) approach to investigate the ion mobility in Na3OBr. With the deep potential model for Na3OBr constructed based on first-principles density-functional theory (DFT) calculations, we directly calculate the Na+ diffusion coefficient at various temperatures, and obtain an activation energy of 0.42-0.43 eV. This in comparison with the 0 K migration barrier (0.41-0.43 eV) suggests that the finite temperature effect is negligible for Na3OBr. The model gives an extrapolated room temperature ionic conductivity of 1 x 10(-4)-2 x 10(-4) mS cm(-1), roughly in the same order of magnitude as the experimental results. We also confirm the proportionality of the diffusion coefficient with respect to the vacancy concentration, and find that the migration barrier is relatively insensitive to the vacancy concentration. This work further demonstrates the promising role of the DeePMD method in the study of the transport properties of solid-state electrolytes.},
    annotation = {WOS:000611857300017},
    author = {Li, Han-Xiao and Zhou, Xu-Yuan and Wang, Yue-Chao and Jiang, Hong},
    date = {2021-01-21},
    doi = {10.1039/d0qi00921k},
    issn = {2052-1553},
    journaltitle = {Inorganic Chemistry Frontiers},
    langid = {english},
    location = {{Cambridge}},
    number = {2},
    pages = {425--432},
    publisher = {{Royal Soc Chemistry}},
    shortjournal = {Inorg. Chem. Front.},
    title = {Theoretical Study of {{Na}}+ Transport in the Solid-State Electrolyte {{Na3OBr}} Based on Deep Potential Molecular Dynamics},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {8}
}

@article{liUnifiedDeepNeural2020a,
    abstract = {Molecular dynamics (MD) simulations have been extensively used to predict thermal properties, but simulating different phases with similar precision using a unified force field is often difficult because of the lack of accurate and transferrable interatomistic potential fields. As a result, this issue has become a major barrier to predicting the phase change of materials and their transport properties with atomistic-level modeling techniques. Recently, machine learning-based algorithms have emerged as promising tools to develop accurate potentials for MD simulations. In this work, we approach the problem of predicting the thermal conductivity of silicon in different phases by performing MD simulations with a deep neural network potential (NNP). This NNP is trained with ab initio data of silicon in the crystalline, liquid, and amorphous phases. The accuracy of our potential is first validated through reproducing the atomistic structures during the phase transition, where other empirical potentials usually fail. The thermal conductivity of different phases is then calculated, showing a good agreement with the experimental results and ab initio calculation results. Our work shows that a unified neural network ebased potential can be a promising tool for studying phase change and thermal transport of materials with high accuracy. (c) 2020 Elsevier Ltd. All rights reserved.},
    annotation = {WOS:000528878800010},
    author = {Li, R. and Lee, E. and Luo, T.},
    date = {2020-03},
    doi = {10.1016/j.mtphys.2020.100181},
    issn = {2542-5293},
    journaltitle = {Materials Today Physics},
    langid = {english},
    location = {{Amsterdam}},
    pages = {100181},
    publisher = {{Elsevier}},
    shortjournal = {Mater. Today Phys.},
    title = {A Unified Deep Neural Network Potential Capable of Predicting Thermal Conductivity of Silicon in Different Phases},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {12}
}

@article{liuRapidDetectionStrong2020a,
    abstract = {Despite its widespread use in chemical discovery, approximate density functional theory (DFT) is poorly suited to many targets, such as those containing open-shell, 3d transition metals that can be expected to have strong multireference (MR) character. For discovery workflows to be predictive, we need automated, low-cost methods that can distinguish the regions of chemical space where DFT should be applied from those where it should not. We curate more than 4800 open-shell transition-metal complexes up to hundreds of atoms in size from prior high-throughput DFT studies and evaluate affordable, finite-temperature DFT fractional occupation number (FON)-based MR diagnostics. We show that intuitive measures of strong correlation (i.e., the HOMO-LUMO gap) are not predictive of MR character as judged by FON-based diagnostics. Analysis of independently trained machine learning (ML) models to predict HOMO-LUMO gaps and FON-based diagnostics reveals differences in the metal and ligand sensitivity of the two quantities. We use our trained ML models to rapidly evaluate MR character over a space of similar to 187000 theoretical complexes, identifying large-scale trends in spin-state-dependent MR character and finding small HOMO-LUMO gap complexes while ensuring low MR character.},
    annotation = {WOS:000577152900021},
    author = {Liu, Fang and Duan, Chenru and Kulik, Heather J.},
    date = {2020-10-01},
    doi = {10.1021/acs.jpclett.0c02288},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {19},
    pages = {8067--8076},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    title = {Rapid {{Detection}} of {{Strong Correlation}} with {{Machine Learning}} for {{Transition}}-{{Metal Complex High}}-{{Throughput Screening}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {11}
}

@article{liuStructureDynamicsWarm2020,
    abstract = {We perform a systematic study on the structure and dynamics of warm dense aluminum (Al) at temperatures ranging from 0.5 to 5.0 eV with molecular dynamics utilizing both density functional theory (DFT) and the deep potential (DP) method. On one hand, unlike the Thomas-Fermi kinetic energy density functional (KEDF), we find that the orbital-free DFT method with the Wang-Teter non-local KEDF yields properties of warm dense Al that agree well with the Kohn-Sham DFT method, enabling accurate orbital-free DFT simulations of warm dense Al at relatively low temperatures. On the other hand, the DP method constructs a deep neural network that has a high accuracy in reproducing short- and long-ranged properties of warm dense Al when compared to the DFT methods. The DP method is orders of magnitudes faster than DFT and is well-suited for simulating large systems and long trajectories to yield accurate properties of warm dense Al. Our results suggest that the combination of DFT methods and the DP model is a powerful tool for accurately and efficiently simulating warm dense matter.},
    annotation = {WOS:000507895000001},
    author = {Liu, Qianrui and Lu, Denghui and Chen, Mohan},
    date = {2020-04-03},
    doi = {10.1088/1361-648X/ab5890},
    issn = {0953-8984},
    journaltitle = {Journal of Physics-Condensed Matter},
    langid = {english},
    location = {{Bristol}},
    number = {14},
    pages = {144002},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {J. Phys.-Condes. Matter},
    shorttitle = {Structure and Dynamics of Warm Dense Aluminum},
    title = {Structure and Dynamics of Warm Dense Aluminum: A Molecular Dynamics Study with Density Functional Theory and Deep Potential},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {32}
}

@article{liuThermalTransportElectrons2021,
    abstract = {We propose an efficient scheme that combines density functional theory (DFT) with deep potentials (DPs), to systematically study convergence issues in the computation of the electronic thermal conductivity of warm dense aluminum (2.7 g/cm3 and temperatures ranging from 0.5 eV to 5.0 eV) with respect to the number of k-points, the number of atoms, the broadening parameter, the exchange-correlation functionals, and the pseudopotentials. Furthermore, we obtain the ionic thermal conductivity using the Green-Kubo method in conjunction with DP molecular dynamics simulations, and we study size effects on the ionic thermal conductivity. This work demonstrates that the proposed method is efficient in evaluating both electronic and ionic thermal conductivities of materials.},
    author = {Liu, Qianrui and Li, Junyi and Chen, Mohan},
    date = {2021-03},
    doi = {10.1063/5.0030123},
    journaltitle = {Matter and Radiation at Extremes},
    number = {2},
    publisher = {{American Institute of Physics Inc.}},
    title = {Thermal Transport by Electrons and Ions in Warm Dense Aluminum: {{A}} Combined Density Functional Theory and Deep Potential Study},
    url = {https://aip.scitation.org/doi/full/10.1063/5.0030123},
    volume = {6}
}

@article{liuTransferableMultilevelAttention2021a,
    abstract = {The development of efficient models for predicting specific properties through machine learning is of great importance for the innovation of chemistry and material science. However, predicting global electronic structure properties like Frontier molecular orbital highest occupied molecular orbital (HOMO) and lowest unoccupied molecular orbital (LUMO) energy levels and their HOMO-LUMO gaps from the small-sized molecule data to larger molecules remains a challenge. Here, we develop a multilevel attention neural network, named DeepMoleNet, to enable chemical interpretable insights being fused into multitask learning through (1) weighting contributions from various atoms and (2) taking the atom-centered symmetry functions (ACSFs) as the teacher descriptor. The efficient prediction of 12 properties including dipole moment, HOMO, and Gibbs free energy with in chemical accuracy is achieved by using multiple benchmarks, both at the equilibrium and nonequilibrium geometries, including up to 110,000 records of data in QM9, 400,000 records in MD17, and 280,000 records in ANI-1ccx for random split evaluation. The good transferability for predicting larger molecules outside the training set is demonstrated in both equilibrium QM9 and Alchemy data sets at the density functional theory (DFT) level. Additional tests on nonequilibrium molecular conformations from DFT-based MD17 data set and ANI-1ccx data set with coupled cluster accuracy as well as the public test sets of singlet fission molecules, biomolecules, long oligomers, and protein with up to 140 atoms show reasonable predictions for thermodynamics and electronic structure properties. The proposed multilevel attention neural network is applicable to high-throughput screening of numerous chemical species in both equilibrium and nonequilibrium molecular spaces to accelerate rational designs of drug-like molecules, material candidates, and chemical reactions.},
    annotation = {WOS:000636723700005},
    author = {Liu, Ziteng and Lin, Liqiang and Jia, Qingqing and Cheng, Zheng and Jiang, Yanyan and Guo, Yanwen and Ma, Jing},
    date = {2021-03-22},
    doi = {10.1021/acs.jcim.0c01224},
    issn = {1549-9596},
    journaltitle = {Journal of Chemical Information and Modeling},
    langid = {english},
    location = {{Washington}},
    number = {3},
    pages = {1066--1082},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem Inf. Model.},
    title = {Transferable {{Multilevel Attention Neural Network}} for {{Accurate Prediction}} of {{Quantum Chemistry Properties}} via {{Multitask Learning}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {61}
}

@article{longPDENetLearningPDEs2019a,
    abstract = {Partial differential equations (PDEs) are commonly derived based on empirical observations. However, recent advances of technology enable us to collect and store massive amount of data, which offers new opportunities for data-driven discovery of PDEs. In this paper, we propose a new deep neural network, called PDE-Net 2.0, to discover (time-dependent) PDEs from observed dynamic data with minor prior knowledge on the underlying mechanism that drives the dynamics. The design of PDE-Net 2.0 is based on our earlier work [1] where the original version of PDE-Net was proposed. PDE-Net 2.0 is a combination of numerical approximation of differential operators by convolutions and a symbolic multi-layer neural network for model recovery. Comparing with existing approaches, PDE-Net 2.0 has the most flexibility and expressive power by learning both differential operators and the nonlinear response function of the underlying PDE model. Numerical experiments show that the PDE-Net 2.0 has the potential to uncover the hidden PDE of the observed dynamics, and predict the dynamical behavior for a relatively long time, even in a noisy environment. (C) 2019 Elsevier Inc. All rights reserved.},
    annotation = {WOS:000490766100033},
    author = {Long, Zichao and Lu, Yiping and Dong, Bin},
    date = {2019-12-15},
    doi = {10.1016/j.jcp.2019.108925},
    issn = {0021-9991},
    journaltitle = {Journal of Computational Physics},
    langid = {english},
    location = {{San Diego}},
    pages = {108925},
    publisher = {{Academic Press Inc Elsevier Science}},
    shortjournal = {J. Comput. Phys.},
    shorttitle = {{{PDE}}-{{Net}} 2.0},
    title = {{{PDE}}-{{Net}} 2.0: {{Learning PDEs}} from Data with a Numeric-Symbolic Hybrid Deep Network},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {399}
}

@article{lotPANNAPropertiesArtificial2020,
    abstract = {Prediction of material properties from first principles is often a computationally expensive task. Recently, artificial neural networks and other machine learning approaches have been successfully employed to obtain accurate models at a low computational cost by leveraging existing example data. Here, we present a software package "Properties from Artificial Neural Network Architectures'' (PANNA) that provides a comprehensive toolkit for creating neural network models for atomistic systems following the Behler-Parrinello topology. Besides the core routines for neural network training, it includes data parser, descriptor builder for Behler-Parrinello class of symmetry functions and forcefield generator suitable for integration within molecular dynamics packages. PANNA offers a variety of activation and cost functions, regularization methods, as well as the possibility of using fullyconnected networks with custom size for each atomic species. PANNA benefits from the optimization and hardware-flexibility of the underlying TensorFlow engine which allows it to be used on multiple CPU/GPU/TPU systems, making it possible to develop and optimize neural network models based on large datasets. Program summary Program title: PANNA-Properties from Artificial Neural Network Architectures CPC Library link to program files: http://dx.doi.org/10.17632/mcryj6cnnh.1 Licensing provisions: MIT Programming language: Python, C++ Nature of problem: A workflow for machine learning atomistic properties and interatomic potentials using neural networks. Solution method: This package first transforms the user supplied data into pairs of precomputed input (Behler-Parrinello [1] class of symmetry functions) and target output (energy and forces) for the neural network model. The data are then packed to enable efficient reading. A user-friendly interface to TensorFlow [2] is provided to instantiate and train neural network models with varying architectures within Behler-Parrinello topology and with varying training schedules. The training can be monitored and validated with the provided tools. The derivative of the target output with respect to the input can also be used jointly in training, e.g. in the case of energy and force training. The interface with molecular dynamics codes such as LAMMPS [3] allows the neural network model to be used as an interatomic potential. Additional comments including restrictions and unusual features: The underlying neural network training engine, TensorFlow, is a prerequisite of PANNA. While there is a special LAMMPS integration performed via a patch distributed within PANNA, the network potentials can be deposited into OpenKIM [4] database and can be used with a wide range of molecular dynamics codes. The package allows different network architectures to be used for each atomic species, with different trainability setting for each network layer. It provides tools of exchanging weights between atomic species, and provides the option of building a Radial Basis Function network. The software is parallelized to take advantage of hardware architectures with multiple CPU/GPU/TPUs. (C) 2020 Elsevier B.V. All rights reserved.},
    annotation = {WOS:000590251400006},
    author = {Lot, Ruggero and Pellegrini, Franco and Shaidu, Yusuf and Kucukbenli, Emine},
    date = {2020-11},
    doi = {10.1016/j.cpc.2020.107402},
    issn = {0010-4655},
    journaltitle = {Computer Physics Communications},
    langid = {english},
    location = {{Amsterdam}},
    pages = {107402},
    publisher = {{Elsevier}},
    shortjournal = {Comput. Phys. Commun.},
    shorttitle = {{{PANNA}}},
    title = {{{PANNA}}: {{Properties}} from {{Artificial Neural Network Architectures}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {256}
}

@article{louieDiscoveringUnderstandingMaterials2021a,
    abstract = {Materials modelling and design using computational quantum and classical approaches is by now well established as an essential pillar in condensed matter physics, chemistry and materials science research, in addition to experiments and analytical theories. The past few decades have witnessed tremendous advances in methodology development and applications to understand and predict the ground-state, excited-state and dynamical properties of materials, ranging from molecules to nanoscopic/mesoscopic materials to bulk and reduced-dimensional systems. This issue of Nature Materials presents four in-depth Review Articles on the field. This Perspective aims to give a brief overview of the progress, as well as provide some comments on future challenges and opportunities. We envision that increasingly powerful and versatile computational approaches, coupled with new conceptual understandings and the growth of techniques such as machine learning, will play a guiding role in the future search and discovery of materials for science and technology. This Perspective provides an overview of the different approaches used to understand the behaviour of materials at different length scales and timescales through computation, and outlines future challenges in the description of complex systems or ultrafast non-equilibrium behaviour.},
    annotation = {WOS:000655912800009},
    author = {Louie, Steven G. and Chan, Yang-Hao and da Jornada, Felipe H. and Li, Zhenglu and Qiu, Diana Y.},
    date = {2021-06},
    doi = {10.1038/s41563-021-01015-1},
    issn = {1476-1122},
    journaltitle = {Nature Materials},
    langid = {english},
    location = {{Berlin}},
    number = {6},
    options = {useprefix=true},
    pages = {728--735},
    publisher = {{Nature Research}},
    shortjournal = {Nat. Mater.},
    title = {Discovering and Understanding Materials through Computation},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {20}
}

@article{Lu2019,
    abstract = {The microstructures of materials determine their macroscopic properties. The traditional bottom up multi-scale approach provides a general strategy for studying the relationship between microstructures and physical properties. However, there are still many difficulties in microscopic, mesoscopic and macroscopic modeling of materials, and the bridging of different scale models is extremely challenging. With the advancement of computing power in Moore's Law, and the explosive development of artificial intelligence, especially deep learning, data-driven methods are commonly used, and in crystal structure prediction, stability analysis, equation of states, optical properties, chemical synthesis, etc. have achieved good application results. The fast calculation speed and reliable prediction capabilities of deep learning can greatly improve the efficiency of material simulation. Its wide applicability provides new research ideas for some traditional problems in material microstructures and multi-scale simulations. It is expected to promote the study of material microstructures and physical properties, and to provide new research directions for modelling macroscopic physical properties based on micro-mesoscopic mechanism and prediction of material properties to meet engineering application requirements. This review article will briefly introduce the basic principles of deep learning and main types of commonly used neural networks, outline the main methods of material microstructures and multi-scale modeling, and then introduce the recent progress of deep learning method in the study of material microstruc-ture and physical properties, and review the developments and prospects of deep learning method in the field of multi-scale simulation of materials. Keywords Deep Learning, Material Microstructures and Materials Properties, Multiscale Simulation 深度学习：材料微结构与物性研究中的新动力 卢 果，段素青},
    author = {Lu, G and Duan, S},
    date = {2019},
    doi = {10.12677/mp.2019.96026},
    journaltitle = {Modern Physics 现代物理},
    number = {6},
    pages = {263--276},
    title = {Deep Learning: {{New}} Engine for the Study of Material Microstructures and Physical Properties},
    url = {https://pdf.hanspub.org/MP20190600000_96988416.pdf},
    volume = {2019}
}

@article{luDatasetConstructionExplore2021,
    abstract = {A dataset is the basis of deep learning model development, and the success of deep learning models heavily relies on the quality and size of the dataset. In this work, we present a new data preparation protocol and build a large fragment-based dataset Frag20, which consists of optimized 3D geometries and calculated molecular properties from Merck molecular force field (MMFF) and DFT at the B3LYP/6-31G* level of theory for more than half a million molecules composed of H, B, C, O, N, F, P, S, Cl, and Br with no larger than 20 heavy atoms. Based on the new dataset, we develop robust molecular energy prediction models using a simplified PhysNet architecture for both DFT-optimized and MMFF-optimized geometries, which achieve better than or close to chemical accuracy (1 kcal/mol) on multiple test sets, including CSD20 and Plati20 based on experimental crystal structures.},
    annotation = {WOS:000636723700007},
    author = {Lu, Jianing and Xia, Song and Lu, Jieyu and Zhang, Yingkai},
    date = {2021-03-22},
    doi = {10.1021/acs.jcim.1c00007},
    issn = {1549-9596},
    journaltitle = {Journal of Chemical Information and Modeling},
    langid = {english},
    location = {{Washington}},
    number = {3},
    pages = {1095--1104},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem Inf. Model.},
    title = {Dataset {{Construction}} to {{Explore Chemical Space}} with {{3D Geometry}} and {{Deep Learning}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {61}
}

@article{luDeepPotentialMolecular2021a,
    abstract = {We present the GPU version of DeePMD-kit, which, upon training a deep neural network model using ab initio data, can drive extremely large-scale molecular dynamics (MD) simulation with ab initio accuracy. Our tests show that for a water system of 12, 582, 912 atoms, the GPU version can be 7 times faster than the CPU version under the same power consumption. The code can scale up to the entire Summit supercomputer. For a copper system of 113, 246, 208 atoms, the code can perform one nanosecond MD simulation per day, reaching a peak performance of 86 PFLOPS (43\% of the peak). Such unprecedented ability to perform MD simulation with ab initio accuracy opens up the possibility of studying many important issues in materials and molecules, such as heterogeneous catalysis, electrochemical cells, irradiation damage, crack propagation, and biochemical reactions. (C) 2020 Elsevier B.V. All rights reserved.},
    annotation = {WOS:000599925900017},
    author = {Lu, Denghui and Wang, Han and Chen, Mohan and Lin, Lin and Car, Roberto and E, Weinan and Jia, Weile and Zhang, Linfeng},
    date = {2021-02},
    doi = {10.1016/j.cpc.2020.107624},
    issn = {0010-4655},
    journaltitle = {Computer Physics Communications},
    langid = {english},
    location = {{Amsterdam}},
    pages = {107624},
    publisher = {{Elsevier}},
    shortjournal = {Comput. Phys. Commun.},
    title = {Deep {{Potential Molecular Dynamics}} Simulation of 100 Million Atoms with Ab Initio Accuracy},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {259}
}

@article{lunghiUnifiedPictureCovalent2019,
    abstract = {Computational studies of chemical processes taking place over extended size and time scales are inaccessible by electronic structure theories and can be tackled only by atomistic models such as force fields. These have evolved over the years to describe the most diverse systems. However, as we improve the performance of a force field for a particular physical/chemical situation, we are also moving away from a unified description. Here, we demonstrate that a unified picture of the covalent bond is achievable within the framework of machine learning-based force fields. Ridge regression, together with a representation of the atomic environment in terms of bispectrum components, can be used to map a general potential energy surface for molecular systems at chemical accuracy. This protocol sets the ground for the generation of an accurate and universal class of potentials for both organic and organometallic compounds with no specific assumptions on the chemistry involved.},
    annotation = {WOS:000470125000103},
    author = {Lunghi, Alessandro and Sanvito, Stefano},
    date = {2019-05},
    doi = {10.1126/sciadv.aaw2210},
    issn = {2375-2548},
    journaltitle = {Science Advances},
    langid = {english},
    location = {{Washington}},
    number = {5},
    pages = {eaaw2210},
    publisher = {{Amer Assoc Advancement Science}},
    shortjournal = {Sci. Adv.},
    shorttitle = {A Unified Picture of the Covalent Bond within Quantum-Accurate Force Fields},
    title = {A Unified Picture of the Covalent Bond within Quantum-Accurate Force Fields: {{From}} Organic Molecules to Metallic Complexes' Reactivity},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {5}
}

@article{luoDeepNeuralNetwork2021a,
    abstract = {Diffusional isotope fractionation has been widely used to explain lithium (Li) isotope variations in minerals and rocks. Isotopic mass dependence of Li diffusion can be empirically expressed as (D)7(Li)/(D)6(Li) = (6/7)(beta), where D is the diffusivity of a Li isotope. The knowledge about temperature and compositional dependence of the b factor which is essential for understanding diffusion profiles and mechanisms remains unclear. Based on the potential energy and interatomic forces generated by deep neural networks trained with ab initio data, we performed deep potential molecular dynamics (DPMD) simulations of several Li pseudo-isotopes (with mass = 2, 7, 21, 42 g/mol) in albite, hydrous albite, and model basalt melts to evaluate the beta factor. Our calculated diffusivities for Li-7 in albite and model basalt melts at 1800 K compare well with experimental results. We found that b in albite melt decreases from 0:267 +/- 0:006 at 4000 K to 0:225 +/- 0:004 at 1800 K. The presence of water appears to slightly weaken the temperature dependence of beta, with beta decreasing from 0:250 +/- 0:012 to 0:228 +/- 0:031 in hydrous albite melt. The calculated b in model basalt melt takes much smaller values, decreasing from 0:215 +/- 0:006 at 4000 K to 0:132 +/- 0:015 at 1800 K. Our prediction of beta in albite and hydrous albite melts is in good agreement with experimental data. More importantly, our results suggest that Li isotope diffusion in silicate melts is strongly dependent on melt composition. The temperature and compositional effects on beta can be qualitatively explained in terms of ionic porosity and the coupled relationship between Li diffusion and the mobility of the silicate melt network. Two types of diffusion experiments are suggested to test our predicted temperature and compositional dependence of beta. This study shows that DPMD is a promising tool to simulate the diffusion of elements and isotopes in silicate melts. (C) 2021 Elsevier Ltd. All rights reserved.},
    annotation = {WOS:000651801700003},
    author = {Luo, Haiyang and Karki, Bijaya B. and Ghosh, Dipta B. and Bao, Huiming},
    date = {2021-06-15},
    doi = {10.1016/j.gca.2021.03.031},
    issn = {0016-7037},
    journaltitle = {Geochimica Et Cosmochimica Acta},
    langid = {english},
    location = {{Oxford}},
    pages = {38--50},
    publisher = {{Pergamon-Elsevier Science Ltd}},
    shortjournal = {Geochim. Cosmochim. Acta},
    title = {Deep Neural Network Potentials for Diffusional Lithium Isotope Fractionation in Silicate Melts},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {303}
}

@article{luPredictingMolecularEnergy2019a,
    abstract = {The use of neural networks to predict molecular properties calculated from high level quantum mechanical calculations has made significant advances in recent years, but most models need input geometries from DFT optimizations which limit their applicability in practice. In this work, we explored how machine learning can be used to predict molecular atomization energies and conformation stability using optimized geometries from Merck Molecular Force Field (MMFF). On the basis of the recently introduced deep tensor neural network (DTNN) approach, we first improved its training efficiency and performed an extensive search of its hyperparameters, and developed a DTNN\_7ib model which has a test accuracy of 0.34 kcal/mol mean absolute error (MAE) on QM9 data set. Then using atomic vector representations in the DTNN\_7ib model, we employed transfer learning (TL) strategy to train readout layers on the QM9(M) data set, in which QM properties are the same as in QM9 [calculated at the B3LYP/6-31G(2df,p) level] while molecular geometries are corresponding local minima optimized with MMFF94 force field. The developed TL\_QM9(M) model can achieve an MAE of 0.79 kcal/mol using MMFF optimized geometries. Furthermore, we demonstrated that the same transfer learning strategy with the same atomic vector representation can be used to develop a machine learning model that can achieve an MAE of 0.51 kcal/mol in molecular energy prediction using MMFF geometries for an eMol9\_C-M conformation data set, which consists of 9959 molecules and 88 234 conformations with energies calculated at the B3LYP/6-31G* level. Our results indicate that DFT-level accuracy of molecular energy prediction can be achieved using force-field optimized geometries and atomic vector representations learned from deep tensor neural network, and integrated molecular modeling and machine learning would be a promising approach to develop more powerful computational tools for molecular conformation analysis.},
    annotation = {WOS:000475409000019},
    author = {Lu, Jianing and Wang, Cheng and Zhang, Yingkai},
    date = {2019-07},
    doi = {10.1021/acs.jctc.9b00001},
    issn = {1549-9618},
    journaltitle = {Journal of Chemical Theory and Computation},
    langid = {english},
    location = {{Washington}},
    number = {7},
    pages = {4113--4121},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem. Theory Comput.},
    title = {Predicting {{Molecular Energy Using Force}}-{{Field Optimized Geometries}} and {{Atomic Vector Representations Learned}} from an {{Improved Deep Tensor Neural Network}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {15}
}

@article{mailoaFastNeuralNetwork2019a,
    abstract = {Neural network force field (NNFF) is a method for performing regression on atomic structure-force relationships, bypassing the expensive quantum mechanics calculations that prevent the execution of long ab initio quality molecular dynamics (MD) simulations. However, most NNFF methods for complex multi-element atomic systems indirectly predict atomic force vectors by exploiting just atomic structure rotation-invariant features and network-feature spatial derivatives, which are computationally expensive. Here, we show a staggered NNFF architecture that exploits both rotation-invariant and -covariant features to directly predict atomic force vectors without using spatial derivatives, and we demonstrate 2.2x NNFF-MD acceleration over a state-of-the-art C++ engine using a Python engine. This fast architecture enables us to develop NNFF for complex ternary- and quaternary-element extended systems composed of long polymer chains, amorphous oxide and surface chemical reactions. The rotation-invariant-covariant architecture described here can also directly predict complex covariant vector outputs from local environments, in other domains beyond computational material science. Neural network force fields promise to bypass the computationally expensive quantum mechanical calculations typically required to investigate complex materials, such as lithium-ion batteries. Mailoa et al. accelerate these approaches with an architecture that exploits both rotation-invariant and -covariant features separately.},
    annotation = {WOS:000571255900005},
    author = {Mailoa, Jonathan P. and Kornbluth, Mordechai and Batzner, Simon and Samsonidze, Georgy and Lam, Stephen T. and Vandermause, Jonathan and Ablitt, Chris and Molinari, Nicola and Kozinsky, Boris},
    date = {2019-10},
    doi = {10.1038/s42256-019-0098-0},
    journaltitle = {Nature Machine Intelligence},
    langid = {english},
    location = {{London}},
    number = {10},
    pages = {471--479},
    publisher = {{Springernature}},
    shortjournal = {Nat. Mach. Intell.},
    title = {A Fast Neural Network Approach for Direct Covariant Forces Prediction in Complex Multi-Element Extended Systems},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {1}
}

@article{maityEvaluationExperimentalAlkali2019,
    abstract = {We applied the domain based local pair natural orbital coupled cluster approach with single, double, and perturbative triple excitations, DLPNO-CCSD(T), to rationalize more than 130 experimental bond dissociation enthalpies collected in the work of Rodgers and Armentrout [Chem. Rev. 116, 5642-5687 (2016)] and involving alkali metal cations and versatile neutral organic and inorganic ligands ranging from common solvents to amino acids. In general, a remarkable agreement has been obtained between predicted and experimental alkali metal ion-ligand noncovalent bond strengths, highlighting a high degree of reliability of data assembled by Rodgers and Armentrout. In the case of some inconsistent experimental data given for some species, we pointed to a number for which best agreement with DLPNO-CCSD(T) calculations has been achieved. In addition, we refined a couple of Delta H-0 for which DLPNO-CCSD(T) values turned out to be significantly different from their experimental counterparts. We suggest an application of the DLPNO-CCSD(T) to derive the reference values to train/validate force field and neural network methods to be further applied in molecular dynamic simulations to unravel the mechanisms in biological systems and alkali metal ion batteries.},
    annotation = {WOS:000474214600014},
    author = {Maity, Bholanath and Minenkov, Yury and Cavallo, Luigi},
    date = {2019-07-07},
    doi = {10.1063/1.5099580},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {1},
    pages = {014301},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Evaluation of Experimental Alkali Metal Ion-Ligand Noncovalent Bond Strengths with {{DLPNO}}-{{CCSD}}({{T}}) Method},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {151}
}

@article{martelliConnectionLiquidNoncrystalline2020a,
    abstract = {The origin of water anomalies hides in an experimentally inaccessible region of the phase diagram known as no-man's land, bounded at low temperature by the domain of stability of amorphous glasses, and at high temperature by the homogeneous nucleation line, below which liquid water loses its metastability. The existence of at least two different forms of glass on one side, i.e., the low-density amorphous (LDA) and the high-density amorphous (HDA) ices, and of one anomalous liquid on the other side, points to a hidden connection between these states, whose understanding has the potential to uncover what happens in no-man's land and shed light on the complex nature of water's behavior. Here, we develop a Neural Network scheme capable of discerning local structures beyond tetrahedrality. Applied over a wide region of the water's phase diagram, we show that the local structures that characterize both LDA and HDA amorphous phases are indeed embedded in the supercooled liquid phase. Remarkably, the rapid increase in the LDA-like population with supercooling occurs in the same temperature and pressure region where thermodynamic fluctuations are maximized, linking these structures with water's anomalies. At the same time, the population of HDA-like environments rapidly increases with pressure, becoming the majority component at high density. Our results show that both LDA and HDA are genuine glasses, and provide a microscopic connection between the non-equilibrium and equilibrium phase diagrams of water.},
    annotation = {WOS:000570952000003},
    author = {Martelli, Fausto and Leoni, Fabio and Sciortino, Francesco and Russo, John},
    date = {2020-09-14},
    doi = {10.1063/5.0018923},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {10},
    pages = {104503},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Connection between Liquid and Non-Crystalline Solid Phases in Water},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {153}
}

@article{materDeepLearningChemistry2019a,
    abstract = {Machine learning enables computers to address problems by learning from data. Deep learning is a type of machine learning that uses a hierarchical recombination of features to extract pertinent information and then learn the patterns represented in the data. Over the last eight years, its abilities have increasingly been applied to a wide variety of chemical challenges, from improving computational chemistry to drug and materials design and even synthesis planning. This review aims to explain the concepts of deep learning to chemists from any background and follows this with an overview of the diverse applications demonstrated in the literature. We hope that this will empower the broader chemical community to engage with this burgeoning field and foster the growing movement of deep learning accelerated chemistry.},
    annotation = {WOS:000473116500006},
    author = {Mater, Adam C. and Coote, Michelle L.},
    date = {2019-06},
    doi = {10.1021/acs.jcim.9b00266},
    issn = {1549-9596},
    journaltitle = {Journal of Chemical Information and Modeling},
    langid = {english},
    location = {{Washington}},
    number = {6},
    pages = {2545--2559},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem Inf. Model.},
    title = {Deep {{Learning}} in {{Chemistry}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {59}
}

@article{meldgaardMachineLearningEnhanced2018,
    abstract = {We show how to speed up global optimization of molecular structures using machine learning methods. To represent the molecular structures, we introduce the auto-bag feature vector that combines (i) a local feature vector for each atom, (ii) an unsupervised clustering of such feature vectors for many atoms across several structures, and (iii) a count for a given structure of how many times each cluster is represented. During subsequent global optimization searches, accumulated structure-energy relations of relaxed structural candidates are used to assign local energies to each atom using supervised learning. Specifically, the local energies follow from assigning energies to each cluster of local feature vectors and demanding the sum of local energies to amount to the structural energies in the least squares sense. The usefulness of the method is demonstrated in basin hopping searches for 19-atom structures described by single- or double-well Lennard-Jones type potentials and for 24-atom carbon structures described by density functional theory. In all cases, utilizing the local energy information derived on-the-fly enhances the rate at which the global minimum energy structure is found. Published by AIP Publishing.},
    annotation = {WOS:000446815600007},
    author = {Meldgaard, Soren A. and Kolsbjerg, Esben L. and Hammer, Bjork},
    date = {2018-10-07},
    doi = {10.1063/1.5048290},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {13},
    pages = {134104},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Machine Learning Enhanced Global Optimization by Clustering Local Environments to Enable Bundled Atomic Energies},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/6},
    urldate = {2021-08-06},
    volume = {149}
}

@article{mikschStrategiesConstructionMachinelearning2021,
    abstract = {Recent advances in machine-learning interatomic potentials have enabled the efficient modeling of complex atomistic systems with an accuracy that is comparable to that of conventional quantum-mechanics based methods. At the same time, the construction of new machine-learning potentials can seem a daunting task, as it involves data-science techniques that are not yet common in chemistry and materials science. Here, we provide a tutorial-style overview of strategies and best practices for the construction of artificial neural network (ANN) potentials. We illustrate the most important aspects of (a) data collection, (b) model selection, (c) training and validation, and (d) testing and refinement of ANN potentials on the basis of practical examples. Current research in the areas of active learning and delta learning are also discussed in the context of ANN potentials. This tutorial review aims at equipping computational chemists and materials scientists with the required background knowledge for ANN potential construction and application, with the intention to accelerate the adoption of the method, so that it can facilitate exciting research that would otherwise be challenging with conventional strategies.},
    annotation = {WOS:000674925500001},
    author = {Miksch, April M. and Morawietz, Tobias and Kaestner, Johannes and Urban, Alexander and Artrith, Nongnuch},
    date = {2021-09},
    doi = {10.1088/2632-2153/abfd96},
    journaltitle = {Machine Learning-Science and Technology},
    langid = {english},
    location = {{Bristol}},
    number = {3},
    pages = {031001},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {Mach. Learn.-Sci. Technol.},
    title = {Strategies for the Construction of Machine-Learning Potentials for Accurate and Efficient Atomic-Scale Simulations},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {2}
}

@article{minenkovaGasPhaseSilver2019a,
    abstract = {Domain-based local pair natural orbital coupled cluster approach with single, double, and perturbative triple excitations, DLPNO-CCSD(T), has been applied within a framework of a reduced version of the reaction-based Feller-Peterson-Dixon (FPD) scheme to predict gas phase heats of formation and absolute entropies of silver inorganic and organometallic compounds. First, we evaluated all existing experimental data currently limited by thermodynamic functions of 10 silver substances (AgH, AgF, AgBr, AgI, Ag-2, Ag2S, Ag2Se, Ag2Te, AgCN, AgPO2). The mean average deviation between computed and experimental heats of formation was found to be 1.9 kcal/mol. Notably, all predicted heats of formation turned out to be within the error bounds of their experimental counterparts. Second, we predicted heats of formation and entropies for additional 90 silver species with no experimental data available, substantially enriching silver thermochemistry. Combination of gas phase heats of formation Delta H-f and entropies S degrees of AgNO2, AgSCN, Ag2SO4, and Ag2SeO4 obtained in this work, with respective solid-state information, resulted in accurate sublimation thermochemistry of these compounds. Complementation of predicted Delta H-f with heats of formation of some neutrals and positive ions produced 33 silver bond strengths of high reliability. Obtained thermochemical data are promising for developing the concepts of silver chemistry. In addition, derived heats of formation and bond dissociation enthalpies, due to their high diversity, are found to be relevant for testing and training of computational chemistry methods.},
    annotation = {WOS:000472241400029},
    author = {Minenkova, Irina and Slizney, Valery V. and Cavallo, Luigi and Minenkov, Yury},
    date = {2019-06-17},
    doi = {10.1021/acs.inorgchem.9b00556},
    issn = {0020-1669},
    journaltitle = {Inorganic Chemistry},
    langid = {english},
    location = {{Washington}},
    number = {12},
    pages = {7873--7885},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {Inorg. Chem.},
    title = {Gas {{Phase Silver Thermochemistry}} from {{First Principles}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {58}
}

@article{moradzadehMolecularDynamicsProperties2019a,
    abstract = {Molecular dynamics (MD) simulation is a popularly used computational tool to compute microscopic and macroscopic properties of a variety of systems including liquids, solids, biological systems, etc. To determine properties of atomic systems to a good level of accuracy with minimal noise or fluctuation, MD simulations are performed over a long time ranging from a few nanoseconds to several tens to hundreds of nanoseconds depending on the system and the properties of interest. In this study, by considering simple liquids, we explore the feasibility of significantly reducing the MD simulation time to compute various properties of monatomic systems such as the structure, pressure, and isothermal compressibility. To do so, extensive MD simulations are performed on 12 000 distinct Lennard-Jones systems at various thermodynamic states. Then, a deep denoising autoencoder network is trained to take the radial distribution function (RDF) from a single snapshot of a Lennard-Jones liquid to compute the mean, temporally averaged RDF. We show that the method is successful in the prediction of RDF and other properties such as the pressure and isothermal compressibility that can be computed based on the RDF not only for Lennard-Jones liquids at various thermodynamic states but also for various simple liquids described by exponential, Yukawa, and inverse-power-law pair potentials.},
    annotation = {WOS:000503919300005},
    author = {Moradzadeh, Alireza and Aluru, N. R.},
    date = {2019-12-19},
    doi = {10.1021/acs.jpclett.9b02820},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {24},
    pages = {7568--7576},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    shorttitle = {Molecular {{Dynamics Properties}} without the {{Full Trajectory}}},
    title = {Molecular {{Dynamics Properties}} without the {{Full Trajectory}}: {{A Denoising Autoencoder Network}} for {{Properties}} of {{Simple Liquids}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {10}
}

@article{morawietzMachineLearningacceleratedQuantum2021a,
    abstract = {Atomistic simulations have become an invaluable tool for industrial applications ranging from the optimization of protein-ligand interactions for drug discovery to the design of new materials for energy applications. Here we review recent advances in the use of machine learning (ML) methods for accelerated simulations based on a quantum mechanical (QM) description of the system. We show how recent progress in ML methods has dramatically extended the applicability range of conventional QM-based simulations, allowing to calculate industrially relevant properties with enhanced accuracy, at reduced computational cost, and for length and time scales that would have otherwise not been accessible. We illustrate the benefits of ML-accelerated atomistic simulations for industrial R\&D processes by showcasing relevant applications from two very different areas, drug discovery (pharmaceuticals) and energy materials. Writing from the perspective of both a molecular and a materials modeling scientist, this review aims to provide a unified picture of the impact of ML-accelerated atomistic simulations on the pharmaceutical, chemical, and materials industries and gives an outlook on the exciting opportunities that could emerge in the future.},
    annotation = {WOS:000577932000001},
    author = {Morawietz, Tobias and Artrith, Nongnuch},
    date = {2021-04},
    doi = {10.1007/s10822-020-00346-6},
    issn = {0920-654X},
    journaltitle = {Journal of Computer-Aided Molecular Design},
    langid = {english},
    location = {{Dordrecht}},
    number = {4},
    pages = {557--586},
    publisher = {{Springer}},
    shortjournal = {J. Comput.-Aided Mol. Des.},
    title = {Machine Learning-Accelerated Quantum Mechanics-Based Atomistic Simulations for Industrial Applications},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {35}
}

@article{moussaAssessmentLocalizedRandomized2019a,
    abstract = {As electronic structure simulations continue to grow in size, the system-size scaling of computational costs increases in importance relative to cost prefactors. Presently, linear-scaling costs for three-dimensional systems are only attained by localized or randomized algorithms that have large cost prefactors in the difficult regime of low-temperature metals. Using large copper clusters in a minimal-basis semiempirical model as our reference system, we study the costs of these algorithms relative to a conventional cubic-scaling algorithm using matrix diagonalization and a recent quadratic-scaling algorithm using sparse matrix factorization and rational function approximation. The linear-scaling algorithms are competitive at the high temperatures relevant for warm dense matter, but their cost prefactors are prohibitive near ambient temperatures. To further reduce costs, we consider hybridized algorithms that combine localized and randomized algorithms. While simple hybridized algorithms do not improve performance, more sophisticated algorithms using recent concepts from structured linear algebra show promising initial performance results on a simple-cubic orthogonal tight-binding model.},
    annotation = {WOS:000604917000001},
    author = {Moussa, Jonathan E. and Baczewski, Andrew D.},
    date = {2019-09},
    doi = {10.1088/2516-1075/ab2022},
    issn = {2516-1075},
    journaltitle = {Electronic Structure},
    langid = {english},
    location = {{Bristol}},
    number = {3},
    pages = {033001},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {Electron. Struct.},
    title = {Assessment of Localized and Randomized Algorithms for Electronic Structure},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {1}
}

@inproceedings{mrazekTrafficControlTraffic2019,
    abstract = {The article is focused on dynamic driving in road transport. Current systems on the market are referred to as dynamic but have their limitations. Traffic management is an important aspect to ensure the proper functioning and safety of transport as an element of critical infrastructure. Ensuring the prevention of possible threats can minimize the possibility of threatening other elements of critical infrastructure. The proposed method deals with dynamic driving in road transport. Dynamically controlled traffic lights should ensure greater smoothness, safety and reduce the number of harmful substances emitting vehicles.},
    annotation = {WOS:000647641800004},
    author = {Mrazek, Jan and Mrazkova, Lucia Duricova and Hromada, Martin},
    booktitle = {2019 3rd {{European Conference}} on {{Electrical Engineering}} and {{Computer Science}} (Eecs 2019)},
    date = {2019},
    doi = {10.1109/EECS49779.2019.00017},
    isbn = {978-1-72816-109-9},
    langid = {english},
    location = {{Los Alamitos}},
    pages = {19--21},
    publisher = {{Ieee Computer Soc}},
    title = {Traffic {{Control Through Traffic Density}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/6},
    urldate = {2021-08-06}
}

@article{muellerMachineLearningInteratomic2020a,
    abstract = {The use of supervised machine learning to develop fast and accurate interatomic potential models is transforming molecular and materials research by greatly accelerating atomic-scale simulations with little loss of accuracy. Three years ago, Jorg Behler published a perspective in this journal providing an overview of some of the leading methods in this field. In this perspective, we provide an updated discussion of recent developments, emerging trends, and promising areas for future research in this field. We include in this discussion an overview of three emerging approaches to developing machine-learned interatomic potential models that have not been extensively discussed in existing reviews: moment tensor potentials, message-passing networks, and symbolic regression. Published under license by AIP Publishing.},
    annotation = {WOS:000531243200002},
    author = {Mueller, Tim and Hernandez, Alberto and Wang, Chuhong},
    date = {2020-02-07},
    doi = {10.1063/1.5126336},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {5},
    pages = {050902},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Machine Learning for Interatomic Potential Models},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {152}
}

@article{mujalSupervisedLearningFew,
    abstract = {Supervised machine learning is emerging as a powerful computational tool to predict the properties of complex quantum systems at a limited computational cost. In this article, we quantify how accurately deep neural networks can learn the properties of disordered quantum systems as a function of the system size. We implement a scalable convolutional network that can address arbitrary system sizes. This network is compared with a recently introduced extensive convolutional architecture [K. Mills et al., Chem. Sci. 10, 4129 (2019)] and with conventional dense networks with all-to-all connectivity. The networks are trained to predict the exact ground-state energies of various disordered systems, namely a continuous-space single-particle Hamiltonian for cold-atoms in speckle disorder, and different setups of a quantum Ising chain with random couplings, including one with only short-range interactions and one augmented with a long-range term. In all testbeds we consider, the scalable network retains high accuracy as the system size increases. Furthermore, we demonstrate that the network scalability enables a transfer-learning protocol, whereby a pre-training performed on small systems drastically accelerates the learning of large-system properties, allowing reaching high accuracy with small training sets. In fact, with the scalable network one can even extrapolate to sizes larger than those included in the training set, accurately reproducing the results of state-of-the-art quantum Monte Carlo simulations.},
    archiveprefix = {arXiv},
    arxivid = {2005.14290v1},
    author = {Mujal, P and Miguel, À Martínez and Polls, A},
    eprint = {2005.14290v1},
    eprinttype = {arxiv},
    journaltitle = {scipost.org},
    title = {Supervised Learning of Few Dirty Bosons with Variable Particle Number},
    url = {https://www.scipost.org/SciPostPhys.10.3.073/pdf}
}

@article{musilMachineLearningAtomic2019,
    abstract = {Statistical learning algorithms are finding more and more applications in science and technology. Atomic-scale modeling is no exception, with machine learning becoming commonplace as a tool to predict energy, forces and properties of molecules and condensed-phase systems. This short review summarizes recent progress in the field, focusing in particular on the problem of representing an atomic configuration in a mathematically robust and computationally efficient way. We also discuss some of the regression algorithms that have been used to construct surrogate models of atomic-scale properties. We then show examples of how the optimization of the machine-learning models can both incorporate and reveal insights onto the physical phenomena that underlie structure-property relations.},
    annotation = {WOS:000504762100002},
    author = {Musil, Felix and Ceriotti, Michele},
    date = {2019-12},
    doi = {10.2533/chimia.2019.972},
    issn = {0009-4293},
    journaltitle = {Chimia},
    langid = {english},
    location = {{Bern}},
    number = {12},
    pages = {972--982},
    publisher = {{Swiss Chemical Soc}},
    shortjournal = {Chimia},
    title = {Machine {{Learning}} at the {{Atomic Scale}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {73}
}

@article{nigamRecursiveEvaluationIterative2020a,
    abstract = {Mapping an atomistic configuration to a symmetrized N-point correlation of a field associated with the atomic positions (e.g., an atomic density) has emerged as an elegant and effective solution to represent structures as the input of machine-learning algorithms. While it has become clear that low-order density correlations do not provide a complete representation of an atomic environment, the exponential increase in the number of possible N-body invariants makes it difficult to design a concise and effective representation. We discuss how to exploit recursion relations between equivariant features of different order (generalizations of N-body invariants that provide a complete representation of the symmetries of improper rotations) to compute high-order terms efficiently. In combination with the automatic selection of the most expressive combination of features at each order, this approach provides a conceptual and practical framework to generate systematically improvable, symmetry adapted representations for atomistic machine learning.},
    annotation = {WOS:000575139300001},
    author = {Nigam, Jigyasa and Pozdnyakov, Sergey and Ceriotti, Michele},
    date = {2020-09-28},
    doi = {10.1063/5.0021116},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {12},
    pages = {121101},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Recursive Evaluation and Iterative Contraction of {{N}}-Body Equivariant Features},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {153}
}

@article{niuInitioPhaseDiagram2020a,
    abstract = {Elemental gallium possesses several intriguing properties, such as a low melting point, a density anomaly and an electronic structure in which covalent and metallic features coexist. In order to simulate this complex system, we construct an ab initio quality interaction potential by training a neural network on a set of density functional theory calculations performed on configurations generated in multithermal-multibaric simulations. Here we show that the relative equilibrium between liquid gallium, alpha-Ga, beta-Ga, and Ga-II is well described. The resulting phase diagram is in agreement with the experimental findings. The local structure of liquid gallium and its nucleation into alpha-Ga and beta-Ga are studied. We find that the formation of metastable beta-Ga is kinetically favored over the thermodinamically stable alpha-Ga. Finally, we provide insight into the experimental observations of extreme undercooling of liquid Ga.},
    annotation = {WOS:000538030400010},
    author = {Niu, Haiyang and Bonati, Luigi and Piaggi, Pablo M. and Parrinello, Michele},
    date = {2020-05-27},
    doi = {10.1038/s41467-020-16372-9},
    issn = {2041-1723},
    journaltitle = {Nature Communications},
    langid = {english},
    location = {{London}},
    number = {1},
    pages = {2654},
    publisher = {{Nature Publishing Group}},
    shortjournal = {Nat. Commun.},
    title = {Ab Initio Phase Diagram and Nucleation of Gallium},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {11}
}

@article{novikovMLIPPackageMoment2021,
    abstract = {The subject of this paper is the technology (the 'how') of constructing machine-learning interatomic potentials, rather than science (the 'what' and 'why') of atomistic simulations using machine-learning potentials. Namely, we illustrate how to construct moment tensor potentials using active learning as implemented in the MLIP package, focusing on the efficient ways to automatically sample configurations for the training set, how expanding the training set changes the error of predictions, how to set up ab initio calculations in a cost-effective manner, etc. The MLIP package (short for Machine-Learning Interatomic Potentials) is available at https://mlip.skoltech.ru/download/.},
    annotation = {WOS:000660868300001},
    author = {Novikov, Ivan S. and Gubaev, Konstantin and Podryabinkin, Evgeny and Shapeev, Alexander},
    date = {2021-06},
    doi = {10.1088/2632-2153/abc9fe},
    journaltitle = {Machine Learning-Science and Technology},
    langid = {english},
    location = {{Bristol}},
    number = {2},
    pages = {025002},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {Mach. Learn.-Sci. Technol.},
    shorttitle = {The {{MLIP}} Package},
    title = {The {{MLIP}} Package: Moment Tensor Potentials with {{MPI}} and Active Learning},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {2}
}

@inproceedings{oehmckeModelingH2ORutileTiO22020,
    abstract = {Successful and cost-effective water splitting could be one of the most interesting energy sources of the future. The modification of the transition metal oxide rutile-TiO2 as photocatalyst can lead to improved performance in the water splitting process. For that purpose, an accurate description of the interaction potential of a water molecule and the rutile-TiO2(110) surface in the ground and electronically excited state after photoexcitation is crucial. The electronic Schrodinger equation for the states involved is solved pointwise for different nuclear configurations within the Born-Oppenheimer approximation, and accurate fits to these energy points are required to obtain an analytic expression for the potential energy surface. This is too computationally expensive for fine-grained surface calculations of quantum chemical models. In this paper, we propose to use state-of-the-art deep learning techniques to provide accurate fits for this problem. Namely, we employ a fully connected variant of ResNet and DenseNet with heavy regularization (L2, RReLU, Dropout, and BatchNormalization). Previous literature applied neural network approaches before, but with unsatisfactory accuracy. In an experimental evaluation we show that the root mean squared error (RMSE) can be 6.8 times lower for the exited state and 12.7 times lower for the ground state compared to former approaches.},
    annotation = {WOS:000626021405049},
    author = {Oehmcke, Stefan and Teusch, Thomas and Petersen, Thorben and Kluener, Thorsten and Kramer, Oliver},
    booktitle = {2020 {{International Joint Conference}} on {{Neural Networks}} (Ijcnn)},
    date = {2020},
    isbn = {978-1-72816-926-2},
    issn = {2161-4393},
    langid = {english},
    location = {{New York}},
    publisher = {{Ieee}},
    title = {Modeling {{H2O}}/{{Rutile}}-{{TiO2}}(110) {{Potential Energy Surfaces}} with {{Deep Networks}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06}
}

@article{orupatturCatalyticMaterialsChemistry2020a,
    abstract = {First principles-based molecular modelling plays a crucial role in the development of novel catalytic materials and in the investigation of catalytic chemical reactions. However, the computational cost and/or the accuracy of these models remains a bottleneck in carrying out these simulations for complex or large scale systems, as in the case of catalysis. Over the past two decades, machine learning (ML) has made an impact in the field of computational catalysis. Modern-day researchers have started using machine learning-based data-driven techniques to overcome the limitations of these molecular simulations. In this review, we summarize the recent progress in the utilization of ML algorithms to assist molecular simulations, followed by its applications in the field of catalysis. Furthermore, we provide our perspective on promising avenues for research in the future regarding the incorporation of ML in molecular simulations in catalysis.},
    annotation = {WOS:000509344200006},
    author = {Orupattur, Nilesh Varadan and Mushrif, Samir H. and Prasad, Vinay},
    date = {2020-03},
    doi = {10.1016/j.commatsci.2019.109474},
    issn = {0927-0256},
    journaltitle = {Computational Materials Science},
    langid = {english},
    location = {{Amsterdam}},
    pages = {109474},
    publisher = {{Elsevier}},
    shortjournal = {Comput. Mater. Sci.},
    title = {Catalytic Materials and Chemistry Development Using a Synergistic Combination of Machine Learning and Ab Initio Methods},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {174}
}

@article{paleicoBinHashMethod2021,
    abstract = {In recent years the development of machine learning potentials (MLPs) has become a very active field of research. Numerous approaches have been proposed, which allow one to perform extended simulations of large systems at a small fraction of the computational costs of electronic structure calculations. The key to the success of modern MLPs is the close-to first principles quality description of the atomic interactions. This accuracy is reached by using very flexible functional forms in combination with high-level reference data from electronic structure calculations. These data sets can include up to hundreds of thousands of structures covering millions of atomic environments to ensure that all relevant features of the potential energy surface are well represented. The handling of such large data sets is nowadays becoming one of the main challenges in the construction of MLPs. In this paper we present a method, the bin-and-hash (BAH) algorithm, to overcome this problem by enabling the efficient identification and comparison of large numbers of multidimensional vectors. Such vectors emerge in multiple contexts in the construction of MLPs. Examples are the comparison of local atomic environments to identify and avoid unnecessary redundant information in the reference data sets that is costly in terms of both the electronic structure calculations as well as the training process, the assessment of the quality of the descriptors used as structural fingerprints in many types of MLPs, and the detection of possibly unreliable data points. The BAH algorithm is illustrated for the example of high-dimensional neural network potentials using atom-centered symmetry functions for the geometrical description of the atomic environments, but the method is general and can be combined with any current type of MLP.},
    annotation = {WOS:000660870100001},
    author = {Paleico, Martin Leandro and Behler, Joerg},
    date = {2021-09},
    doi = {10.1088/2632-2153/abe663},
    journaltitle = {Machine Learning-Science and Technology},
    langid = {english},
    location = {{Bristol}},
    number = {3},
    pages = {037001},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {Mach. Learn.-Sci. Technol.},
    title = {A Bin and Hash Method for Analyzing Reference Data and Descriptors in Machine Learning Potentials},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {2}
}

@article{panDFTAccurateMachine2020a,
    abstract = {Molten eutectic salts consisting of ZnCl2 and other alkali chlorides are promising thermal storage and heat transfer fluid materials in the next generation concentrated solar thermal power. To go deep into the thermal and transport properties for a high order mixture, the microstructure information, as well as thermodynamics properties of individual components, have to be identified first. This work develops interatomic potentials of molten ZnCl2 based on neural-network machine learning approach for the first time. The machine learning potential is trained by fitting to the energies and forces of liquid structures ab initio molecular dynamics calculations. The developed machine learning potential is validated by comparing partial radial distribution functions, coordination numbers, and partial structure factors with AIMD and PIM potential. The machine learning potential yields a more precise description of the microstructures than the PIM potential which suffers from the analytical form. Furthermore, structural and thermophysical evolution with temperature are studied and the results are in good agreement with experimental values. The efficient machine learning potential with DFT accuracy from our study will provide a promising scheme for accurate molecular simulations of structures and dynamics of molten ZnCl2 mixtures.},
    annotation = {WOS:000579155500014},
    author = {Pan, Gechuanqi and Chen, Pin and Yan, Hui and Lu, Yutong},
    date = {2020-12},
    doi = {10.1016/j.commatsci.2020.109955},
    issn = {0927-0256},
    journaltitle = {Computational Materials Science},
    langid = {english},
    location = {{Amsterdam}},
    pages = {109955},
    publisher = {{Elsevier}},
    shortjournal = {Comput. Mater. Sci.},
    shorttitle = {A {{DFT}} Accurate Machine Learning Description of Molten {{ZnCl2}} and Its Mixtures},
    title = {A {{DFT}} Accurate Machine Learning Description of Molten {{ZnCl2}} and Its Mixtures: 1. {{Potential}} Development and Properties Prediction of Molten {{ZnCl2}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {185}
}

@article{panDFTAccurateMachine2021a,
    abstract = {ZnCl2-NaCl-KCl ternary salts are promising thermal storage and heat transfer fluid materials with a freezing point below 250 degrees C, thermal stability up to 800 degrees C, and other favorable properties that fit the use in the next generation concentrated solar thermal power. This work for the first time developed a machine learning-based interatomic potential for ZnCl2-NaCl-KCl ternary salt (0.6:0.2:0.2 in mole fraction) on the basis of energies and forces estimated by ab initio molecular dynamics calculations. The proposed machine learning potential was validated with the obtained partial radial distribution functions and the coordination numbers with the AIMD. The structural and thermophysical evolutions with temperature over the entire operating temperature range were documented. Adding Na+ and K+ ions deteriorated the network by corner-sharing and edge-sharing ZnCl4 tetrahedra, and apparently affected self-diffusion coefficient, thermal conductivity, and viscosity of the melt. The calculated thermophysical properties agreed with experimental data. A negative temperature dependence of thermal conductivity was noted and discussed. Based on the experimental data, viscosity data by Li et al. and those of this work, yielded reliable experimental values in the Vogel-Tamman-Fulcher form.},
    annotation = {WOS:000600373900010},
    author = {Pan, Gechuanqi and Ding, Jing and Du, Yunfei and Lee, Duu-Jong and Lu, Yutong},
    date = {2021-02-01},
    doi = {10.1016/j.commatsci.2020.110055},
    issn = {0927-0256},
    journaltitle = {Computational Materials Science},
    langid = {english},
    location = {{Amsterdam}},
    pages = {110055},
    publisher = {{Elsevier}},
    shortjournal = {Comput. Mater. Sci.},
    shorttitle = {A {{DFT}} Accurate Machine Learning Description of Molten {{ZnCl2}} and Its Mixtures},
    title = {A {{DFT}} Accurate Machine Learning Description of Molten {{ZnCl2}} and Its Mixtures: 2. {{Potential}} Development and Properties Prediction of {{ZnCl2}}-{{NaCl}}-{{KCl}} Ternary Salt for {{CSP}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {187}
}

@article{parkAccurateScalableGraph2021a,
    abstract = {Recently, machine learning (ML) has been used to address the computational cost that has been limiting ab initio molecular dynamics (AIMD). Here, we present GNNFF, a graph neural network framework to directly predict atomic forces from automatically extracted features of the local atomic environment that are translationally-invariant, but rotationally-covariant to the coordinate of the atoms. We demonstrate that GNNFF not only achieves high performance in terms of force prediction accuracy and computational speed on various materials systems, but also accurately predicts the forces of a large MD system after being trained on forces obtained from a smaller system. Finally, we use our framework to perform an MD simulation of Li7P3S11, a superionic conductor, and show that resulting Li diffusion coefficient is within 14\% of that obtained directly from AIMD. The high performance exhibited by GNNFF can be easily generalized to study atomistic level dynamics of other material systems.},
    annotation = {WOS:000658724900002},
    author = {Park, Cheol Woo and Kornbluth, Mordechai and Vandermause, Jonathan and Wolverton, Chris and Kozinsky, Boris and Mailoa, Jonathan P.},
    date = {2021-05-21},
    doi = {10.1038/s41524-021-00543-3},
    journaltitle = {Npj Computational Materials},
    langid = {english},
    location = {{Berlin}},
    number = {1},
    pages = {73},
    publisher = {{Nature Research}},
    shortjournal = {npj Comput. Mater.},
    title = {Accurate and Scalable Graph Neural Network Force Field and Molecular Dynamics with Direct Force Architecture},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {7}
}

@article{peigneyFourierbasedMachineLearning2021a,
    abstract = {The generic problem in supervised machine learning is to learn a function f from a collection of samples, with the objective of predicting the value taken by f for any given input. In effect, the learning procedure consists in constructing an explicit function that approximates f in some sense. In this article is introduced a Fourier-based machine learning method which could be an alternative or a complement to neural networks for applications in engineering. The basic idea is to extend f into a periodic function so as to use partial sums of the Fourier series as approximations. For this approach to be effective in high dimension, it proved necessary to use several ideas and concepts such as regularization, Sobol sequences and hyperbolic crosses. An attractive feature of the proposed method is that the training stage reduces to a quadratic programming problem. The presented method is first applied to some examples of high-dimensional analytical functions, which allows some comparisons with neural networks to be made. An application to a homogenization problem in nonlinear conduction is discussed in detail. Various examples related to global sensitivity analysis, assessing effective energies of microstructures, and solving boundary value problems are presented.},
    annotation = {WOS:000588550500001},
    author = {Peigney, Michael},
    date = {2021-02-15},
    doi = {10.1002/nme.6565},
    issn = {0029-5981},
    journaltitle = {International Journal for Numerical Methods in Engineering},
    langid = {english},
    location = {{Hoboken}},
    number = {3},
    pages = {866--897},
    publisher = {{Wiley}},
    shortjournal = {Int. J. Numer. Methods Eng.},
    title = {A {{Fourier}}-Based Machine Learning Technique with Application in Engineering},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {122}
}

@article{petousisBenchmarkingDensityFunctional2016,
    abstract = {We demonstrate a high-throughput density functional perturbation theory (DFPT) methodology capable of screening compounds for their dielectric properties. The electronic and ionic dielectric tensors are calculated for 88 compounds, where the eigenvalues of the total dielectric tensors are compared with single crystal and polycrystalline experimental values reported in the literature. We find that GGA/PBE has a smaller mean average deviation from experiments (MARD=16.2\%) when compared to LDA. The prediction accuracy of DFPT is lowest for compounds that exhibit complex structural relaxation effects (e.g., octahedra rotation in perovskites) and/or strong anharmonicity. Despite some discrepancies between DFPT results and reported experimental values, the high-throughput methodology is found to be useful in identifying interesting compounds by ranking. This is demonstrated by the high Spearman correlation factor (ρ=0.92). Finally, we demonstrate that DFPT provides a good estimate for the refractive index of a compound without calculating the frequency dependence of the dielectric matrix (MARD=5.7\%).},
    author = {Petousis, Ioannis and Chen, Wei and Hautier, Geoffroy and Graf, Tanja and Schladt, Thomas D. and Persson, Kristin A. and Prinz, Fritz B.},
    date = {2016-03},
    doi = {10.1103/physrevb.93.115151},
    journaltitle = {Physical Review B},
    number = {11},
    publisher = {{American Physical Society}},
    title = {Benchmarking Density Functional Perturbation Theory to Enable High-Throughput Screening of Materials for Dielectric Constant and Refractive Index},
    url = {https://journals.aps.org/prb/abstract/10.1103/PhysRevB.93.115151},
    volume = {93}
}

@article{piaggiEnhancingFormationIonic,
    abstract = {Ice Ih, the common form of ice in the biosphere, contains proton disorder. Its proton-ordered counterpart, ice XI, is thermodynamically stable below 72 K. However, the formation of ice XI is kinetically hindered, and experimentally it is obtained by doping with KOH. Doping creates ionic defects that promote the migration of protons and the associated change in proton configuration. In this article, we mimic the effect of doping with a bias potential that enhances the formation of ionic defects in molecular dynamics simulations. The recombination of the ions thus formed proceeds through fast migration of the hydroxide along hydrogen bond loops, providing a physical and expedite way to change the proton configuration. A key ingredient of this approach is a machine learning potential trained with density functional theory data and capable of modelling molecular dissociation. We exemplify the usefulness of this idea by studying the order-disorder transition using an appropriate order parameter that distinguishes the proton environments in ice Ih and XI. We calculate the changes in free energy, enthalpy, and entropy associated with the transition. Our estimated entropy agrees with experiment within the error bars of the calculation.},
    annotation = {WOS:000646841900001},
    author = {Piaggi, Pablo M. and Car, Roberto},
    doi = {10.1080/00268976.2021.1916634},
    issn = {0026-8976},
    journaltitle = {Molecular Physics},
    langid = {english},
    location = {{Abingdon}},
    publisher = {{Taylor \& Francis Ltd}},
    shortjournal = {Mol. Phys.},
    title = {Enhancing the Formation of Ionic Defects to Study the Ice {{Ih}}/{{XI}} Transition with Molecular Dynamics Simulations},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06}
}

@article{piaggiPhaseEquilibriumWater2021,
    abstract = {Machine learning models are rapidly becoming widely used to simulate complex physicochemical phenomena with ab initio accuracy. Here, we use one such model as well as direct density functional theory (DFT) calculations to investigate the phase equilibrium of water, hexagonal ice (Ih), and cubic ice (Ic), with an eye toward studying ice nucleation. The machine learning model is based on deep neural networks and has been trained on DFT data obtained using the SCAN exchange and correlation functional. We use this model to drive enhanced sampling simulations aimed at calculating a number of complex properties that are out of reach of DFT-driven simulations and then employ an appropriate reweighting procedure to compute the corresponding properties for the SCAN functional. This approach allows us to calculate the melting temperature of both ice polymorphs, the driving force for nucleation, the heat of fusion, the densities at the melting temperature, the relative stability of ices Ih and Ic, and other properties. We find a correct qualitative prediction of all properties of interest. In some cases, quantitative agreement with experiment is better than for state-of-the-art semiempirical potentials for water. Our results also show that SCAN correctly predicts that ice Ih is more stable than ice Ic.},
    annotation = {WOS:000651540200031},
    author = {Piaggi, Pablo M. and Panagiotopoulos, Athanassios Z. and Debenedetti, Pablo G. and Car, Roberto},
    date = {2021-05-11},
    doi = {10.1021/acs.jctc.1c00041},
    issn = {1549-9618},
    journaltitle = {Journal of Chemical Theory and Computation},
    langid = {english},
    location = {{Washington}},
    number = {5},
    pages = {3065--3077},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem. Theory Comput.},
    title = {Phase {{Equilibrium}} of {{Water}} with {{Hexagonal}} and {{Cubic Ice Using}} the {{SCAN Functional}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {17}
}

@article{poltavskyMachineLearningForce2021a,
    abstract = {In chemistry and physics, machine learning (ML) methods promise transformative impacts by advancing modeling and improving our understanding of complex molecules and materials. Each ML method comprises a mathematically well-defined procedure, and an increasingly larger number of easy-to-use ML packages for modeling atomistic systems are becoming available. In this Perspective, we discuss the general aspects of ML techniques in the context of creating ML force fields. We describe common features of ML modeling and quantum-mechanical approximations, so-called global and local ML models, and the physical differences behind these two classes of approaches. Finally, we describe the recent developments and emerging directions in the field of ML-driven molecular modeling. This Perspective aims to inspire interdisciplinary collaborations crossing the borders between physical chemistry, chemical physics, computer science, and data science.},
    annotation = {WOS:000677581400014},
    author = {Poltavsky, Igor and Tkatchenko, Alexandre},
    date = {2021-07-22},
    doi = {10.1021/acs.jpclett.1c01204},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {28},
    pages = {6551--6564},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    shorttitle = {Machine {{Learning Force Fields}}},
    title = {Machine {{Learning Force Fields}}: {{Recent Advances}} and {{Remaining Challenges}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {12}
}

@article{posenitskiyApplicationDeepLearning2021a,
    abstract = {Deep learning (DL) is applied to simulate non-adiabatic molecular dynamics of phenanthrene, using the time-dependent density functional based tight binding (TD-DFTB) approach for excited states combined with mixed quantum-classical propagation. Reference calculations rely on Tully's fewest-switches surface hopping (FSSH) algorithm coupled to TD-DFTB, which provides electronic relaxation dynamics in fair agreement with various available experimental results. Aiming at describing the coupled electron-nuclei dynamics in large molecular systems, we then examine the combination of DL for excited-state potential energy surfaces (PESs) with a simplified trajectory surface hopping propagation based on the Belyaev-Lebedev (BL) scheme. We start to assess the accuracy of the TD-DFTB approach upon comparison of the optical spectrum with experimental and higher-level theoretical results. Using the recently developed SchNetPack (Schutt et al 2019 J. Chem. Theory Comput. 15 448-55) for DL applications, we train several models and evaluate their performance in predicting excited-state energies and forces. Then, the main focus is given to the analysis of the electronic population of low-lying excited states computed with the aforementioned methods. We determine the relaxation timescales and compare them with experimental data. Our results show that DL demonstrates its ability to describe the excited-state PESs. When coupled to the simplified BL scheme considered in this study, it provides reliable description of the electronic relaxation in phenanthrene as compared with either the experimental data or the higher-level FSSH/TD-DFTB theoretical results. Furthermore, the DL performance allows high-throughput analysis at a negligible cost.},
    annotation = {WOS:000674930500001},
    author = {Posenitskiy, Evgeny and Spiegelman, Fernand and Lemoine, Didier},
    date = {2021-09},
    doi = {10.1088/2632-2153/abfe3f},
    journaltitle = {Machine Learning-Science and Technology},
    langid = {english},
    location = {{Bristol}},
    number = {3},
    pages = {035039},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {Mach. Learn.-Sci. Technol.},
    title = {On Application of Deep Learning to Simplified Quantum-Classical Dynamics in Electronically Excited States},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {2}
}

@article{pulighedduAtomisticSimulationsThermal2020a,
    abstract = {We present a method based on sinusoidal approach to equilibrium molecular dynamics (SAEMD) to compute the thermal conductivity of liquids Similar to nonequilibrium molecular dynamics, and unlike equilibrium simulations based on the Green-Kubo formalism, the method only requires the calculation of forces and total energies. The evaluation of heat fluxes and energy densities is not necessary, thus offering the promise of efficiently implementing first principles simulations based on density functional theory or deep molecular dynamics. Our approach is a generalization of SAEMD for solids, where the thermal conductivity is computed in the steady state, instead of a transient regime, thus properly taking into account diffusive terms in the heat equation. We present results for liquid water at ambient conditions and under pressure and discuss simulation requirements to obtain converged values of the thermal conductivity as a function of size and simulation time.},
    annotation = {WOS:000531460800001},
    author = {Puligheddu, Marcello and Galli, Giulia},
    date = {2020-05-11},
    doi = {10.1103/PhysRevMaterials.4.053801},
    issn = {2475-9953},
    journaltitle = {Physical Review Materials},
    langid = {english},
    location = {{College Pk}},
    number = {5},
    pages = {053801},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. Mater.},
    title = {Atomistic Simulations of the Thermal Conductivity of Liquids},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {4}
}

@article{qianComprehensiveAssessmentEmpirical2021,
    abstract = {Carbon materials and their unique properties have been extensively studied by molecular dynamics, thanks to the wide range of available carbon bond order potentials (CBOPs). Recently, with the increase in popularity of machine learning (ML), potentials such as Gaussian approximation potential (GAP), trained using ML, can accurately predict results for carbon. However, selecting the right potential is crucial as each performs differently for different carbon allotropes, and these differences can lead to inaccurate results. This work compares the widely used CBOPs and the GAP-20 ML potential with density functional theory results, including lattice constants, cohesive energies, defect formation energies, van der Waals interactions, thermal stabilities, and mechanical properties for different carbon allotropes. We find that GAP-20 can more accurately predict the structure, defect properties, and formation energies for a variety of crystalline phase carbon compared to CBOPs. Importantly, GAP-20 can simulate the thermal stability of C60 and the fracture of carbon nanotubes and graphene accurately, where CBOPs struggle. However, similar to CBOPs, GAP-20 is unable to accurately account for van der Waals interactions. Despite this, we find that GAP-20 outperforms all CBOPs assessed here and is at present the most suitable potential for studying thermal and mechanical properties for pristine and defective carbon.},
    author = {Qian, Cheng and McLean, Ben and Hedman, Daniel and Ding, Feng},
    date = {2021-06},
    doi = {10.1063/5.0052870},
    journaltitle = {APL Materials},
    number = {6},
    publisher = {{American Institute of Physics Inc.}},
    title = {A Comprehensive Assessment of Empirical Potentials for Carbon Materials},
    url = {https://aip.scitation.org/doi/full/10.1063/5.0052870},
    volume = {9}
}

@article{qiaoOrbNetDeepLearning2020a,
    abstract = {We introduce a machine learning method in which energy solutions from the Schrodinger equation are predicted using symmetry adapted atomic orbital features and a graph neural-network architecture. OrbNet is shown to outperform existing methods in terms of learning efficiency and transferability for the prediction of density functional theory results while employing low-cost features that are obtained from semi-empirical electronic structure calculations. For applications to datasets of drug-like molecules, including QM7b-T, QM9, GDB-13-T, DrugBank, and the conformer benchmark dataset of Folmsbee and Hutchison [Int. J. Quantum Chem. (published online) (2020)], OrbNet predicts energies within chemical accuracy of density functional theory at a computational cost that is 1000-fold or more reduced.},
    annotation = {WOS:000577137700001},
    author = {Qiao, Zhuoran and Welborn, Matthew and Anandkumar, Animashree and Manby, Frederick R. and Miller, Thomas F.},
    date = {2020-09-28},
    doi = {10.1063/5.0021955},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {12},
    pages = {124111},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    shorttitle = {{{OrbNet}}},
    title = {{{OrbNet}}: {{Deep}} Learning for Quantum Chemistry Using Symmetry-Adapted Atomic-Orbital Features},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {153}
}

@article{qiInteractionEnergyPrediction2021a,
    abstract = {The interaction energy of two molecules system plays a critical role in analyzing the interacting effect in molecular dynamic simulation. Since the limitation of quantum mechanics calculating resources, the interaction energy based on quantum mechanics can not be merged into molecular dynamic simulation for a long time scale. A deep learning framework, deep tensor neural network, is applied to predict the interaction energy of three organic related systems within the quantum mechanics level of accuracy. The geometric structure and atomic types of molecular conformation, as the data descriptors, are applied as the network inputs to predict the interaction energy in the system. The neural network is trained with the hierarchically generated conformations data set. The complex tensor hidden layers are simplified and trained in the optimization process. The predicted results of different molecular systems indicate that deep tensor neural network is capable to predict the interaction energy with 1 kcal/mol of the mean absolute error in a relatively short time. The prediction highly improves the efficiency of interaction energy calculation. The whole proposed framework provides new insights to introducing deep learning technology into the interaction energy calculation.},
    annotation = {WOS:000629031200012},
    author = {Qi, Yuan and Ren, Hong and Li, Hong and Zhang, Ding-lin and Cui, Hong-qiang and Weng, Jun-ben and Li, Guo-hui and Wang, Gui-yan and Li, Yan},
    date = {2021-02},
    doi = {10.1063/1674-0068/cjcp2009163},
    issn = {1674-0068},
    journaltitle = {Chinese Journal of Chemical Physics},
    langid = {english},
    location = {{Beijing}},
    number = {1},
    pages = {112--124},
    publisher = {{Chinese Physical Soc}},
    shortjournal = {Chin. J. Chem. Phys.},
    title = {Interaction Energy Prediction of Organic Molecules Using Deep Tensor Neural Network},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {34}
}

@article{reinhartUnsupervisedLearningAtomic2021a,
    abstract = {I present a strategy for unsupervised manifold learning on local atomic environments in molecular simulations based on simple rotation- and permutation-invariant three-body features. These features are highly descriptive, generalize to multiple chemical species, and are human-interpretable. The low-dimensional embeddings of each atomic environment can be used to understand and quantify messy crystal structures such as those near interfaces and defects or well-ordered crystal lattices such as in bulk materials without modification. The same method can also yield collective variables describing collections of particles such as for an entire simulation domain. I demonstrate the method on colloidal crystallization, ice crystals, and binary mesophases to illustrate its broad applicability. In each case, the learned latent space yields insights into the details of the observed microstructures. For ices and mesophases, supervised classifiers are trained based on the learned manifolds and directly compared against a recent neural-network-based approach. Notably, while this method provides comparable classification performance, it can also be deployed on even a handful of observed environments without labels or a priori knowledge. Thus, the current approach provides an incredibly versatile strategy to characterize and classify local atomic environments, and may unlock insights in a wide variety of molecular simulation contexts.},
    annotation = {WOS:000663758800002},
    author = {Reinhart, Wesley F.},
    date = {2021-08},
    doi = {10.1016/j.commatsci.2021.110511},
    issn = {0927-0256},
    journaltitle = {Computational Materials Science},
    langid = {english},
    location = {{Amsterdam}},
    pages = {110511},
    publisher = {{Elsevier}},
    shortjournal = {Comput. Mater. Sci.},
    title = {Unsupervised Learning of Atomic Environments from Simple Features},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {196}
}

@article{remsingHalogenBondStructure2019,
    abstract = {Halogen bonding has emerged as an important noncovalent interaction in a myriad of applications, including drug design, supramolecular assembly, and catalysis. The current understanding of the halogen bond is informed by electronic structure calculations on isolated molecules and/or crystal structures that are not readily transferable to liquids and disordered phases. To address this issue, we present a first-principles simulation-based approach for quantifying halogen bonds in molecular systems rooted in an understanding of nuclei-nuclei and electron-nuclei spatial correlations. We then demonstrate how this approach can be used to quantify the structure and dynamics of halogen bonds in condensed phases, using solid and liquid molecular chlorine as prototypical examples with high concentrations of halogen bonds. We close with a discussion of how the knowledge generated by our first-principles approach may inform the development of classical empirical models, with a consistent representation of halogen bonding.},
    annotation = {WOS:000477786700014},
    author = {Remsing, Richard C. and Klein, Michael L.},
    date = {2019-07-25},
    doi = {10.1021/acs.jpcb.9b04820},
    issn = {1520-6106},
    journaltitle = {Journal of Physical Chemistry B},
    langid = {english},
    location = {{Washington}},
    number = {29},
    pages = {6266--6273},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. B},
    title = {Halogen {{Bond Structure}} and {{Dynamics}} from {{Molecular Simulations}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {123}
}

@article{renMachineLearningKinetic2021a,
    abstract = {Kinetic energy (KE) functional is crucial to speed up density functional theory calculation. However, deriving it accurately through traditional physics reasoning is challenging. We develop a generally applicable KE functional estimator for a one-dimensional (1D) extended system using a machine learning method. Our end-to-end solution combines the dimensionality reduction method with the Gaussian process regression, and simple scaling method to adapt to various 1D lattices. In addition to reaching chemical accuracy in KE calculation, our estimator also performs well on KE functional derivative prediction. Integrating this machine learning KE functional into the current orbital free density functional theory scheme is able to provide us with expected ground state electron density.},
    annotation = {WOS:000658013000001},
    author = {Ren, Hong-Bin and Wang, Lei and Dai, Xi},
    date = {2021-06},
    doi = {10.1088/0256-307X/38/5/050701},
    issn = {0256-307X},
    journaltitle = {Chinese Physics Letters},
    langid = {english},
    location = {{Bristol}},
    number = {5},
    pages = {050701},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {Chin. Phys. Lett.},
    title = {Machine {{Learning Kinetic Energy Functional}} for a {{One}}-{{Dimensional Periodic System}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {38}
}

@article{rodriguezSpatialDensityNeural2020a,
    abstract = {Constrained by the fixed mathematical form of most empirical potentials used in classical molecular dynamics (MD) simulations, many properties of materials cannot be captured within experimental accuracy. On the other hand, accurate electronic structure calculations based on quantum theory, most notably density functional theory (DFT), are limited to several hundred atoms within a picosecond, which makes the method inadequate for modeling systems beyond the nanoscale. A combination of speed from classical MD and fidelity from DFT can be achieved through machine learning methods. Herewith, we developed an approach named spatial density neural network force fields (SDNNFFs) by training neural networks to "learn" and predict DFT-level forces. Our model focuses on the usage of a three-dimensional mesh of density functions, which together act as a mapping of the atomic environment and provides a physical representation of the forces acting on the central atom. Several notable advantages arise from the SDNNFF, including (1) the avoidance of the chain rule on the total energy and other variables by direct calculation of the forces from the neural network, (2) the ever large N x t scaling of the training data, where N is the number of atoms in a supercell and t is the number of evaluated structures by first-principles, and (3) the significant reduction in parameters and human effort needed to successfully train a force- and/or property-converged neural network force field. Overall, we focus on modeling DFT-level forces with minimal computational cost and parametrization for rapid prediction of phonon-based properties and future molecular dynamics of large-scale systems. To demonstrate the SDNNFF, we trained several models on diamond structures, including bulk silicon (Si), diamond, silicon carbide (SiC), and boron arsenide (BAs), and predicted their phonon dispersions and lattice thermal conductivities using the direct solution to the phonon Boltzmann transport equation. For phonon properties, we utilized a fitting method for obtaining the second-and third-order force constants, which outperforms the highly force-sensitive finite displacement method when employing neural network force fields. In comparison to DFT lattice thermal conductivity, we obtained high precision results from our SDNNFF within 0.7\% for Si, 6.2\% for diamond, 2.76\% for SiC, and 7.46\% for BAs, with further agreement with experiments. The phonon dispersions from the SDNNFF also matched those from direct DFT and experiments. The developed approach for accurately predicting phonon transport properties of crystalline materials would largely benefit the design of advanced materials with improved performance, such as complex thermoelectric devices and low thermal resistance interfaces for nanoelectronics. Future applications of our SDNNFF model could be extended toward including atomic energy into the algorithm and simulating large-scale heterogeneous systems for quasielectronic representations for various properties.},
    annotation = {WOS:000549756900003},
    author = {Rodriguez, Alejandro and Liu, Yinqiao and Hu, Ming},
    date = {2020-07-17},
    doi = {10.1103/PhysRevB.102.035203},
    issn = {2469-9950},
    journaltitle = {Physical Review B},
    langid = {english},
    location = {{College Pk}},
    number = {3},
    pages = {035203},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. B},
    title = {Spatial Density Neural Network Force Fields with First-Principles Level Accuracy and Application to Thermal Transport},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {102}
}

@article{sabbihBiophysicalAnalysisSARSCoV22021a,
    abstract = {Recently, SARS-CoV-2 has been identified as the causative factor of viral infection called COVID-19 that belongs to the zoonotic beta coronavirus family known to cause respiratory disorders or viral pneumonia, followed by an extensive attack on organs that express angiotensin-converting enzyme II (ACE2). Human transmission of this virus occurs via respiratory droplets from symptomatic and asymptomatic patients, which are released into the environment after sneezing or coughing. These droplets are capable of staying in the air as aerosols or surfaces and can be transmitted to persons through inhalation or contact with contaminated surfaces. Thus, there is an urgent need for advanced theranostic solutions to control the spread of COVID-19 infection. The development of such fit-for-purpose technologies hinges on a proper understanding of the transmission, incubation, and structural characteristics of the virus in the external environment and within the host. Hence, this article describes the development of an intrinsic model to describe the incubation characteristics of the virus under varying environmental factors. It also discusses on the evaluation of SARS-CoV-2 structural nucleocapsid protein properties via computational approaches to generate high-affinity binding probes for effective diagnosis and targeted treatment applications by specific targeting of viruses. In addition, this article provides useful insights on the transmission behavior of the virus and creates new opportunities for theranostics development.},
    annotation = {WOS:000587567800001},
    author = {Sabbih, Godfred O. and Korsah, Maame A. and Jeevanandam, Jaison and Danquah, Michael K.},
    date = {2021-03},
    doi = {10.1002/btpr.3096},
    issn = {8756-7938},
    journaltitle = {Biotechnology Progress},
    langid = {english},
    location = {{Hoboken}},
    number = {2},
    pages = {e3096},
    publisher = {{Wiley}},
    shortjournal = {Biotechnol. Prog.},
    title = {Biophysical Analysis of {{SARS}}-{{CoV}}-2 Transmission and Theranostic Development via {{N}} Protein Computational Characterization},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {37}
}

@article{sampathClosingGapModeling2020a,
    abstract = {Molecular self-assembly is a powerful tool in materials design, wherein noncovalent interactions like electrostatic, hydrophobic, hydrogen bonding, and van der Waals can be exploited to produce supramolecular nanostructures that are functional and highly tunable. Biomolecules are attractive building blocks, as they are biocompatible, biodegradable, and adopt a wide array of higher order structures. Moreover, naturally occurring protein systems display a manifold of structures and interactions that can be replicated in synthetic biomolecules. In this perspective, we highlight advances in multiscale simulation techniques across broad spatiotemporal scales that can aid in characterizing self-assembly of hybrid and hierarchical bionanomaterial systems, with an emphasis on physics-based simulation approaches currently employed to study biomolecules at mineral interfaces. The power of these approaches is highlighted across a few recent areas where molecular simulations have advanced our understanding of self-assembly spanning peptides to protein self-assembly. Looking forward, we discuss how in the near future emerging methods in statistical and machine learning will advance this research field in all areas from expanding the capabilities of physics-based simulation methods to enabling new analyses of high throughput experiments. These advances will pave the way for understanding the molecular recognition patterns in systems that are dictated by self-assembly-biomineralizing peptides, hierarchical peptoids, and large protein assemblies-and will aid in the development of a new synthesis science for achieving precise molecular control in materials design.},
    annotation = {WOS:000580949200001},
    author = {Sampath, Janani and Alamdari, Sarah and Pfaendtner, Jim},
    date = {2020-10-13},
    doi = {10.1021/acs.chemmater.0c01891},
    issn = {0897-4756},
    journaltitle = {Chemistry of Materials},
    langid = {english},
    location = {{Washington}},
    number = {19},
    pages = {8043--8059},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {Chem. Mat.},
    title = {Closing the {{Gap Between Modeling}} and {{Experiments}} in the {{Self}}-{{Assembly}} of {{Biomolecules}} at {{Interfaces}} and in {{Solution}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {32}
}

@article{saraceniScalableNeuralNetworks2020,
    abstract = {Supervised machine learning is emerging as a powerful computational tool to predict the properties of complex quantum systems at a limited computational cost. In this article, we quantify how accurately deep neural networks can learn the properties of disordered quantum systems as a function of the system size. We implement a scalable convolutional network that can address arbitrary system sizes. This network is compared with a recently introduced extensive convolutional architecture [Mills, Chem. Sci. 10, 4129 (2019)2041-652010.1039/C8SC04578J] and with conventional dense networks with all-to-all connectivity. The networks are trained to predict the exact ground-state energies of various disordered systems, namely, a continuous-space single-particle Hamiltonian for cold-atoms in speckle disorder, and different setups of a quantum Ising chain with random couplings, including one with only short-range interactions and one augmented with a long-range term. In all testbeds we consider, the scalable network retains high accuracy as the system size increases. Furthermore, we demonstrate that the network scalability enables a transfer-learning protocol, whereby a pretraining performed on small systems drastically accelerates the learning of large-system properties, allowing reaching high accuracy with small training sets. In fact, with the scalable network one can even extrapolate to sizes larger than those included in the training set, accurately reproducing the results of state-of-the-art quantum Monte Carlo simulations.},
    author = {Saraceni, N. and Cantori, S. and Pilati, S.},
    date = {2020-09},
    doi = {10.1103/physreve.102.033301},
    journaltitle = {Physical Review E},
    number = {3},
    publisher = {{American Physical Society}},
    title = {Scalable Neural Networks for the Efficient Learning of Disordered Quantum Systems},
    url = {https://journals.aps.org/pre/abstract/10.1103/PhysRevE.102.033301},
    volume = {102}
}

@article{schererKernelBasedMachineLearning2020,
    abstract = {Current machine learning (ML) models aimed at learning force fields are plagued by their high computational cost at every integration time step. We describe a number of practical and computationally efficient strategies to parametrize traditional force fields for molecular liquids from ML: the particle decomposition ansatz to two- and three-body force fields, the use of kernel-based ML models that incorporate physical symmetries, the incorporation of switching functions close to the cutoff, and the use of covariant meshing to boost the training set size. Results are presented for model molecular liquids: pairwise Lennard-Jones, three-body Stillinger-Weber, and bottom-up coarse-graining of water. Here, covariant meshing proves to be an efficient strategy to learn canonically averaged instantaneous forces. We show that molecular dynamics simulations with tabulated two-and three-body ML potentials are computationally efficient and recover two-and three-body distribution functions. Many-body representations, decomposition, and kernel regression schemes are all implemented in the open-source software package VOTCA.},
    annotation = {WOS:000535226900025},
    author = {Scherer, Christoph and Scheid, Rene and Andrienko, Denis and Bereau, Tristan},
    date = {2020-05-12},
    doi = {10.1021/acs.jctc.9b01256},
    issn = {1549-9618},
    journaltitle = {Journal of Chemical Theory and Computation},
    langid = {english},
    location = {{Washington}},
    number = {5},
    pages = {3194--3204},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem. Theory Comput.},
    title = {Kernel-{{Based Machine Learning}} for {{Efficient Simulations}} of {{Molecular Liquids}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {16}
}

@article{schlederDFTMachineLearning2019,
    abstract = {Recent advances in experimental and computational methods are increasing the quantity and complexity of generated data. This massive amount of raw data needs to be stored and interpreted in order to advance the materials science field. Identifying correlations and patterns from large amounts of complex data is being performed by machine learning algorithms for decades. Recently, the materials science community started to invest in these methodologies to extract knowledge and insights from the accumulated data. This review follows a logical sequence starting from density functional theory as the representative instance of electronic structure methods, to the subsequent high-throughput approach, used to generate large amounts of data. Ultimately, data-driven strategies which include data mining, screening, and machine learning techniques, employ the data generated. We show how these approaches to modern computational materials science are being used to uncover complexities and design novel materials with enhanced properties. Finally, we point to the present research problems, challenges, and potential future perspectives of this new exciting field.},
    annotation = {WOS:000560033900003},
    author = {Schleder, Gabriel R. and Padilha, Antonio C. M. and Acosta, Carlos Mera and Costa, Marcio and Fazzio, Adalberto},
    date = {2019-07-01},
    doi = {10.1088/2515-7639/ab084b},
    journaltitle = {Journal of Physics-Materials},
    langid = {english},
    location = {{Bristol}},
    number = {3},
    pages = {032001},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {J. Phys-Mater.},
    shorttitle = {From {{DFT}} to Machine Learning},
    title = {From {{DFT}} to Machine Learning: Recent Approaches to Materials Science-a Review},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {2}
}

@article{schmidtRecentAdvancesApplications2019a,
    abstract = {One of the most exciting tools that have entered the material science toolbox in recent years is machine learning. This collection of statistical methods has already proved to be capable of considerably speeding up both fundamental and applied research. At present, we are witnessing an explosion of works that develop and apply machine learning to solid-state systems. We provide a comprehensive overview and analysis of the most recent research in this topic. As a starting point, we introduce machine learning principles, algorithms, descriptors, and databases in materials science. We continue with the description of different machine learning approaches for the discovery of stable materials and the prediction of their crystal structure. Then we discuss research in numerous quantitative structure-property relationships and various approaches for the replacement of first-principle methods by machine learning. We review how active learning and surrogate-based optimization can be applied to improve the rational design process and related examples of applications. Two major questions are always the interpretability of and the physical understanding gained from machine learning models. We consider therefore the different facets of interpretability and their importance in materials science. Finally, we propose solutions and future research paths for various challenges in computational materials science.},
    annotation = {WOS:000479252200001},
    author = {Schmidt, Jonathan and Marques, Mario R. G. and Botti, Silvana and Marques, Miguel A. L.},
    date = {2019-08-08},
    doi = {10.1038/s41524-019-0221-0},
    journaltitle = {Npj Computational Materials},
    langid = {english},
    location = {{Berlin}},
    pages = {83},
    publisher = {{Nature Research}},
    shortjournal = {npj Comput. Mater.},
    title = {Recent Advances and Applications of Machine Learning in Solid-State Materials Science},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {5}
}

@article{schranCommitteeNeuralNetwork2020,
    abstract = {It is well known in the field of machine learning that committee models improve accuracy, provide generalization error estimates, and enable active learning strategies. In this work, we adapt these concepts to interatomic potentials based on artificial neural networks. Instead of a single model, multiple models that share the same atomic environment descriptors yield an average that outperforms its individual members as well as a measure of the generalization error in the form of the committee disagreement. We not only use this disagreement to identify the most relevant configurations to build up the model's training set in an active learning procedure but also monitor and bias it during simulations to control the generalization error. This facilitates the adaptive development of committee neural network potentials and their training sets while keeping the number of ab initio calculations to a minimum. To illustrate the benefits of this methodology, we apply it to the development of a committee model for water in the condensed phase. Starting from a single reference ab initio simulation, we use active learning to expand into new state points and to describe the quantum nature of the nuclei. The final model, trained on 814 reference calculations, yields excellent results under a range of conditions, from liquid water at ambient and elevated temperatures and pressures to different phases of ice, and the air-water interface-all including nuclear quantum effects. This approach to committee models will enable the systematic development of robust machine learning models for a broad range of systems.},
    annotation = {WOS:000570950400002},
    author = {Schran, Christoph and Brezina, Krystof and Marsalek, Ondrej},
    date = {2020-09-14},
    doi = {10.1063/5.0016004},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {10},
    pages = {104105},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Committee Neural Network Potentials Control Generalization Errors and Enable Active Learning},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {153}
}

@article{schranTransferabilityMachineLearning2021a,
    abstract = {A previously published neural network potential for the description of protonated water clusters up to the protonated water tetramer, H+(H2O)(4), at an essentially converged coupled cluster accuracy [C. Schran, J. Behler, and D. Marx, J. Chem. Theory Comput. 16, 88 (2020)] is applied to the protonated water hexamer, H+(H2O)(6)-a system that the neural network has never seen before. Although being in the extrapolation regime, it is shown that the potential not only allows for quantum simulations from ultra-low temperatures similar to 1 K up to 300 K but is also able to describe the new system very accurately compared to explicit coupled cluster calculations. This transferability of the model is rationalized by the similarity of the atomic environments encountered for the larger cluster compared to the environments in the training set of the model. Compared to the interpolation regime, the quality of the model is reduced by roughly one order of magnitude, but most of the difference to the coupled cluster reference comes from global shifts of the potential energy surface, while local energy fluctuations are well recovered. These results suggest that the application of neural network potentials in extrapolation regimes can provide useful results and might be more general than usually thought.},
    annotation = {WOS:000630497100001},
    author = {Schran, Christoph and Brieuc, Fabien and Marx, Dominik},
    date = {2021-02-07},
    doi = {10.1063/5.0035438},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {5},
    pages = {051101},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    shorttitle = {Transferability of Machine Learning Potentials},
    title = {Transferability of Machine Learning Potentials: {{Protonated}} Water Neural Network Potential Applied to the Protonated Water Hexamer},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {154}
}

@article{shangAnharmonicRamanSpectra2021,
    abstract = {Raman spectroscopy is an effective tool to analyze the structures of various materials as it provides chemical and compositional information. However, the computation demands for Raman spectra are typically significant because quantum perturbation calculations need to be performed beyond ground state calculations. This work introduces a novel route based on deep neural networks (DNNs) and density-functional perturbation theory to access anharmonic Raman spectra for extended systems. Both the dielectric susceptibility and the potential energy surface are trained using DNNs. The ab initio anharmonic vibrational Raman spectra can be reproduced well with machine learning and DNNs. Silicon and paracetamol crystals are used as showcases to demonstrate the computational efficiency.},
    annotation = {WOS:000630480600005},
    author = {Shang, Honghui and Wang, Haidi},
    date = {2021-03-01},
    doi = {10.1063/5.0040190},
    journaltitle = {Aip Advances},
    langid = {english},
    location = {{Melville}},
    number = {3},
    pages = {035105},
    publisher = {{Amer Inst Physics}},
    shortjournal = {AIP Adv.},
    title = {Anharmonic {{Raman}} Spectra Simulation of Crystals from Deep Neural Networks},
    url = {https://aip.scitation.org/doi/full/10.1063/5.0040190},
    urldate = {2021-08-06},
    volume = {11}
}

@article{shaoModellingBulkElectrolytes2021,
    abstract = {Batteries and supercapacitors are electrochemical energy storage systems which involve multiple time-scales and length-scales. In terms of the electrolyte which serves as the ionic conductor, a molecular-level understanding of the corresponding transport phenomena, electrochemical (thermal) stability and interfacial properties is crucial for optimizing the device performance and achieving safety requirements. To this end, atomistic machine learning is a promising technology for bridging microscopic models and macroscopic phenomena. Here, we provide a timely snapshot of recent advances in this area. This includes technical considerations that are particularly relevant for modelling electrolytes as well as specific examples of both bulk electrolytes and associated interfaces. A perspective on methodological challenges and new applications is also discussed.},
    annotation = {WOS:000604327200001},
    author = {Shao, Yunqi and Knijff, Lisanne and Dietrich, Florian M. and Hermansson, Kersti and Zhang, Chao},
    date = {2021-04},
    doi = {10.1002/batt.202000262},
    journaltitle = {Batteries \& Supercaps},
    langid = {english},
    location = {{Weinheim}},
    number = {4},
    pages = {585--595},
    publisher = {{Wiley-V C H Verlag Gmbh}},
    shortjournal = {Batteries Supercaps},
    title = {Modelling {{Bulk Electrolytes}} and {{Electrolyte Interfaces}} with {{Atomistic Machine Learning}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {4}
}

@article{shaoPiNNPythonLibrary2020a,
    abstract = {Atomic neural networks (ANNs) constitute a class of machine learning methods for predicting potential energy surfaces and physicochemical properties of molecules and materials. Despite many successes, developing interpretable ANN architectures and implementing existing ones efficiently are still challenging. This calls for reliable, general-purpose, and open-source codes. Here, we present a python library named PiNN as a solution toward this goal. In PiNN, we designed a new interpretable and high-performing graph convolutional neural network variant, PiNet, as well as implemented the established Behler-Parrinello neural network. These implementations were tested using datasets of isolated small molecules, crystalline materials, liquid water, and an aqueous alkaline electrolyte. PiNN comes with a visualizer called PiNNBoard to extract chemical insight "learned" by ANNs. It provides analytical stress tensor calculations and interfaces to both the atomic simulation environment and a development version of the Amsterdam Modeling Suite. Moreover, PiNN is highly modularized, which makes it useful not only as a standalone package but also as a chain of tools to develop and to implement novel ANNs. The code is distributed under a permissive BSD license and is freely accessible at https://github.com/Teoroo-CMC/PiNN/with full documentation and tutorials.},
    annotation = {WOS:000526390800014},
    author = {Shao, Yunqi and Hellstrom, Matti and Mitev, Pavlin D. and Knijff, Lisanne and Zhang, Chao},
    date = {2020-03-23},
    doi = {10.1021/acs.jcim.9b00994},
    issn = {1549-9596},
    journaltitle = {Journal of Chemical Information and Modeling},
    langid = {english},
    location = {{Washington}},
    number = {3},
    pages = {1184--1193},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem Inf. Model.},
    shorttitle = {{{PiNN}}},
    title = {{{PiNN}}: {{A Python Library}} for {{Building Atomic Neural Networks}} of {{Molecules}} and {{Materials}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {60}
}

@article{shapeevElinvarEffectBetaTi2020,
    abstract = {A combination of quantum mechanics calculations with machine learning techniques can lead to a paradigm shift in our ability to predict materials properties from first principles. Here we show that on-the-fly training of an interatomic potential described through moment tensors provides the same accuracy as state-of-the-art ab initio molecular dynamics in predicting high-temperature elastic properties of materials with two orders of magnitude less computational effort. Using the technique, we investigate high-temperature bcc phase of titanium and predict very weak, Elinvar, temperature dependence of its elastic moduli, similar to the behavior of the so-called GUM Ti-based alloys (Sato et al 2003 Science 300 464). Given the fact that GUM alloys have complex chemical compositions and operate at room temperature, Elinvar properties of elemental bcc-Ti observed in the wide temperature interval 1100-1700 K is unique.},
    annotation = {WOS:000584988800001},
    author = {Shapeev, Alexander and Podryabinkin, Evgeny and Gubaev, Konstantin and Tasnadi, Ferenc and Abrikosov, Igor A.},
    date = {2020-11},
    doi = {10.1088/1367-2630/abc392},
    issn = {1367-2630},
    journaltitle = {New Journal of Physics},
    langid = {english},
    location = {{Bristol}},
    number = {11},
    pages = {113005},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {New J. Phys.},
    title = {Elinvar Effect in Beta-{{Ti}} Simulated by on-the-Fly Trained Moment Tensor Potential},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {22}
}

@article{shengPFNNPenaltyfreeNeural2021a,
    abstract = {We present PFNN, a penalty-free neural network method, to efficiently solve a class of second-order boundary-value problems on complex geometries. To reduce the smoothness requirement, the original problem is reformulated to a weak form so that the evaluations of high-order derivatives are avoided. Two neural networks, rather than just one, are employed to construct the approximate solution, with one network satisfying the essential boundary conditions and the other handling the rest part of the domain. In this way, an unconstrained optimization problem, instead of a constrained one, is solved without adding any penalty terms. The entanglement of the two networks is eliminated with the help of a length factor function that is scale invariant and can adapt with complex geometries. We prove the convergence of the PFNN method and conduct numerical experiments on a series of linear and nonlinear second-order boundary-value problems to demonstrate that PFNN is superior to several existing approaches in terms of accuracy, flexibility and robustness. (c) 2020 Elsevier Inc. All rights reserved.},
    annotation = {WOS:000612233300005},
    author = {Sheng, Hailong and Yang, Chao},
    date = {2021-03-01},
    doi = {10.1016/j.jcp.2020.110085},
    issn = {0021-9991},
    journaltitle = {Journal of Computational Physics},
    langid = {english},
    location = {{San Diego}},
    pages = {110085},
    publisher = {{Academic Press Inc Elsevier Science}},
    shortjournal = {J. Comput. Phys.},
    shorttitle = {{{PFNN}}},
    title = {{{PFNN}}: {{A}} Penalty-Free Neural Network Method for Solving a Class of Second-Order Boundary-Value Problems on Complex Geometries},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {428}
}

@article{shenQuantumTrajectoryMeanField2019a,
    abstract = {The mixed quantum-classical dynamical approaches have been widely used to study nonadiabatic phenomena in photochemistry and photobiology, in which the time evolutions of the electronic and nuclear subsystems are treated based on quantum and classical mechanics, respectively. The key issue is how to deal with coherence and decoherence during the propagation of the two subsystems, which has been the subject of numerous investigations for a few decades. A brief description on Ehrenfest mean-field and surface-hopping (SH) methods is first provided, and then different algorithms for treatment of quantum decoherence are reviewed in the present paper. More attentions were paid to quantum trajectory meanfield (QTMF) method under the picture of quantum measurements, which is able to overcome the overcoherence problem. Furthermore, the combined QTMF and SH algorithm is proposed in the present work, which takes advantages of the QTMF and SH methods. The potential to extend the applicability of the QTMF method was briefly discussed, such as the generalization to other type of nonadiabatic transitions, the combination with multiscale computational models, and possible improvements on its accuracy and efficiency by using machine-learning techniques.},
    annotation = {WOS:000485829100002},
    author = {Shen, Lin and Tang, Diandong and Xie, Binbin and Fang, Wei-Hai},
    date = {2019-08-29},
    doi = {10.1021/acs.jpca.9b03480},
    issn = {1089-5639},
    journaltitle = {Journal of Physical Chemistry A},
    langid = {english},
    location = {{Washington}},
    number = {34},
    pages = {7337--7350},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. A},
    title = {Quantum {{Trajectory Mean}}-{{Field Method}} for {{Nonadiabatic Dynamics}} in {{Photochemistry}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {123}
}

@article{shimamuraComputationalTrainingRequirements2020a,
    abstract = {We examined the estimation of thermal conductivity through molecular dynamics simulations for a superionic conductor, alpha-Ag2Se, using the interatomic potential based on an artificial neural network (ANN potential). The training data were created using the existing empirical potential of Ag2Se to help find suitable computational and training requirements for the ANN potential, with the intent to apply them to first-principles calculations. The thermal conductivities calculated using different definitions of heat flux were compared, and the effect of explicit long-range Coulomb interaction on the conductivities was investigated. We clarified that using a rigorous heat flux formula for the ANN potential, even for highly ionic alpha-Ag2Se, the resulting thermal conductivity was reasonably consistent with the reference value without explicitly considering Coulomb interaction. It was found that ANN training including the virial term played an important role in reducing the dependency of thermal conductivity on the initial values of the weight parameters of the ANN.},
    annotation = {WOS:000599772600001},
    author = {Shimamura, Kohei and Takeshita, Yusuke and Fukushima, Shogo and Koura, Akihide and Shimojo, Fuyuki},
    date = {2020-12-21},
    doi = {10.1063/5.0027058},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {23},
    pages = {234301},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Computational and Training Requirements for Interatomic Potential Based on Artificial Neural Network for Estimating Low Thermal Conductivity of Silver Chalcogenides},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {153}
}

@article{shimamuraEstimatingThermalConductivity2021,
    abstract = {We clarified computational conditions required to construct artificial neural network (ANN) potentials with a Chebyshev descriptor that accurately estimate thermal conductivity (TC). To this end, training data were generated by an empirical potential; alpha-Ag2Se was selected as the test system and TC was estimated by the homogeneous nonequilibrium molecular dynamics method based on the Green \& ndash;Kubo formula. The ANN potentials trained with first-principles molecular dynamics data were constructed and the TCs were estimated based on the clarified conditions. Since ANN potentials for multicomponent systems are constructed efficiently with the Chebyshev descriptor, accurate TCs for such systems can be relatively easily estimated.},
    annotation = {WOS:000674626800006},
    author = {Shimamura, Kohei and Takeshita, Yusuke and Fukushima, Shogo and Koura, Akihide and Shimojo, Fuyuki},
    date = {2021-09},
    doi = {10.1016/j.cplett.2021.138748},
    issn = {0009-2614},
    journaltitle = {Chemical Physics Letters},
    langid = {english},
    location = {{Amsterdam}},
    pages = {138748},
    publisher = {{Elsevier}},
    shortjournal = {Chem. Phys. Lett.},
    title = {Estimating Thermal Conductivity of Alpha-{{Ag2Se}} Using {{ANN}} Potential with {{Chebyshev}} Descriptor},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {778}
}

@article{shimamuraGuidelinesCreatingArtificial2019,
    abstract = {First-principles molecular dynamics (FPMD) simulations are highly accurate, but due to their high calculation cost, the computational scale is often limited to hundreds of atoms and few picoseconds under specific temperature and pressure conditions. We present here the guidelines for creating artificial neural network empirical interatomic potential (ANN potential) trained with such a limited FPMD data, which can perform long time scale MD simulations at least under the same conditions. The FPMD data for training are prepared on the basis of the convergence of radial distribution function [g(r)]. While training the ANN using total energy and atomic forces of the FPMD data, the error of pressure is also monitored and minimized. To create further robust potential, we add a small amount of FPMD data to reproduce the interaction between two atoms that are close to each other. ANN potentials for alpha-Ag2Se were created as an application example, and it has been confirmed that not only g(r) and mean square displacements but also the specific heat requiring a long time scale simulation matched the FPMD and the experimental values. In addition, the MD simulation using the ANN potential achieved over 10(4) acceleration over the FPMD one. The guidelines proposed here mitigate the creation difficulty of the ANN potential, and a lot of FPMD data sleeping on the hard disk after the research may be put on the front stage again. Published under license by AIP Publishing.},
    annotation = {WOS:000488830300005},
    author = {Shimamura, Kohei and Fukushima, Shogo and Koura, Akihide and Shimojo, Fuyuki and Misawa, Masaaki and Kalia, Rajiv K. and Nakano, Aiichiro and Vashishta, Priya and Matsubara, Takashi and Tanaka, Shigenori},
    date = {2019-09-28},
    doi = {10.1063/1.5116420},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {12},
    pages = {124303},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Guidelines for Creating Artificial Neural Network Empirical Interatomic Potential from First-Principles Molecular Dynamics Data under Specific Conditions and Its Application to Alpha-{{Ag2Se}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {151}
}

@article{sinzWaveletScatteringNetworks2020a,
    abstract = {The dream of machine learning in materials science is for a model to learn the underlying physics of an atomic system, allowing it to move beyond the interpolation of the training set to the prediction of properties that were not present in the original training data. In addition to advances in machine learning architectures and training techniques, achieving this ambitious goal requires a method to convert a 3D atomic system into a feature representation that preserves rotational and translational symmetries, smoothness under small perturbations, and invariance under re-ordering. The atomic orbital wavelet scattering transform preserves these symmetries by construction and has achieved great success as a featurization method for machine learning energy prediction. Both in small molecules and in the bulk amorphous Li alpha Si system, machine learning models using wavelet scattering coefficients as features have demonstrated a comparable accuracy to density functional theory at a small fraction of the computational cost. In this work, we test the generalizability of our Li alpha Si energy predictor to properties that were not included in the training set, such as elastic constants and migration barriers. We demonstrate that statistical feature selection methods can reduce over-fitting and lead to remarkable accuracy in these extrapolation tasks.},
    annotation = {WOS:000565704800004},
    author = {Sinz, Paul and Swift, Michael W. and Brumwell, Xavier and Liu, Jialin and Kim, Kwang Jin and Qi, Yue and Hirn, Matthew},
    date = {2020-08-28},
    doi = {10.1063/5.0016020},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {8},
    pages = {084109},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Wavelet Scattering Networks for Atomistic Systems with Extrapolation of Material Properties},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {153}
}

@article{sugisawaGaussianProcessModel2020a,
    abstract = {The goal of the present work is to obtain accurate potential energy surfaces (PESs) for high-dimensional molecular systems with a small number of ab initio calculations in a system-agnostic way. We use probabilistic modeling based on Gaussian processes (GPs). We illustrate that it is possible to build an accurate GP model of a 51-dimensional PES based on 5000 randomly distributed ab initio calculations with a global accuracy of {$<$}0.2 kcal/mol. Our approach uses GP models with composite kernels designed to enhance the Bayesian information content and represents the global PES as a sum of a full-dimensional GP and several GP models for molecular fragments of lower dimensionality. We demonstrate the potency of these algorithms by constructing the global PES for the protonated imidazole dimer, a molecular system with 19 atoms. We illustrate that GP models thus constructed can extrapolate the PES from low energies ({$<$}10 000 cm(-1)), yielding a PES at high energies ({$>$}20 000 cm(-1)). This opens the prospect for new applications of GPs, such as mapping out phase transitions by extrapolation or accelerating Bayesian optimization, for high-dimensional physics and chemistry problems with a restricted number of inputs, i.e., for high-dimensional problems where obtaining training data is very difficult.},
    annotation = {WOS:000573433800001},
    author = {Sugisawa, Hiroki and Ida, Tomonori and Krems, R.},
    date = {2020-09-21},
    doi = {10.1063/5.0023492},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {11},
    pages = {114101},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Gaussian Process Model of 51-Dimensional Potential Energy Surface for Protonated Imidazole Dimer},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {153}
}

@article{tangDevelopmentInteratomicPotential2020,
    abstract = {An interatomic potential for the Al-Tb alloy around the composition of Al(90)Tb(10)is developed using the deep neural network (DNN) learning method. The atomic configurations and the corresponding total potential energies and forces on each atom obtained fromab initiomolecular dynamics (AIMD) simulations are collected to train a DNN model to construct the interatomic potential for the Al-Tb alloy. We show that the obtained DNN model can well reproduce the energies and forces calculated by AIMD simulations. Molecular dynamics (MD) simulations using the DNN interatomic potential also accurately describe the structural properties of the Al(90)Tb(10)liquid, such as partial pair correlation functions (PPCFs) and bond angle distributions, in comparison with the results from AIMD simulations. Furthermore, the developed DNN interatomic potential predicts the formation energies of the crystalline phases of the Al-Tb system with an accuracy comparable toab initiocalculations. The structure factors of the Al(90)Tb(10)metallic liquid and glass obtained by MD simulations using the developed DNN interatomic potential are also in good agreement with the experimental X-ray diffraction data. The development of short-range order (SRO) in the Al(90)Tb(10)liquid and the undercooled liquid is also analyzed and three dominant SROs,i.e., Al-centered distorted icosahedron (DISICO) and Tb-centered '3661' and '15551' clusters, respectively, are identified.},
    annotation = {WOS:000565157900018},
    author = {Tang, L. and Yang, Z. J. and Wen, T. Q. and Ho, K. M. and Kramer, M. J. and Wang, C. Z.},
    date = {2020-09-07},
    doi = {10.1039/d0cp01689f},
    issn = {1463-9076},
    journaltitle = {Physical Chemistry Chemical Physics},
    langid = {english},
    location = {{Cambridge}},
    number = {33},
    pages = {18467--18479},
    publisher = {{Royal Soc Chemistry}},
    shortjournal = {Phys. Chem. Chem. Phys.},
    title = {Development of Interatomic Potential for {{Al}}-{{Tb}} Alloys Using a Deep Neural Network Learning Method},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {22}
}

@article{tangShortMediumrangeOrders2021,
    abstract = {Molecular dynamics simulations using an interatomic potential developed by artificial neural network deep machine learning are performed to study the local structural order in Al90Tb10 metallic glass. We show that more than 80\% of the Tb-centered clusters in Al90Tb10 glass have short-range order (SRO) with their 17 first coordination shell atoms stacked in a '3661' or '15551' sequence. Medium-range order (MRO) in Bergman-type packing extended out to the second and third coordination shells is also clearly observed. Analysis of the network formed by the '3661' and '15551' clusters show that similar to 82\% of such SRO units share their faces or vertexes, while only similar to 6\% of neighboring SRO pairs are interpenetrating. Such a network topology is consistent with the Bergman-type MRO around the Tb-centers. Moreover, crystal structure searches using genetic algorithm and the neural network interatomic potential reveal several low-energy metastable crystalline structures in the composition range close to Al90Tb10. Some of these crystalline structures have the '3661' SRO while others have the '15551' SRO. While the crystalline structures with the '3661' SRO also exhibit the MRO very similar to that observed in the glass, the ones with the '15551' SRO have very different atomic packing in the second and third shells around the Tb centers from that of the Bergman-type MRO observed in the glassy phase. (c) 2020 Acta Materialia Inc. Published by Elsevier Ltd. All rights reserved.},
    annotation = {WOS:000603994900021},
    author = {Tang, L. and Yang, Z. J. and Wen, T. Q. and Ho, K. M. and Kramer, M. J. and Wang, C. Z.},
    date = {2021-02-01},
    doi = {10.1016/j.actamat.2020.116513},
    issn = {1359-6454},
    journaltitle = {Acta Materialia},
    langid = {english},
    location = {{Oxford}},
    pages = {116513},
    publisher = {{Pergamon-Elsevier Science Ltd}},
    shortjournal = {Acta Mater.},
    title = {Short- and Medium-Range Orders in {{Al90Tb10}} Glass and Their Relation to the Structures of Competing Crystalline Phases},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {204}
}

@article{thurstonMachineLearningMolecular2018,
    abstract = {Self-assembling oligopeptides present a means to fabricate biocompatible supramolecular aggregates with engineered electronic and optical functionality. We conducted molecular dynamics simulations of self-assembling synthetic oligopeptides with Asp-X cores that mediate hydrophobic stacking and electron delocalisation within the self-assembled nanostructure. The larger PDI cores elevated oligomerisation free energies by a factor of 2-3 relative to NDI and also improved alignment of the oligopeptides within the stack. Training of a quantitative structure-property relationship (QSPR) model revealed key physicochemical determinants of the oligomerisation free energies and produced a predictive model for the oligomerisation thermodynamics. Oligopeptides with moderate dimerisation and trimerisation free energies of produced aggregates with the best in-register parallel stacking, and we used this criterion within our QSPR model to perform high-throughput virtual screening to identify promising candidates for the spontaneous assembly of ordered nanoaggregates. We identified a small number of oligopeptide candidates for direct testing in large scale molecular simulations, and predict a novel chemistry DAVG-PDI-GVAD previously unstudied by experiment or simulation to produce well-aligned nanoaggregates expected to possess good optical and electronic functionality.},
    annotation = {WOS:000433319500007},
    author = {Thurston, Bryce A. and Ferguson, Andrew L.},
    date = {2018},
    doi = {10.1080/08927022.2018.1469754},
    issn = {0892-7022},
    journaltitle = {Molecular Simulation},
    langid = {english},
    location = {{Abingdon}},
    number = {11},
    pages = {930--945},
    publisher = {{Taylor \& Francis Ltd}},
    shortjournal = {Mol. Simul.},
    title = {Machine Learning and Molecular Design of Self-Assembling Pi-Conjugated Oligopeptides},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/6},
    urldate = {2021-08-06},
    volume = {44}
}

@article{tongCombiningMachineLearning2020,
    abstract = {The theoretical structure prediction method via quantum mechanical atomistic simulations such as density functional theory (DFT), based solely on chemical composition, has already become a routine tool to determine the structures of physical and chemical systems, e.g., solids and clusters. However, the application of DFT to more realistic simulations, to a large extent, is impeded because of the unfavorable scaling of the computational cost with respect to the system size. During recent years, the machine learning potential (MLP) method has been rapidly rising as an accurate and efficient tool for atomistic simulations. In this Perspective, we provide an introduction to the basic principles and advantages of the combination of structure prediction and MLP, as well as the challenges and opportunities associated with this promising approach.},
    author = {Tong, Qunchao and Gao, Pengyue and Liu, Hanyu and Xie, Yu and Lv, Jian and Wang, Yanchao and Zhao, Jijun},
    date = {2020-10},
    doi = {10.1021/acs.jpclett.0c02357},
    journaltitle = {Journal of Physical Chemistry Letters},
    number = {20},
    pages = {8710--8720},
    publisher = {{American Chemical Society}},
    title = {Combining Machine Learning Potential and Structure Prediction for Accelerated Materials Design and Discovery},
    url = {https://pubs.acs.org/doi/abs/10.1021/acs.jpclett.0c02357},
    volume = {11}
}

@article{tongMachineLearningMetadynamics2021a,
    abstract = {Simulating reconstructive phase transition requires an accurate description of potential energy surface (PES). Density-functional-theory (DFT) based molecular dynamics can achieve the desired accuracy but it is computationally unfeasible for large systems and/or long simulation times. Here we introduce an approach that combines the metadynamics simulation and machine learning representation of PES at the accuracy close to the DFT calculations, but with the computational cost several orders of magnitude less, and scaling with system size approximately linear. The high accuracy of the method is demonstrated in the simulation of pressure-induced B4-B1 phase transition in gallium nitride (GaN). The large-scale simulation using a 4096-atom simulation box reveals the phase transition with excellent detail, revealing different simulated transition paths under particular stress conditions. With well-trained machine learning potentials, this method can be easily applied to all types of systems for accurate scalable simulations of solid-solid reconstructive phase transition.},
    annotation = {WOS:000616409400001},
    author = {Tong, Qunchao and Luo, Xiaoshan and Adeleke, Adebayo A. and Gao, Pengyue and Xie, Yu and Liu, Hanyu and Li, Quan and Wang, Yanchao and Lv, Jian and Yao, Yansun and Ma, Yanming},
    date = {2021-02-09},
    doi = {10.1103/PhysRevB.103.054107},
    issn = {2469-9950},
    journaltitle = {Physical Review B},
    langid = {english},
    location = {{College Pk}},
    number = {5},
    pages = {054107},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. B},
    title = {Machine Learning Metadynamics Simulation of Reconstructive Phase Transition},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {103}
}

@article{tuoMachineLearningBased2020,
    abstract = {A Machine-Learning based Deep Potential (DP) model for Al clusters is developed through training with an extended database including ab initio data of both bulk and several clusters in only 6 CPU/h. This DP model has good performance in accurately predicting the low-lying candidates of Al clusters in a broad size range. Based on our developed DP model, the low-lying structures of 101 different sized Al clusters are extensively searched, among which the lowest-energy candidates of 69 sized clusters are updated. Our calculations demonstrate that machine-learning is indeed powerful in generating potentials to describe the interaction of atoms in complex materials. Published under license by AIP Publishing.},
    annotation = {WOS:000521360300001},
    author = {Tuo, P. and Ye, X. B. and Pan, B. C.},
    date = {2020-03-21},
    doi = {10.1063/5.0001491},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {11},
    pages = {114105},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {A Machine Learning Based Deep Potential for Seeking the Low-Lying Candidates of {{Al}} Clusters},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {152}
}

@article{unkePhysNetNeuralNetwork2019a,
    abstract = {In recent years, machine learning (ML) methods have become increasingly popular in computational chemistry. After being trained on appropriate ab initio reference data, these methods allow for accurately predicting the properties of chemical systems, circumventing the need for explicitly solving the electronic Schrodinger equation. Because of their computational efficiency and scalability to large data sets, deep neural networks (DNNs) are a particularly promising ML algorithm for chemical applications. This work introduces PhysNet, a DNN architecture designed for predicting energies, forces, and dipole moments of chemical systems. PhysNet achieves stateof-the-art performance on the QM9, MD17, and ISO17 benchmarks. Further, two new data sets are generated in order to probe the performance of ML models for describing chemical reactions, long-range interactions, and condensed phase systems. It is shown that explicitly including electrostatics in energy predictions is crucial for a qualitatively correct description of the asymptotic regions of a potential energy surface (PES). PhysNet models trained on a systematically constructed set of small peptide fragments (at most eight heavy atoms) are able to generalize to considerably larger proteins like deca-alanine (Ala(10)): The optimized geometry of helical Ala10 predicted by PhysNet is virtually identical to ab initio results (RMSD = 0.21 A). By running unbiased molecular dynamics (MD) simulations of Ala(10) on the PhysNet-PES in gas phase, it is found that instead of a helical structure, Ala(10) folds into a "wreath-shaped" configuration, which is more stable than the helical form by 0.46 kcal mol(-1) according to the reference ab initio calculations.},
    annotation = {WOS:000471728500021},
    author = {Unke, Oliver T. and Meuwly, Markus},
    date = {2019-06},
    doi = {10.1021/acs.jctc.9b00181},
    issn = {1549-9618},
    journaltitle = {Journal of Chemical Theory and Computation},
    langid = {english},
    location = {{Washington}},
    number = {6},
    pages = {3678--3693},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Chem. Theory Comput.},
    shorttitle = {{{PhysNet}}},
    title = {{{PhysNet}}: {{A Neural Network}} for {{Predicting Energies}}, {{Forces}}, {{Dipole Moments}}, and {{Partial Charges}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {15}
}

@article{vandermauseOntheflyActiveLearning2020,
    abstract = {Machine learned force fields typically require manual construction of training sets consisting of thousands of first principles calculations, which can result in low training efficiency and unpredictable errors when applied to structures not represented in the training set of the model. This severely limits the practical application of these models in systems with dynamics governed by important rare events, such as chemical reactions and diffusion. We present an adaptive Bayesian inference method for automating the training of interpretable, low-dimensional, and multi-element interatomic force fields using structures drawn on the fly from molecular dynamics simulations. Within an active learning framework, the internal uncertainty of a Gaussian process regression model is used to decide whether to accept the model prediction or to perform a first principles calculation to augment the training set of the model. The method is applied to a range of single- and multi-element systems and shown to achieve a favorable balance of accuracy and computational efficiency, while requiring a minimal amount of ab initio training data. We provide a fully open-source implementation of our method, as well as a procedure to map trained models to computationally efficient tabulated force fields.},
    annotation = {WOS:000520616900001},
    author = {Vandermause, Jonathan and Torrisi, Steven B. and Batzner, Simon and Xie, Yu and Sun, Lixin and Kolpak, Alexie M. and Kozinsky, Boris},
    date = {2020-03-18},
    doi = {10.1038/s41524-020-0283-z},
    journaltitle = {Npj Computational Materials},
    langid = {english},
    location = {{London}},
    number = {1},
    pages = {20},
    publisher = {{Nature Publishing Group}},
    shortjournal = {npj Comput. Mater.},
    title = {On-the-Fly Active Learning of Interpretable {{Bayesian}} Force Fields for Atomistic Rare Events},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {6}
}

@article{vandermauseOntheflyBayesianActive2019,
    abstract = {Machine learning based interatomic potentials currently require manual construction of training sets consisting of thousands of first principles calculations and are often restricted to single-component and nonreactive systems. This severely limits the practical application of these models due to both low training efficiency and limited accuracy in treating important rare events such as reactions and diffusion. We present an adaptive Bayesian inference method for automating and accelerating the on-the-fly construction of accurate interatomic force fields using structures drawn from molecular dynamics simulations. Within an online active learning algorithm, the internal uncertainty of a Gaussian process regression model is used to decide whether to accept the model prediction or to perform a first principles calculation to augment the training set of the model. The method is applied to a range of single-and multi-component systems and shown to achieve state-of-the-art accuracy with minimal ab initio data.},
    archiveprefix = {arXiv},
    arxivid = {1904.02042v2},
    author = {Vandermause, J and Torrisi, SB and Batzner, S},
    date = {2019},
    eprint = {1904.02042v2},
    eprinttype = {arxiv},
    journaltitle = {projects.iq.harvard.edu},
    title = {On-the-Fly {{Bayesian}} Active Learning of Interpretable Force-Fields for Atomistic Rare Events},
    url = {https://projects.iq.harvard.edu/files/bkoz/files/2_vandermause_et_al._-_2019_-_on-the-fly_bayesian_active_learning_of_interpretable_force-fields_for_atomistic_rare_events.pdf}
}

@article{vassilev-galindoChallengesMachineLearning2021a,
    abstract = {Dynamics of flexible molecules are often determined by an interplay between local chemical bond fluctuations and conformational changes driven by long-range electrostatics and van der Waals interactions. This interplay between interactions yields complex potential-energy surfaces (PESs) with multiple minima and transition paths between them. In this work, we assess the performance of the state-of-the-art Machine Learning (ML) models, namely, sGDML, SchNet, Gaussian Approximation Potentials/Smooth Overlap of Atomic Positions (GAPs/SOAPs), and Behler-Parrinello neural networks, for reproducing such PESs, while using limited amounts of reference data. As a benchmark, we use the cis to trans thermal relaxation in an azobenzene molecule, where at least three different transition mechanisms should be considered. Although GAP/SOAP, SchNet, and sGDML models can globally achieve a chemical accuracy of 1 kcal mol(-1) with fewer than 1000 training points, predictions greatly depend on the ML method used and on the local region of the PES being sampled. Within a given ML method, large differences can be found between predictions of close-to-equilibrium and transition regions, as well as for different transition mechanisms. We identify key challenges that the ML models face mainly due to the intrinsic limitations of commonly used atom-based descriptors. All in all, our results suggest switching from learning the entire PES within a single model to using multiple local models with optimized descriptors, training sets, and architectures for different parts of the complex PES.},
    annotation = {WOS:000630524000019},
    author = {Vassilev-Galindo, Valentin and Fonseca, Gregory and Poltavsky, Igor and Tkatchenko, Alexandre},
    date = {2021-03-07},
    doi = {10.1063/5.0038516},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {9},
    pages = {094119},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Challenges for Machine Learning Force Fields in Reproducing Potential Energy Surfaces of Flexible Molecules},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {154}
}

@article{vergadouMolecularModelingInvestigations2019a,
    abstract = {With a wide range of applications, from energy and environmental engineering, such as in gas separations and water purification, to biomedical engineering and packaging, glassy polymeric materials remain in the core of novel membrane and state-of the art barrier technologies. This review focuses on molecular simulation methodologies implemented for the study of sorption and diffusion of small molecules in dense glassy polymeric systems. Basic concepts are introduced and systematic methods for the generation of realistic polymer configurations are briefly presented. Challenges related to the long length and time scale phenomena that govern the permeation process in the glassy polymer matrix are described and molecular simulation approaches developed to address the multiscale problem at hand are discussed.},
    annotation = {WOS:000482952800009},
    author = {Vergadou, Niki and Theodorou, Doros N.},
    date = {2019-08},
    doi = {10.3390/membranes9080098},
    journaltitle = {Membranes},
    langid = {english},
    location = {{Basel}},
    number = {8},
    pages = {98},
    publisher = {{Mdpi}},
    shortjournal = {Membranes},
    title = {Molecular {{Modeling Investigations}} of {{Sorption}} and {{Diffusion}} of {{Small Molecules}} in {{Glassy Polymers}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {9}
}

@article{vinsonFasterExactExchange2020a,
    abstract = {Density-functional theory simplifies many-electron calculations by approximating the exchange and correlation interactions with a one-electron operator that is a functional of the density. Hybrid functionals incorporate some amount of exact exchange, improving agreement with measured electronic and structural properties. However, calculations with hybrid functionals require substantial computational resources, limiting their use. By calculating the exchange interaction of periodic systems with single-precision arithmetic, the computation time is cut nearly in half with a negligible loss in accuracy. This improvement makes exact exchange calculations quicker and more feasible, especially for high-throughput calculations. Example hybrid density-functional theory calculations of band energies, forces, and x-ray absorption spectra show that this single-precision implementation maintains accuracy with significantly reduced runtime and memory requirements.},
    annotation = {WOS:000595858100002},
    author = {Vinson, John},
    date = {2020-11-28},
    doi = {10.1063/5.0030493},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {20},
    pages = {204106},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Faster Exact Exchange in Periodic Systems Using Single-Precision Arithmetic},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {153}
}

@article{wangCombiningFragmentationApproach2020a,
    abstract = {Accurate and efficient all-atom quantum mechanical (QM) calculations for biomolecules still present a challenge to computational physicists and chemists. In this study, an extensible generalized molecular fractionation with a conjugate caps method combined with neural networks (NN-GMFCC) is developed for efficient QM calculation of protein energy. In the NN-GMFCC scheme, the total energy of a given protein is calculated by taking a proper combination of the high-precision neural network potential energies of all capped residues and overlapping conjugate caps. In addition, the two-body interaction energies of residue pairs are calculated by molecular mechanics (MM). With reference to the GMFCC/MM calculation at the.B97XD/6-31G* level, the overall mean unsigned errors of the energy deviations and atomic force root-mean-squared errors calculated by NN-GMFCC are only 2.01 kcal/mol and 0.68 kcal/mol/A, respectively, for 14 proteins (containing up to 13,728 atoms). Meanwhile, the NN-GMFCC approach is about 4 orders of magnitude faster than the GMFCC/MM method. The NN-GMFCC method could be systematically improved by inclusion of two-body QM interaction and multibody electronic polarization effect. Moreover, the NN-GMFCC approach can also be applied to other macromolecular systems such as DNA/RNA, and it is capable of providing a powerful and efficient approach for exploration of structures and functions of proteins with QM accuracy.},
    annotation = {WOS:000526368900006},
    author = {Wang, Zhilong and Han, Yanqiang and Li, Jinjin and He, Xiao},
    date = {2020-04-16},
    doi = {10.1021/acs.jpcb.0c01370},
    issn = {1520-6106},
    journaltitle = {Journal of Physical Chemistry B},
    langid = {english},
    location = {{Washington}},
    number = {15},
    pages = {3027--3035},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. B},
    title = {Combining the {{Fragmentation Approach}} and {{Neural Network Potential Energy Surfaces}} of {{Fragments}} for {{Accurate Calculation}} of {{Protein Energy}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {124}
}

@article{wangElectronicallyDriven1D2021a,
    abstract = {Atomic diffusion is a spontaneous process and significantly influences properties of materials, such as fracture toughness, creep-fatigue properties, thermal conductivity, thermoelectric properties, etc. Here, using extensive molecular dynamics simulations based on both ab initio and machine-learning potentials, we demonstrate that an atomic one dimensional cooperative diffusion exists in the simple cubic high-pressure finite-temperature phase of calcium in the premelting regime, where some atoms diffuse cooperatively as chains or even rings, while others remain in the solid state. This intermediate regime is triggered by anharmonicity of the system at high temperature and is stabilized by the competition between the internal energy minimization and the entropy maximization, and has close connections with the unique electronic structures of simple cubic Ca as an electride with a pseudogap. This cooperative diffusion regime explains the abnormal enhancement of the melting line of Ca under high pressure and suggests that the cooperative chain melting is a much more common high-temperature feature among metals under extreme conditions than hitherto thought. The microscopic electronic investigations of these systems combining ab initio and machine-learning data point out the direction for further understanding of other metallic systems such as the glass transition, liquid metals, etc.},
    annotation = {WOS:000606733200001},
    author = {Wang, Yong and Wang, Junjie and Hermann, Andreas and Liu, Cong and Gao, Hao and Tosatti, Erio and Wang, Hui-Tian and Xing, Dingyu and Sun, Jian},
    date = {2021-01-11},
    doi = {10.1103/PhysRevX.11.011006},
    issn = {2160-3308},
    journaltitle = {Physical Review X},
    langid = {english},
    location = {{College Pk}},
    number = {1},
    pages = {011006},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. X},
    title = {Electronically {{Driven 1D Cooperative Diffusion}} in a {{Simple Cubic Crystal}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {11}
}

@article{wangEnsembleLearningCoarsegrained2020a,
    abstract = {Gradient-domain machine learning (GDML) is an accurate and efficient approach to learn a molecular potential and associated force field based on the kernel ridge regression algorithm. Here, we demonstrate its application to learn an effective coarse-grained (CG) model from all-atom simulation data in a sample efficient manner. The CG force field is learned by following the thermodynamic consistency principle, here by minimizing the error between the predicted CG force and the all-atom mean force in the CG coordinates. Solving this problem by GDML directly is impossible because coarse-graining requires averaging over many training data points, resulting in impractical memory requirements for storing the kernel matrices. In this work, we propose a data-efficient and memory-saving alternative. Using ensemble learning and stratified sampling, we propose a 2-layer training scheme that enables GDML to learn an effective CG model. We illustrate our method on a simple biomolecular system, alanine dipeptide, by reconstructing the free energy landscape of a CG variant of this molecule. Our novel GDML training scheme yields a smaller free energy error than neural networks when the training set is small, and a comparably high accuracy when the training set is sufficiently large.},
    annotation = {WOS:000536701800001},
    author = {Wang, Jiang and Chmiela, Stefan and Mueller, Klaus-Robert and Noe, Frank and Clementi, Cecilia},
    date = {2020-05-21},
    doi = {10.1063/5.0007276},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {19},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Ensemble Learning of Coarse-Grained Molecular Dynamics Force Fields with a Kernel Approach},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {152}
}

@article{wangExtendibleGraphneuralnetworkbasedApproach2021,
    abstract = {An accurate force field is the key to the success of all molecular mechanics simulations on organic polymers and biomolecules. Accuracy beyond density functional theory is often needed to describe the intermolecular interactions, while most correlated wavefunction (CW) methods are prohibitively expensive for large molecules. Therefore, it posts a great challenge to develop an extendible ab initio force field for large flexible organic molecules at CW level of accuracy. In this work, we face this challenge by combining the physics-driven nonbonding potential with a data-driven subgraph neural network bonding model (named sGNN). Tests on polyethylene glycol polymer chains show that our strategy is highly accurate and robust for molecules of different sizes. Therefore, we can develop the force field from small molecular fragments (with sizes easily accessible to CW methods) and safely transfer it to large polymers, thus opening a new path to the next-generation organic force fields.},
    archiveprefix = {arXiv},
    arxivid = {2106.00927},
    author = {Wang, Xufei and Xu, Yuanda and Zheng, Han and Yu, Kuang},
    date = {2021-06},
    eprint = {2106.00927},
    eprinttype = {arxiv},
    journaltitle = {arxiv.org},
    title = {An Extendible, Graph-Neural-Network-Based Approach for Accurate Force Field Development of Large Flexible Organic Molecules},
    url = {https://arxiv.org/abs/2106.00927}
}

@article{wangInvestigationsWaterOxide,
    abstract = {Water/oxide interfaces are ubiquitous on earth and show significant influence on many chemical processes. For example, understanding water and solute adsorption as well as catalytic water splitting can help build better fuel cells and solar cells to overcome our looming energy crisis; the interaction between biomolecules and water/oxide interfaces is one hypothesis to explain the origin of life. However, knowledge in this area is still limited due to the difficulty of studying water/solid interfaces. As a result, research using increasingly sophisticated experimental techniques and computational simulations has been carried out in recent years. Although it is difficult for experimental techniques to provide detailed microscopic structural information, molecular dynamics (MD) simulations have satisfactory performance. In this review, we discuss classical and ab initio MD simulations of water/oxide interfaces. Generally, we are interested in the following questions: How do solid surfaces perturb interfacial water structure? How do interfacial water molecules and adsorbed solutes affect solid surfaces and how do interfacial environments affect solvent and solute behavior? Finally, we discuss progress in the application of neural network potential based MD simulations, which offer a promising future because this approach has already enabled ab initio level accuracy for very large systems and long trajectories. This article is categorized under: Theoretical and Physical Chemistry {$>$} Spectroscopy Molecular and Statistical Mechanics {$>$} Molecular Interactions Structure and Mechanism {$>$} Molecular Structures},
    annotation = {WOS:000657137500001},
    author = {Wang, Ruiyu and Klein, Michael L. and Carnevale, Vincenzo and Borguet, Eric},
    doi = {10.1002/wcms.1537},
    issn = {1759-0876},
    journaltitle = {Wiley Interdisciplinary Reviews-Computational Molecular Science},
    langid = {english},
    location = {{Hoboken}},
    pages = {e1537},
    publisher = {{Wiley}},
    shortjournal = {Wiley Interdiscip. Rev.-Comput. Mol. Sci.},
    title = {Investigations of Water/Oxide Interfaces by Molecular Dynamics Simulations},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06}
}

@article{wangMachineLearningCoarseGrained2019,
    abstract = {Atomistic or ab initio molecular dynamics simulations are widely used to predict thermodynamics and kinetics and relate them to molecular structure. A common approach to go beyond the time- and length-scales accessible with such computationally expensive simulations is the definition of coarse-grained molecular models. Existing coarse-graining approaches define an effective interaction potential to match defined properties of high-resolution models or experimental data. In this paper, we reformulate coarse-graining as a supervised machine learning problem. We use statistical learning theory to decompose the coarse-graining error and cross-validation to select and compare the performance of different models. We introduce CGnets, a deep learning approach, that learns coarse-grained free energy functions and can be trained by a force-matching scheme. CGnets maintain all physically relevant invariances and allow one to incorporate prior physics knowledge to avoid sampling of unphysical structures. We show that CGnets can capture all-atom explicit-solvent free energy surfaces with models using only a few coarse-grained beads and no solvent, while classical coarse-graining methods fail to capture crucial features of the free energy surface. Thus, CGnets are able to capture multibody terms that emerge from the dimensionality reduction.},
    annotation = {WOS:000468624100008},
    author = {Wang, Jiang and Olsson, Simon and Wehmeyer, Christoph and Perez, Adria and Charron, Nicholas E. and de Fabritiis, Gianni and Noe, Frank and Clementi, Cecilia},
    date = {2019-05-22},
    doi = {10.1021/acscentsci.8b00913},
    issn = {2374-7943},
    journaltitle = {Acs Central Science},
    langid = {english},
    location = {{Washington}},
    number = {5},
    options = {useprefix=true},
    pages = {755--767},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {ACS Central Sci.},
    title = {Machine {{Learning}} of {{Coarse}}-{{Grained Molecular Dynamics Force Fields}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {5}
}

@article{wangMultibodyEffectsCoarsegrained2021,
    abstract = {The use of coarse-grained (CG) models is a popular approach to study complex biomolecular systems. By reducing the number of degrees of freedom, a CG model can explore long time- and length-scales inaccessible to computational models at higher resolution. If a CG model is designed by formally integrating out some of the system's degrees of freedom, one expects multi-body interactions to emerge in the effective CG model's energy function. In practice, it has been shown that the inclusion of multi-body terms indeed improves the accuracy of a CG model. However, no general approach has been proposed to systematically construct a CG effective energy that includes arbitrary orders of multi-body terms. In this work, we propose a neural network based approach to address this point and construct a CG model as a multi-body expansion. By applying this approach to a small protein, we evaluate the relative importance of the different multi-body terms in the definition of an accurate model. We observe a slow convergence in the multi-body expansion, where up to five-body interactions are needed to reproduce the free energy of an atomistic model.},
    author = {Wang, Jiang and Charron, Nicholas and Husic, Brooke and Olsson, Simon and Noé, Frank and Clementi, Cecilia},
    date = {2021-04},
    doi = {10.1063/5.0041022},
    journaltitle = {Journal of Chemical Physics},
    number = {16},
    publisher = {{American Institute of Physics Inc.}},
    title = {Multi-Body Effects in a Coarse-Grained Protein Force Field},
    url = {https://aip.scitation.org/doi/abs/10.1063/5.0041022},
    volume = {154}
}

@article{wangPredictingAdsorptionAbility2021,
    abstract = {Accurately evaluating the adsorption ability of adsorbents for heavy metal ions (HMIs) and organic pollutants in water is critical for the design and preparation of emerging highly efficient adsorbents. However, predicting adsorption capabilities of adsorbents at arbitrary sites is challenging, with currently unavailable measuring technology for active sites and the corresponding activities. Here, we present an efficient artificial intelligence (AI) approach to predict the adsorption ability of adsorbents at arbitrary sites, as a case study of three HMIs (Pb(II), Hg(II), and Cd(II)) adsorbed on the surface of a representative two-dimensional graphitic-C3N4. We apply the deep neural network and transfer learning to predict the adsorption capabilities of three HMIs at arbitrary sites, with the predicted results of Cd(II){$>$}Hg(II){$>$}Pb(II) and the root-mean-squared errors less than 0.1eV. The proposed AI method has the same prediction accuracy as the ab initio DFT calculation, but is millions of times faster than the DFT to predict adsorption abilities at arbitrary sites and only requires one-tenth of datasets compared to training from scratch. We further verify the adsorption capacity of g-C3N4 towards HMIs experimentally and obtain results consistent with the AI prediction. It indicates that the presented approach is capable of evaluating the adsorption ability of adsorbents efficiently, and can be further extended to other interdisciplines and industries for the adsorption of harmful elements in aqueous solution.},
    annotation = {WOS:000616403300003},
    author = {Wang, Zhilong and Zhang, Haikuo and Ren, Jiahao and Lin, Xirong and Han, Tianli and Liu, Jinyun and Li, Jinjin},
    date = {2021-01-29},
    doi = {10.1038/s41524-021-00494-9},
    journaltitle = {Npj Computational Materials},
    langid = {english},
    location = {{Berlin}},
    number = {1},
    pages = {19},
    publisher = {{Nature Research}},
    shortjournal = {npj Comput. Mater.},
    title = {Predicting Adsorption Ability of Adsorbents at Arbitrary Sites for Pollutants Using Deep Transfer Learning},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {7}
}

@article{wangSymmetryadaptedGraphNeural2021,
    abstract = {Molecular dynamics is a powerful simulation tool to explore material properties. Most of the realistic material systems are too large to be simulated with first-principles molecular dynamics. Classical molecular dynamics has lower computational cost but requires accurate force fields to achieve chemical accuracy. In this work, we develop a symmetry-adapted graph neural networks framework, named molecular dynamics graph neural networks (MDGNN), to construct force fields automatically for molecular dynamics simulations for both molecules and crystals. This architecture consistently preserves the translation, rotation and permutation invariance in the simulations. We propose a new feature engineering method including higher order contributions and show that MDGNN accurately reproduces the results of both classical and first-principles molecular dynamics. We also demonstrate that force fields constructed by the model has good transferability. Therefore, MDGNN provides an efficient and promising option for molecular dynamics simulations of large scale systems with high accuracy.},
    archiveprefix = {arXiv},
    author = {Wang, Zun and Wang, Chong and Zhao, Sibo and Du, Shiqiao and Xu, Yong and Gu, Bing-Lin and Duan, Wenhui},
    date = {2021},
    eprint = {2101.02930},
    eprinttype = {arxiv},
    title = {Symmetry-Adapted Graph Neural Networks for Constructing Molecular Dynamics Force Fields},
    url = {https://arxiv.org/abs/2101.02930}
}

@article{weinreichPropertiesAlphaBrassNanoparticles2020,
    abstract = {Binary metal clusters are of high interest for applications in heterogeneous catalysis and have received much attention in recent years. To gain insights into their structure and composition at the atomic scale, computer simulations can provide valuable information if reliable interatomic potentials are available. In this paper we describe the construction of a high-dimensional neural network potential (HDNNP) intended for simulations of large brass nanoparticles with thousands of atoms, which is also applicable to bulk a-brass and its surfaces. The HDNNP, which is based on reference data obtained from density-functional theory calculations, is very accurate with a root-mean-square error of 1.7 meV/atom for total energies and 39 meV angstrom(-1) for the forces of structures not included in the training set. The potential has been thoroughly validated for a wide range of energetic and structural properties of bulk a-brass, its surfaces as well as clusters of different size and composition demonstrating its suitability for large-scale molecular dynamics and Monte Carlo simulations with firstprinciples accuracy.},
    annotation = {WOS:000541745800055},
    author = {Weinreich, Jan and Roemer, Anton and Paleico, Martin Leandro and Behler, Joerg},
    date = {2020-06-11},
    doi = {10.1021/acs.jpcc.0c00559},
    issn = {1932-7447},
    journaltitle = {Journal of Physical Chemistry C},
    langid = {english},
    location = {{Washington}},
    number = {23},
    pages = {12682--12695},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. C},
    title = {Properties of Alpha-{{Brass Nanoparticles}}. 1. {{Neural Network Potential Energy Surface}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {124}
}

@article{wenDevelopmentDeepMachine2019,
    abstract = {Interatomic potentials based on neural-network machine learning (ML) approach to address the long-standing challenge of accuracy versus efficiency in molecular-dynamics simulations have recently attracted a great deal of interest. Here, utilizing Pd-Si system as a prototype, we extend the development of neural-network ML potentials to compounds exhibiting various types of bonding characteristics. The ML potential is trained by fitting to the energies and forces of both liquid and crystal structures first-principles calculations based on density-functional theory (DFT). We show that the generated ML potential captures the structural features and motifs in Pd82Si18 and Pd75Si25 liquids more accurately than the existing interatomic potential based on embedded-atom method (EAM). The ML potential also describes the solid-liquid interface of these systems very well. Moreover, while the existing EAM potential fails to describe the relative energies of various crystalline structures and predict wrong ground-state structures at Pd3Si and Pd9Si2 composition, the developed ML potential predicts correctly the ground-state structures from genetic algorithm search. The efficient ML potential with DFT accuracy from our study will provide a promising scheme for accurate atomistic simulations of structures and dynamics of complex Pd-Si system.},
    annotation = {WOS:000494024200001},
    author = {Wen, Tongqi and Wang, Cai-Zhuang and Kramer, M. J. and Sun, Yang and Ye, Beilin and Wang, Haidi and Liu, Xueyuan and Zhang, Chao and Zhang, Feng and Ho, Kai-Ming and Wang, Nan},
    date = {2019-11-04},
    doi = {10.1103/PhysRevB.100.174101},
    issn = {2469-9950},
    journaltitle = {Physical Review B},
    langid = {english},
    location = {{College Pk}},
    number = {17},
    pages = {174101},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. B},
    title = {Development of a Deep Machine Learning Interatomic Potential for Metalloid-Containing {{Pd}}-{{Si}} Compounds},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {100}
}

@article{westermayrCombiningSchNetSHARC2020,
    abstract = {In recent years, deep learning has become a part of our everyday life and is revolutionizing quantum chemistry as well. In this work, we show how deep learning can be used to advance the research field of photochemistry by learning all important properties-multiple energies, forces, and different couplings-for photodynamics simulations. We simplify such simulations substantially by (i) a phase-free training skipping costly preprocessing of raw quantum chemistry data; (ii) rotationally covariant nonadiabatic couplings, which can either be trained or (iii) alternatively be approximated from only ML potentials, their gradients, and Hessians; and (iv) incorporating spin-orbit couplings. As the deep-learning method, we employ SchNet with its automatically determined representation of molecular structures and extend it for multiple electronic states. In combination with the molecular dynamics program SHARC, our approach termed SchNarc is tested on two polyatomic molecules and paves the way toward efficient photodynamics simulations of complex systems.},
    annotation = {WOS:000537432500012},
    author = {Westermayr, Julia and Gastegger, Michael and Marquetand, Philipp},
    date = {2020-05-21},
    doi = {10.1021/acs.jpclett.0c00527},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {10},
    pages = {3828--3834},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    shorttitle = {Combining {{SchNet}} and {{SHARC}}},
    title = {Combining {{SchNet}} and {{SHARC}}: {{The SchNarc Machine Learning Approach}} for {{Excited}}-{{State Dynamics}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {11}
}

@article{Willard,
    abstract = {Nitrogen oxides are removed from the troposphere through the reactive uptake of N 2 O 5 into aqueous aerosol. This process is thought to occur within the bulk of an aerosol, through solvation and subsequent hydrolysis. However, this perspective is difficult to reconcile with field measurements and cannot be verified directly because of the fast reaction kinetics of N 2 O 5. Here, we use molecular simulations, including reactive potentials and importance sampling, to study the uptake of N 2 O 5 into an aqueous aerosol. Rather than being mediated by the bulk, uptake is dominated by interfacial processes due to facile hydrolysis at the liquid-vapor interface and competitive reevaporation. With this molecular information, we propose an alternative interfacial reactive uptake model consistent with existing experimental observations.},
    author = {Willard, J and Jia, X and Xu, S and Steinbach, M and Kumar, V},
    journaltitle = {arxiv.org},
    title = {Integrating Physics-Based Modeling with Machine Learning: {{A}} Survey},
    url = {https://arxiv.org/abs/2003.04919}
}

@article{willattAtomdensityRepresentationsMachine2019a,
    abstract = {The applications of machine learning techniques to chemistry and materials science become more numerous by the day. The main challenge is to devise representations of atomic systems that are at the same complete and concise, so as to reduce the number of reference calculations that are needed to predict the properties of different types of materials reliably. This has led to a proliferation of alternative ways to convert an atomic structure into an input for a machine-learning model. We introduce an abstract definition of chemical environments that is based on a smoothed atomic density, using a bra-ket notation to emphasize basis set independence and to highlight the connections with some popular choices of representations for describing atomic systems. The correlations between the spatial distribution of atoms and their chemical identities are computed as inner products between these feature kets, which can be given an explicit representation in terms of the expansion of the atom density on orthogonal basis functions, that is equivalent to the smooth overlap of atomic positions power spectrum, but also in real space, corresponding to n-body correlations of the atom density. This formalism lays the foundations for a more systematic tuning of the behavior of the representations, by introducing operators that represent the correlations between structure, composition, and the target properties. It provides a unifying picture of recent developments in the field and indicates a way forward toward more effective and computationally affordable machine-learning schemes for molecules and materials. Published under license by AIP Publishing.},
    annotation = {WOS:000465442100014},
    author = {Willatt, Michael J. and Musil, Flix and Ceriotti, Michele},
    date = {2019-04-21},
    doi = {10.1063/1.5090481},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {15},
    pages = {154110},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Atom-Density Representations for Machine Learning},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {150}
}

@article{wirnsbergerTargetedFreeEnergy2020a,
    abstract = {Free energy perturbation (FEP) was proposed by Zwanzig [J. Chem. Phys. 22, 1420 (1954)] more than six decades ago as a method to estimate free energy differences and has since inspired a huge body of related methods that use it as an integral building block. Being an importance sampling based estimator, however, FEP suffers from a severe limitation: the requirement of sufficient overlap between distributions. One strategy to mitigate this problem, called Targeted FEP, uses a high-dimensional mapping in configuration space to increase the overlap of the underlying distributions. Despite its potential, this method has attracted only limited attention due to the formidable challenge of formulating a tractable mapping. Here, we cast Targeted FEP as a machine learning problem in which the mapping is parameterized as a neural network that is optimized so as to increase the overlap. We develop a new model architecture that respects permutational and periodic symmetries often encountered in atomistic simulations and test our method on a fully periodic solvation system. We demonstrate that our method leads to a substantial variance reduction in free energy estimates when compared against baselines, without requiring any additional data.},
    annotation = {WOS:000584689800003},
    author = {Wirnsberger, Peter and Ballard, Andrew J. and Papamakarios, George and Abercrombie, Stuart and Racaniere, Sebastien and Pritzel, Alexander and Rezende, Danilo Jimenez and Blundell, Charles},
    date = {2020-10-14},
    doi = {10.1063/5.0018903},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {14},
    pages = {144112},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    title = {Targeted Free Energy Estimation via Learned Mappings},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {153}
}

@article{wuDeepLearningAccurate2021,
    abstract = {The discovery of ferroelectricity in HfO2-based thin films opens up new opportunities for using this silicon-compatible ferroelectric to realize low-power logic circuits and high-density nonvolatile memories. The functional performances of ferroelectrics are intimately related to their dynamic responses to external stimuli such as electric fields at finite temperatures. Molecular dynamics is an ideal technique for investigating dynamical processes on large length and time scales, though its applications to new materials are often hindered by the limited availability and accuracy of classical force fields. Here we present a deep neural network-based interatomic force field of HfO2 learned from ab initio data using a concurrent learning procedure. The model potential is able to predict structural properties such as elastic constants, equation of states, phonon dispersion relationships, and phase transition barriers of various hafnia polymorphs with accuracy comparable with density functional theory calculations. The validity of this model potential is further confirmed by the reproduction of experimental sequences of temperature-driven ferroelectric-paraelectric phase transitions of HfO2 with isobaric-isothermal ensemble molecular dynamics simulations. We suggest a general approach to extend the model potential of HfO(2 )to related material systems including dopants and defects.},
    annotation = {WOS:000613140900002},
    author = {Wu, Jing and Zhang, Yuzhi and Zhang, Linfeng and Liu, Shi},
    date = {2021-01-29},
    doi = {10.1103/PhysRevB.103.024108},
    issn = {2469-9950},
    journaltitle = {Physical Review B},
    langid = {english},
    location = {{College Pk}},
    number = {2},
    pages = {024108},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. B},
    title = {Deep Learning of Accurate Force Field of Ferroelectric {{HfO2}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {103}
}

@article{wuModelingMetalNanoparticles2020,
    abstract = {Interatomic potential plays a key role in ensuring the accuracy and reliability of molecular-dynamics simulation results. While most empirical potentials are benchmarked against a set of carefully chosen bulk material properties, recent advances in machine learning have seen the emergence of neural-network-based mathematical potentials capable of describing highly complex potential energy surfaces for a variety of systems. We report here the development of a neural-network interatomic potential (NNIP) with modified embedded-atom method background density as fingerprint functions, which could accurately model the energetics of metallic nanoparticles and clusters (Cu as a representative example) widely used in catalysis. To appropriately account for the diverse chemical environments encountered in nanoparticles/nanoclusters, an extensive set of atomic configurations (totaling 18 084) were calculated using density-functional-theory (DFT) at the Perdew-Burke-Ernzerhof level. In addition to standard bulk properties such as cohesive energies and elastic constants, the sampled configurations also include a substantial number of differently oriented crystal facets and differently sized nanoparticles and nanoclusters, greatly expanding the value range of NNIP features that was otherwise quite limited. The complex energy potential surface of Cu can be faithfully reproduced, with an average error of 0.011 eV/at for energy states within 3 eV of the ground state. As an illustration, the developed NNIP is used to simulate the molecular dynamics of copper nanoparticles, and good agreement is achieved between DFT and the NNIP.},
    annotation = {WOS:000577208700002},
    author = {Wu, Feifeng and Min, Hang and Wen, Yanwei and Chen, Rong and Zhao, Yunkun and Ford, Mike and Shan, Bin},
    date = {2020-10-15},
    doi = {10.1103/PhysRevB.102.144107},
    issn = {2469-9950},
    journaltitle = {Physical Review B},
    langid = {english},
    location = {{College Pk}},
    number = {14},
    pages = {144107},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. B},
    shorttitle = {Modeling of Metal Nanoparticles},
    title = {Modeling of Metal Nanoparticles: {{Development}} of Neural-Network Interatomic Potential Inspired by Features of the Modified Embedded-Atom Method},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {102}
}

@article{xiaHighThroughputStudyLattice2020,
    abstract = {Thermal transport phenomena are ubiquitous and play a critical role in the performance of various microelectronic and energy-conversion devices. Binary rocksalt and zinc blende compounds, despite their rather simple crystal structures, exhibit an extraordinary range of lattice thermal conductivity (kappa(L)) spanning over 3 orders of magnitude. A comprehensive understanding of the underlying heat transfer mechanism through the development of microscopic theories is therefore of fundamental importance, yet it remains elusive because of the challenges arising from explicitly treating higher-order anharmonicity. Recent theoretical and experimental advances have revealed the essential role of quartic anharmonicity in suppressing heat transfer in zinc blende boron arsenide (BAs) with ultrahigh kappa(L). However, critical questions concerning the general effects of higher-order anharmonicity in the broad classes and chemistries of binary solids are still unanswered. Using our recently developed high-throughput phonon framework based on first-principles density functional theory calculations, we systematically investigate the lattice dynamics and thermal transport properties of 37 binary compounds with rocksalt and zinc blende structures at room temperature, with a particular focus on unraveling the impacts of quartic anharmonicity on kappa(L). Our advanced theoretical model for computing kappa(L) embraces current state-of-the-art methods, featuring a complete treatment of quartic anharmonicity for both phonon frequencies and lifetimes at finite temperatures, as well as contributions from off-diagonal terms in the heat-flux operator. We find the impacts of quartic anharmonicity on kappa(L) to be strikingly different in rocksalt and zinc blende compounds, owing to the countervailing effects on finite-temperature-induced shifts in phonon frequencies and scattering rates. By correlating.L with the phonon scattering phase space, we outline a qualitative but efficient route to assess the importance of four-phonon scattering from harmonic phonon calculations. Among notable examples, in zinc blende HgTe, we identify an unprecedented sixfold reduction in kappa(L) due to four-phonon scattering, which dominates over the three-phonon scattering in the acoustic region at room temperature. We also demonstrate a possible breakdown of the phonon gas model in rocksalt AgCl, wherein the phonon states are significantly broadened due to strong intrinsic anharmonicity, inducing off-diagonal contributions to kappa(L) comparable to the diagonal ones. The deep physical insights gained in this work can be used to guide the rational design of thermal management materials.},
    annotation = {WOS:000588236900001},
    author = {Xia, Yi and Hegde, Vinay and Pal, Koushik and Hua, Xia and Gaines, Dale and Patel, Shane and He, Jiangang and Aykol, Muratahan and Wolverton, Chris},
    date = {2020-11-10},
    doi = {10.1103/PhysRevX.10.041029},
    issn = {2160-3308},
    journaltitle = {Physical Review X},
    langid = {english},
    location = {{College Pk}},
    number = {4},
    pages = {041029},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. X},
    title = {High-{{Throughput Study}} of {{Lattice Thermal Conductivity}} in {{Binary Rocksalt}} and {{Zinc Blende Compounds Including Higher}}-{{Order Anharmonicity}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {10}
}

@article{xieBayesianForceFields2021a,
    abstract = {We present a way to dramatically accelerate Gaussian process models for interatomic force fields based on many-body kernels by mapping both forces and uncertainties onto functions of low-dimensional features. This allows for automated active learning of models combining near-quantum accuracy, built-in uncertainty, and constant cost of evaluation that is comparable to classical analytical models, capable of simulating millions of atoms. Using this approach, we perform large-scale molecular dynamics simulations of the stability of the stanene monolayer. We discover an unusual phase transformation mechanism of 2D stanene, where ripples lead to nucleation of bilayer defects, densification into a disordered multilayer structure, followed by formation of bulk liquid at high temperature or nucleation and growth of the 3D bcc crystal at low temperature. The presented method opens possibilities for rapid development of fast accurate uncertainty-aware models for simulating long-time large-scale dynamics of complex materials.},
    annotation = {WOS:000631155400002},
    author = {Xie, Yu and Vandermause, Jonathan and Sun, Lixin and Cepellotti, Andrea and Kozinsky, Boris},
    date = {2021-03-19},
    doi = {10.1038/s41524-021-00510-y},
    journaltitle = {Npj Computational Materials},
    langid = {english},
    location = {{Berlin}},
    number = {1},
    pages = {40},
    publisher = {{Nature Research}},
    shortjournal = {npj Comput. Mater.},
    title = {Bayesian Force Fields from Active Learning for Simulation of Inter-Dimensional Transformation of Stanene},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {7}
}

@article{xieGraphDynamicalNetworks2019a,
    abstract = {Understanding the dynamical processes that govern the performance of functional materials is essential for the design of next generation materials to tackle global energy and environmental challenges. Many of these processes involve the dynamics of individual atoms or small molecules in condensed phases, e.g. lithium ions in electrolytes, water molecules in membranes, molten atoms at interfaces, etc., which are difficult to understand due to the complexity of local environments. In this work, we develop graph dynamical networks, an unsupervised learning approach for understanding atomic scale dynamics in arbitrary phases and environments from molecular dynamics simulations. We show that important dynamical information, which would be difficult to obtain otherwise, can be learned for various multi-component amorphous material systems. With the large amounts of molecular dynamics data generated every day in nearly every aspect of materials design, this approach provides a broadly applicable, automated tool to understand atomic scale dynamics in material systems.},
    annotation = {WOS:000471758500003},
    author = {Xie, Tian and France-Lanord, Arthur and Wang, Yanming and Shao-Horn, Yang and Grossman, Jeffrey C.},
    date = {2019-06-17},
    doi = {10.1038/s41467-019-10663-6},
    issn = {2041-1723},
    journaltitle = {Nature Communications},
    langid = {english},
    location = {{London}},
    pages = {2667},
    publisher = {{Nature Publishing Group}},
    shortjournal = {Nat. Commun.},
    title = {Graph Dynamical Networks for Unsupervised Learning of Atomic Scale Dynamics in Materials},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {10}
}

@article{xuAutomaticallyConstructedNeural2021a,
    abstract = {The development of accurate and efficient potential energy functions for the molecular dynamics simulation of metalloproteins has long been a great challenge for the theoretical chemistry community. An artificial neural network provides the possibility to develop potential energy functions with both the efficiency of the classical force fields and the accuracy of the quantum chemical methods. In this work, neural network potentials were automatically constructed by using the ESOINN-DP method for typical zinc proteins. For the four most common zinc coordination modes in proteins, the potential energy, atomic forces, and atomic charges predicted by neural network models show great agreement with quantum mechanics calculations and the neural network potential can maintain the coordination geometry correctly. In addition, MD simulation and energy optimization with the neural network potential can be readily used for structural refinement. The neural network potential is not limited by the function form and complex parameterization process, and important quantum effects such as polarization and charge transfer can be accurately considered. The algorithm proposed in this work can also be directly applied to proteins containing other metal ions.},
    annotation = {WOS:000668723600001},
    author = {Xu, Mingyuan and Zhu, Tong and Zhang, John Z. H.},
    date = {2021-06-18},
    doi = {10.3389/fchem.2021.692200},
    issn = {2296-2646},
    journaltitle = {Frontiers in Chemistry},
    langid = {english},
    location = {{Lausanne}},
    pages = {692200},
    publisher = {{Frontiers Media Sa}},
    shortjournal = {Front. Chem.},
    title = {Automatically {{Constructed Neural Network Potentials}} for {{Molecular Dynamics Simulation}} of {{Zinc Proteins}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {9}
}

@article{xuDeeplearningPotentialCrystalline2020,
    abstract = {This work investigates the ability of the deep-learning potential (DP) to describe structural, dynamic and energetic properties of crystalline and amorphous Li-Si alloys. Li-Si systems play an important role in the development of high-energy lithium ion batteries. One challenge in simulating Li-Si systems is to balance the proper description of complex Li-Si interactions and the system size. Molecular simulations implemented with DP provide a promising alternative to achieve this balance and enable us to investigate the fine details of Li-Si systems that the classical force fields cannot offer. We develop a DP for Li-Si systems with Li/Si ratio ranging from 0 to 4.2 based on a vast data set generated using the quantum mechanical calculations in an active learning procedure. Then we investigate the structural and dynamic properties of several crystalline and amorphous Li-Si systems using this developed DP. The DP can predict bulk densities, the radial distribution functions, and diffusivity of Li in amorphous Li-Si systems with an accuracy close to quantum mechanical calculations with the benefit of 20 times faster speed than the ab initio molecular dynamics simulations. Several issues related to the development of DP are also discussed.},
    author = {Xu, Nan and Shi, Yao and He, Yi and Shao, Qing},
    date = {2020-07},
    doi = {10.1021/acs.jpcc.0c03333},
    journaltitle = {Journal of Physical Chemistry C},
    number = {30},
    pages = {16278--16288},
    publisher = {{American Chemical Society}},
    title = {A Deep-Learning Potential for Crystalline and Amorphous Li-Si Alloys},
    url = {https://pubs.acs.org/doi/abs/10.1021/acs.jpcc.0c03333},
    volume = {124}
}

@article{xuIsotopeEffectsMolecular2020,
    abstract = {Feynman path-integral deep potential molecular dynamics (PI-DPMD) calculations have been employed to study both light (H2O) and heavy water (D2O) within the isothermalisobaric ensemble. In particular, the deep neural network is trained based on ab initio data obtained from the strongly constrained and appropriately normed (SCAN) exchange-correlation functional. Because of the lighter mass of hydrogen than deuteron, the properties of light water are more influenced by nuclear quantum effect than those of heavy water. Clear isotope effects are observed and analyzed in terms of hydrogen-bond structure and electronic properties of water that are closely associated with experimental observables. The molecular structures of both liquid H2O and D2O agree well with the data extracted from scattering experiments. The delicate isotope effects on radial distribution functions and angular distribution functions are well reproduced as well. Our approach demonstrates that deep neural network combined with SCAN functional based ab initio molecular dynamics provides an accurate theoretical tool for modeling water and its isotope effects.},
    annotation = {WOS:000600833600001},
    author = {Xu, Jianhang and Zhang, Chunyi and Zhang, Linfeng and Chen, Mohan and Santra, Biswajit and Wu, Xifan},
    date = {2020-12-21},
    doi = {10.1103/PhysRevB.102.214113},
    issn = {2469-9950},
    journaltitle = {Physical Review B},
    langid = {english},
    location = {{College Pk}},
    number = {21},
    pages = {214113},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. B},
    title = {Isotope Effects in Molecular Structures and Electronic Properties of Liquid Water via Deep Potential Molecular Dynamics Based on the {{SCAN}} Functional},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {102}
}

@article{xuMolecularDynamicsSimulation2019a,
    abstract = {An artificial neural network provides the possibility to develop molecular potentials with both the efficiency of the classical molecular mechanics and the accuracy of the quantum chemical methods. Here, we develop an ab initio based neural network potential (NN/MM-RESP) to perform molecular dynamics study of zinc ion in liquid water. In this approach, the interaction energy, atomic forces, and atomic charges of zinc ion and water molecules' in the first solvent shell are described by a neural network potential trained with energies and forces generated from density functional calculations. The predicted energies and forces from the NN potential show excellent agreement with the quantum chemistry calculations. Using this approach, we carried out molecular dynamics simulation to study the hydration of zinc ion in water. The experimentally observed zinc-water radial distribution function, as well as the X-ray absorption near edge structure spectrum, is well-reproduced by the MD simulation. Comparison of the results with other theoretical calculations is provided, and important features of the present approach are discussed. The neural network approach used in this study can be applied to construct potentials to study solvation of other metal ions, and its salient features can shed light on the development of more accurate molecular potentials for metal ions in other environments such as proteins.},
    annotation = {WOS:000486361700021},
    author = {Xu, Mingyuan and Zhu, Tong and Zhang, John Z. H.},
    date = {2019-08-01},
    doi = {10.1021/acs.jpca.9b04087},
    issn = {1089-5639},
    journaltitle = {Journal of Physical Chemistry A},
    langid = {english},
    location = {{Washington}},
    number = {30},
    pages = {6587--6595},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. A},
    title = {Molecular {{Dynamics Simulation}} of {{Zinc Ion}} in {{Water}} with an Ab {{Initio Based Neural Network Potential}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {123}
}

@article{xuOptimizingTrainingData2021,
    abstract = {Machine learning potential enables molecular dynamics simulations of systems beyond the capability of traditional force fields. One challenge in developing machine learning potential is how to construct a data set with low sample redundancy. This work investigates the method to optimize the training data set while maintaining the desirable accuracy of the machine learning potential using the structural similarity algorithm. We construct several subsets ranging from 200-1500 sample configurations by selecting representative configurations from a 6183-sample data set using the farthest point sampling method and examine the ability of the machine learning potential trained from the subsets to predict energy, atomic forces and structural properties of Li-Si systems. The simulation results show that the potential developed from 400 configurations can be as accurate as the one developed from the 6183-sample data set. In addition, our computation results highlight that the structure-comparison algorithms can not only effectively remove redundant from training sets, but also achieve an appropriate distribution of samples in training data sets.},
    archiveprefix = {arXiv},
    arxivid = {2103.04347},
    author = {Xu, Nan and Li, Chen and Shi, Yao and Shao, Qing and He, Yi},
    date = {2021-03},
    eprint = {2103.04347},
    eprinttype = {arxiv},
    journaltitle = {arxiv.org},
    title = {Optimizing Training Data Set for the Machine Learning Potential of Li-Si Alloys via Structural Similarity-Based Screening},
    url = {https://arxiv.org/abs/2103.04347 http://arxiv.org/abs/2103.04347}
}

@article{xuPerspectiveComputationalReaction2021,
    abstract = {Heterogeneous catalysis plays a significant role in the modern chemical industry. Towards the rational design of novel catalysts, understanding reactions over surfaces is the most essential aspect. Typical industrial catalytic processes such as syngas conversion and methane utilisation can generate a large reaction network comprising thousands of intermediates and reaction pairs. This complexity not only arises from the permutation of transformations between species but also from the extra reaction channels offered by distinct surface sites. Despite the success in investigating surface reactions at the atomic scale, the huge computational expense of ab initio methods hinders the exploration of such complicated reaction networks. With the proliferation of catalysis studies, machine learning as an emerging tool can take advantage of the accumulated reaction data to emulate the output of ab initio methods towards swift reaction prediction. Here, we briefly summarise the conventional workflow of reaction prediction, including reaction network generation, ab initio thermodynamics and microkinetic modelling. An overview of the frequently used regression models in machine learning is presented. As a promising alternative to full ab initio calculations, machine learning interatomic potentials are highlighted. Furthermore, we survey applications assisted by these methods for accelerating reaction prediction, exploring reaction networks, and computational catalyst design. Finally, we envisage future directions in computationally investigating reactions and implementing machine learning algorithms in heterogeneous catalysis.},
    annotation = {WOS:000648898600001},
    author = {Xu, Jiayan and Cao, Xiao-Ming and Hu, P.},
    date = {2021-05-21},
    doi = {10.1039/d1cp01349a},
    issn = {1463-9076},
    journaltitle = {Physical Chemistry Chemical Physics},
    langid = {english},
    location = {{Cambridge}},
    number = {19},
    pages = {11155--11179},
    publisher = {{Royal Soc Chemistry}},
    shortjournal = {Phys. Chem. Chem. Phys.},
    title = {Perspective on Computational Reaction Prediction Using Machine Learning Methods in Heterogeneous Catalysis},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {23}
}

@article{Yang,
    abstract = {The study of chemical reactions in aqueous media is very important for its implications in several fields of science, from biology to industrial processes. However, modeling these reactions is difficult when water directly participates in the reaction, since it requires a fully quantum mechanical description of the system. Ab-initio molecular dynamics is the ideal candidate to shed light on these processes. However, its scope is limited by a high computational cost. A popular alternative is to perform molecular dynamics simulations powered by machine learning potentials, trained on an extensive set of quantum mechanical calculations. Doing so reliably for reactive processes is difficult because it requires including very many intermediate and transition state configurations. In this study we used an active learning procedure accelerated by enhanced sampling to harvest such structures and to build a neural-network potential to study the urea decomposition process in water. This allowed us to obtain the free energy profiles of this important reaction in a wide range of temperatures, to discover several novel metastable states, and improve the accuracy of the kinetic rates calculations. Furthermore, we found that the formation of the zwitterionic intermediate has the same probability of occurring via an acidic or a basic pathway, which could be the cause of the insensitivity of reaction rates to the solution pH.},
    author = {Yang, M and Bonati, L and Polino, D and M, Parrinello},
    journaltitle = {Elsevier},
    title = {Using Metadynamics to Build Neural Network Potentials for Reactive Events: The Case of Urea Decomposition in Water},
    url = {https://www.sciencedirect.com/science/article/pii/S092058612100136X}
}

@article{yangRoleWaterReaction2019a,
    abstract = {Asymmetric 1,3-dipolar cycloadditions of azomethine ylides with activated olefins are among the most important and versatile methods for the synthesis of enantioenriched pyrroline and pyrrolidine derivatives. Despite both theoretical and practical importance, the role of water molecules in the reactivity and endo/exo selectivity remains unclear. To explore how water accelerates the reactions and improves the endo/exo selectivity of the cycloadditions of 1,3-dipole phthalazinium-2-dicyanomethanide (1) and two dipolarophiles, an ab initio-quality neural network potential that overcomes the computational bottleneck of explicitly considering water molecules was used. It is demonstrated that not only the nature of both the dipolarophile and the 1,3-dipole, but also the solvent medium, can perturb or even alter the reaction mechanism. An extreme case was found for the reaction of 1,3-dipole 1 with methyl vinyl ketone, in which the reaction mechanism changes from a concerted to a stepwise mode on going from MeCN to H2O as solvent, with formation of a zwitterionic intermediate that is a very shallow minimum on the energy surface. Thus, high stereocontrol can still be expected despite the stepwise nature of the mechanism. The results indicate that water can induce global polarization along the reaction coordinate and highlight the role of microsolvation effects and bulk-phase effects in reproducing the experimentally observed aqueous acceleration and enhanced endo/exo selectivity.},
    annotation = {WOS:000474808200018},
    author = {Yang, Xin and Zou, Jun and Wang, Yifei and Xue, Ying and Yang, Shengyong},
    date = {2019-06-21},
    doi = {10.1002/chem.201900617},
    issn = {0947-6539},
    journaltitle = {Chemistry-a European Journal},
    langid = {english},
    location = {{Weinheim}},
    number = {35},
    pages = {8289--8303},
    publisher = {{Wiley-V C H Verlag Gmbh}},
    shortjournal = {Chem.-Eur. J.},
    title = {Role of {{Water}} in the {{Reaction Mechanism}} and Endo/Exo {{Selectivity}} of 1,3-{{Dipolar Cycloadditions Elucidated}} by {{Quantum Chemistry}} and {{Machine Learning}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {25}
}

@article{yaoNuclearQuantumEffect2021a,
    abstract = {We report structural and dynamical properties of liquid water described by the random phase approximation (RPA) correlation together with the exact exchange energy (EXX) within density functional theory. By utilizing thermostated ring polymer molecular dynamics, we examine the nuclear quantum effects and their temperature dependence. We circumvent the computational limitation of performing direct first-principles molecular dynamics simulation at this high level of electronic structure theory by adapting an artificial neural network model. We show that the EXX+RPA level of theory accurately describes liquid water in terms of both dynamical and structural properties.},
    annotation = {WOS:000674852100016},
    author = {Yao, Yi and Kanai, Yosuke},
    date = {2021-07-15},
    doi = {10.1021/acs.jpclett.1c01566},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {27},
    pages = {6354--6362},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    title = {Nuclear {{Quantum Effect}} and {{Its Temperature Dependence}} in {{Liquid Water}} from {{Random Phase Approximation}} via {{Artificial Neural Network}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {12}
}

@article{yooAtomicEnergyMapping2019a,
    abstract = {We investigate the atomic energy mapping inferred by machine-learning potentials, in particular neural network potentials. We first show that the transferable atomic energy can be defined within the density functional theory, which means that the core of machine-learning potentials is to deduce a reference atomic-energy function from the given set of total energies. By utilizing invariant points in the feature space at which the atomic energy has a fixed reference value, we examine the atomic energy mapping of neural network potentials. Examples on Si consistently support that NNPs are capable of learning correct atomic energies. However, we also find that the neural network potential is vulnerable to 'ad hoc' mapping in which the total energy appears to be trained accurately while the atomic energy mapping is incorrect in spite of its capability. We show that the energy mapping can be improved by choosing the training set carefully and monitoring the atomic energy at the invariant points during the training procedure. The energy mapping in multicomponent systems is also discussed.},
    annotation = {WOS:000483590500005},
    author = {Yoo, Dongsun and Lee, Kyuhyun and Jeong, Wonseok and Lee, Dongheon and Watanabe, Satoshi and Han, Seungwu},
    date = {2019-09-03},
    doi = {10.1103/PhysRevMaterials.3.093802},
    issn = {2475-9953},
    journaltitle = {Physical Review Materials},
    langid = {english},
    location = {{College Pk}},
    number = {9},
    pages = {093802},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. Mater.},
    title = {Atomic Energy Mapping of Neural Network Potential},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {3}
}

@article{youngTransferableActivelearningStrategy,
    abstract = {Predictive molecular simulations require fast, accurate and reactive interatomic potentials. Machine learning offers a promising approach to construct such potentials by fitting energies and forces to high-level quantum-mechanical data, but doing so typically requires considerable human intervention and data volume. Here we show that, by leveraging hierarchical and active learning, accurate Gaussian Approximation Potential (GAP) models can be developed for diverse chemical systems in an autonomous manner, requiring only hundreds to a few thousand energy and gradient evaluations on a reference potential-energy surface. The approach uses separate intra- and inter-molecular fits and employs a prospective error metric to assess the accuracy of the potentials. We demonstrate applications to a range of molecular systems with relevance to computational organic chemistry: ranging from bulk solvents, a solvated metal ion and a metallocage onwards to chemical reactivity, including a bifurcating Diels-Alder reaction in the gas phase and non-equilibrium dynamics (a model S(N)2 reaction) in explicit solvent. The method provides a route to routinely generating machine-learned force fields for reactive molecular systems.},
    annotation = {WOS:000674966100001},
    author = {Young, Tom A. and Johnston-Wood, Tristan and Deringer, Volker L. and Duarte, Fernanda},
    doi = {10.1039/d1sc01825f},
    issn = {2041-6520},
    journaltitle = {Chemical Science},
    langid = {english},
    location = {{Cambridge}},
    publisher = {{Royal Soc Chemistry}},
    shortjournal = {Chem. Sci.},
    title = {A Transferable Active-Learning Strategy for Reactive Molecular Force Fields},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06}
}

@article{Zeng2021,
    abstract = {We develop a new Deep Potential-Range Correction (DPRc) machine learning potential for combined quantum mechanical/molecular mechanical (QM/MM) simulations of chemical reactions in the condensed phase. The new range correction enables short-ranged QM/MM interactions to be tuned for higher accuracy, and the correction smoothly vanishes within a specified cutoff. We further develop an active learning procedure for robust neural network training. We test the DPRc model and training procedure against a series of 6 non-enzymatic phosphoryl transfer reactions in solution that are important in mechanistic studies of RNA-cleaving enzymes. Specifically, we apply DPRc corrections to a base QM model and test its ability to reproduce free energy profiles generated from a target QM model. We perform these * To whom correspondence should be addressed 1 comparisons using the MNDO/d and DFTB2 semiempirical models because they produce free energy profiles which differ significantly from each other, thereby providing us a rigorous stress test for the DPRc model and training procedure. The comparisons show that accurate reproduction of the free energy profiles requires correction of the QM/MM interactions out to 6 Å. We further find that the model's initial training benefits from generating data from temperature replica exchange simulations and including high-temperature configurations into the fitting procedure so the resulting models are trained to properly avoid high-energy regions. A single DPRc model was trained to reproduce 4 different reactions and yielded good agreement with the free energy profiles made from the target QM/MM simulations. The DPRc model was further demonstrated to be transferable to 2D free energy surfaces and 1D free energy profiles that were not explicitly considered in the training. Examination of the computational performance of the DPRc model showed that it was fairly slow when run on CPUs, but was sped up almost 100-fold when using an NVIDIA V100 GPUs, resulting in almost negligible overhead. The new DPRc model and training procedure provide a potentially powerful new tool for the creation of next-generation QM/MM potentials for a wide spectrum of free energy applications ranging from drug discovery to enzyme design. 2},
    author = {Zeng, J and Giese, TJ and Ekesan, Ş and York, DM},
    date = {2021},
    title = {Development of Range-Corrected Deep Learning Potentials for Fast, Accurate Quantum Mechanical/Molecular Mechanical Simulations of Chemical Reactions in Solution},
    url = {https://chemrxiv.org/engage/chemrxiv/article-details/60c7557dbdbb899a20a3a828}
}

@article{zengComplexReactionProcesses2020,
    abstract = {Combustion is a complex chemical system which involves thousands of chemical reactions and generates hundreds of molecular species and radicals during the process. In this work, a neural network-based molecular dynamics (MD) simulation is carried out to simulate the benchmark combustion of methane. During MD simulation, detailed reaction processes leading to the creation of specific molecular species including various intermediate radicals and the products are intimately revealed and characterized. Overall, a total of 798 different chemical reactions were recorded and some new chemical reaction pathways were discovered. We believe that the present work heralds the dawn of a new era in which neural network-based reactive MD simulation can be practically applied to simulating important complex reaction systems at ab initio level, which provides atomic-level understanding of chemical reaction processes as well as discovery of new reaction pathways at an unprecedented level of detail beyond what laboratory experiments could accomplish. Gaining insights into combustion processes is challenging due to the complex reactions involved. The present work proposes a neural network potential model trained to ab initio data that enables to simulate the combustion of methane by predicting reactants, products and reaction intermediates.},
    annotation = {WOS:000593975100015},
    author = {Zeng, Jinzhe and Cao, Liqun and Xu, Mingyuan and Zhu, Tong and Zhang, John Z. H.},
    date = {2020-11-11},
    doi = {10.1038/s41467-020-19497-z},
    issn = {2041-1723},
    journaltitle = {Nature Communications},
    langid = {english},
    location = {{Berlin}},
    number = {1},
    pages = {5713},
    publisher = {{Nature Research}},
    shortjournal = {Nat. Commun.},
    title = {Complex Reaction Processes in Combustion Unraveled by Neural Network-Based Molecular Dynamics Simulation},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {11}
}

@article{zengNeuralNetworkBased2019,
    abstract = {Understanding and prediction of the chemical reactions are fundamental demanding in the study of many complex chemical systems. Reactive molecular dynamics (MD) simulation has been widely used for this purpose as it can offer atomic details and can help us better interpret chemical reaction mechanisms. In this study, two reference datasets were constructed and corresponding neural network (NN) potentials were trained based on them. For given large-scale reaction systems, the NN potentials can predict the potential energy and atomic forces of DFT precision, while it is orders of magnitude faster than the conventional DFT calculation. With these two models, reactive MD simulations were performed to explore the combustion mechanisms of hydrogen and methane. Benefit from the high efficiency of the NN model, nanosecond MD trajectories for large-scale systems containing hundreds of atoms were produced and detailed combustion mechanism was obtained. Through further development, the algorithms in this study can be used to explore and discovery reaction mechanisms of many complex reaction systems, such as combustion, synthesis, and heterogeneous catalysis without any predefined reaction coordinates and elementary reaction steps.},
    archiveprefix = {arXiv},
    arxivid = {1911.12252},
    author = {Zeng, Jinzhe and Cao, Liqun and Xu, Mingyuan and Zhu, Tong and Zhang, John ZH},
    date = {2019-11},
    eprint = {1911.12252},
    eprinttype = {arxiv},
    journaltitle = {arxiv.org},
    title = {Neural Network Based in Silico Simulation of Combustion Reactions},
    url = {http://arxiv.org/abs/1911.12252 https://arxiv.org/abs/1911.12252}
}

@article{zhaiActiveLearningManybody2020,
    abstract = {The efficient selection of representative configurations that are used in high-level electronic structure calculations needed for the development of many-body molecular models poses a challenge to current data-driven approaches to molecular simulations. Here, we introduce an active learning (AL) framework for generating training sets corresponding to individual many-body contributions to the energy of an N-body system, which are required for the development of MB-nrg potential energy functions (PEFs). Our AL framework is based on uncertainty and error estimation and uses Gaussian process regression to identify the most relevant configurations that are needed for an accurate representation of the energy landscape of the molecular system under examination. Taking the Cs+-water system as a case study, we demonstrate that the application of our AL framework results in significantly smaller training sets than previously used in the development of the original MB-nrg PEF, without loss of accuracy. Considering the computational cost associated with high-level electronic structure calculations, our AL framework is particularly well-suited to the development of many-body PEFs, with chemical and spectroscopic accuracy, for molecular-level computer simulations from the gas to the condensed phase.},
    annotation = {WOS:000526884200001},
    author = {Zhai, Yaoguang and Caruso, Alessandro and Gao, Sicun and Paesani, Francesco},
    date = {2020-04-14},
    doi = {10.1063/5.0002162},
    issn = {0021-9606},
    journaltitle = {Journal of Chemical Physics},
    langid = {english},
    location = {{Melville}},
    number = {14},
    pages = {144103},
    publisher = {{Amer Inst Physics}},
    shortjournal = {J. Chem. Phys.},
    shorttitle = {Active Learning of Many-Body Configuration Space},
    title = {Active Learning of Many-Body Configuration Space: {{Application}} to the {{Cs}}+-Water {{MB}}-Nrg Potential Energy Function as a Case Study},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {152}
}

@article{zhangAcceleratingAtomisticSimulations2021a,
    abstract = {Recently, machine learning methods have become easy-to-use tools for constructing high-dimensional interatomic potentials with ab initio accuracy. Although machine-learned interatomic potentials are generally orders of magnitude faster than first-principles calculations, they remain much slower than classical force fields, at the price of using more complex structural descriptors. To bridge this efficiency gap, we propose an embedded atom neural network approach with simple piecewise switching function-based descriptors, resulting in a favorable linear scaling with the number of neighbor atoms. Numerical examples validate that this piecewise machine-learning model can be over an order of magnitude faster than various popular machine-learned potentials with comparable accuracy for both metallic and covalent materials, approaching the speed of the fastest embedded atom method (i.e. several ms per atom per CPU core). The extreme efficiency of this approach promises its potential in first-principles atomistic simulations of very large systems and/or in a long timescale.},
    annotation = {WOS:000612961700004},
    author = {Zhang, Yaolong and Hu, Ce and Jiang, Bin},
    date = {2021-01-21},
    doi = {10.1039/d0cp05089j},
    issn = {1463-9076},
    journaltitle = {Physical Chemistry Chemical Physics},
    langid = {english},
    location = {{Cambridge}},
    number = {3},
    pages = {1815--1821},
    publisher = {{Royal Soc Chemistry}},
    shortjournal = {Phys. Chem. Chem. Phys.},
    title = {Accelerating Atomistic Simulations with Piecewise Machine-Learned Ab {{Initio}} Potentials at a Classical Force Field-like Cost},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {23}
}

@article{zhangBridgingGapDirect2019a,
    abstract = {Direct dynamics simulations become increasingly popular in studying reaction dynamics for complex systems where analytical potential energy surfaces (PESs) are unavailable. Yet, the number and/or the propagation time of trajectories are often limited by high computational costs, and numerous energies and forces generated on-the-fly become wasted after simulations. We demonstrate here an example of reusing only a very small portion of existing direct dynamics data to reconstruct a 90-dimensional globally accurate reactive PES describing the interaction of CO2 with a movable Ni(100) surface based on a machine learning approach. In addition to reproducing previous results with much better statistics, we predict scattering probabilities of CO2 at the state-tostate level, which is extremely demanding for direct dynamics. We propose this unified way to investigate gaseous and gas-surface reactions of medium size, initiating with hundreds of preliminary direct dynamics trajectories, followed by low-cost and high-quality simulations on full-dimensional analytical PESs.},
    annotation = {WOS:000463678800005},
    author = {Zhang, Yaolong and Zhou, Xueyao and Jiang, Bin},
    date = {2019-03-21},
    doi = {10.1021/acs.jpclett.9b00085},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {6},
    pages = {1185--1191},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    title = {Bridging the {{Gap}} between {{Direct Dynamics}} and {{Globally Accurate Reactive Potential Energy Surfaces Using Neural Networks}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {10}
}

@article{zhangCrystallizationP3Sn4Phase2021a,
    abstract = {We performed molecular dynamics simulations to study the crystallization of the P3Sn4 phase from P(2)Sn(5 )liquid using a machine learning (ML) interatomic potential with desirable efficiency and accuracy. Our results capture the liquid properties of P2Sn5 at 1300 K, which is well above the melting temperature. The phase separation and crystallization are observed when P2Sn5 liquid is cooled down below 832 and 505 K, respectively. The simulation results are in good agreement with the experimentally observed phase transformation behaviors and provide useful insights into the complex nucleation and crystallization process at the details of atomistic scale. Our work also demonstrated that ML interatomic potentials based on neural network deep learning are robust and capable of accurately describing the energetics and kinetics of complex materials through molecular dynamics simulations.},
    annotation = {WOS:000619760700033},
    author = {Zhang, Chao and Sun, Yang and Wang, Hai-Di and Zhang, Feng and Wen, Tong-Qi and Ho, Kai-Ming and Wang, Cai-Zhuang},
    date = {2021-02-11},
    doi = {10.1021/acs.jpcc.0c08873},
    issn = {1932-7447},
    journaltitle = {Journal of Physical Chemistry C},
    langid = {english},
    location = {{Washington}},
    number = {5},
    pages = {3127--3133},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. C},
    title = {Crystallization of the {{P3Sn4 Phase}} upon {{Cooling P2Sn5 Liquid}} by {{Molecular Dynamics Simulation Using}} a {{Machine Learning Interatomic Potential}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {125}
}

@article{zhangEmbeddedAtomNeural2019,
    abstract = {We propose a simple, but efficient and accurate, machine learning (ML) model for developing a high-dimensional potential energy surface. This so-called embedded atom neural network (EANN) approach is inspired by the well-known empirical embedded atom method (EAM) model used in the condensed phase. It simply replaces the scalar embedded atom density in EAM with a Gaussian-type orbital based density vector and represents the complex relationship between the embedded density vector and atomic energy by neural networks. We demonstrate that the EANN approach is equally accurate as several established ML models in representing both big molecular and extended periodic systems, yet with much fewer parameters and configurations. It is highly efficient as it implicitly contains the three-body information without an explicit sum of the conventional costly angular descriptors. With high accuracy and efficiency, EANN potentials can vastly accelerate molecular dynamics and spectroscopic simulations in complex systems at ab initio level.},
    annotation = {WOS:000484884300030},
    author = {Zhang, Yaolong and Hu, Ce and Jiang, Bin},
    date = {2019-09-05},
    doi = {10.1021/acs.jpclett.9b02037},
    issn = {1948-7185},
    journaltitle = {Journal of Physical Chemistry Letters},
    langid = {english},
    location = {{Washington}},
    number = {17},
    pages = {4962--4967},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. Lett.},
    shorttitle = {Embedded {{Atom Neural Network Potentials}}},
    title = {Embedded {{Atom Neural Network Potentials}}: {{Efficient}} and {{Accurate Machine Learning}} with a {{Physically Inspired Representation}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/5},
    urldate = {2021-08-06},
    volume = {10}
}

@article{zhangGlobalOptimizationChemical2021a,
    abstract = {Chemical clusters are relevant to many applications in catalysis, separations, materials, and energy sciences. Experimentally, the structure of clusters is difficult to determine, but it is very important in understanding their chemistry and properties. Computational methods can be used to examine cluster structure, however finding the most stable structure is not simple, particularly as the cluster size increases. Global optimization techniques have long been used to tackle the problem of the most stable structure, but such approaches would have to look for a global minimum, while sampling local minima over the whole potential energy surface as well. In this review, the state-of-the-art theory of global optimization theory is summarized. First, the definition, significance, relation to experiments, and a brief history of global optimization is presented. We then discuss, in more detail, three versatile global optimization methods: the basin hopping, the artificial bee colony algorithm, and the genetic algorithm. We close with some representative application examples of global optimization of clusters since 2016 and the challenges, open questions and opportunities in this field.},
    annotation = {WOS:000591191600001},
    author = {Zhang, Jun and Glezakou, Vassiliki-Alexandra},
    date = {2021-04-05},
    doi = {10.1002/qua.26553},
    issn = {0020-7608},
    journaltitle = {International Journal of Quantum Chemistry},
    langid = {english},
    location = {{Hoboken}},
    number = {7},
    pages = {e26553},
    publisher = {{Wiley}},
    shortjournal = {Int. J. Quantum Chem.},
    shorttitle = {Global Optimization of Chemical Cluster Structures},
    title = {Global Optimization of Chemical Cluster Structures: {{Methods}}, Applications, and Challenges},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {121}
}

@article{zhangLinearFrequencyPrinciple2021a,
    abstract = {Why heavily parameterized neural networks (NNs) do not overfit the data is an important long standing open question. We propose a phenomenological model of the NN training to explain this non-overfitting puzzle. Our linear frequency principle (LFP) model accounts for a key dynamical feature of NNs: they learn low frequencies first, irrespective of microscopic details. Theory based on our LFP model shows that low frequency dominance of target functions is the key condition for the non-overfitting of NNs and is verified by experiments. Furthermore, through an ideal two-layer NN, we unravel how detailed microscopic NN training dynamics statistically gives rise to an LFP model with quantitative prediction power.},
    annotation = {WOS:000632901300001},
    author = {Zhang, Yaoyu and Luo, Tao and Ma, Zheng and Xu, Zhi-Qin John},
    date = {2021-03},
    doi = {10.1088/0256-307X/38/3/038701},
    issn = {0256-307X},
    journaltitle = {Chinese Physics Letters},
    langid = {english},
    location = {{Bristol}},
    number = {3},
    pages = {038701},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {Chin. Phys. Lett.},
    title = {A {{Linear Frequency Principle Model}} to {{Understand}} the {{Absence}} of {{Overfitting}} in {{Neural Networks}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {38}
}

@article{zhangPerspectiveDeepLearning2020a,
    abstract = {Deep learning is transforming many areas in science, and it has great potential in modeling molecular systems. However, unlike the mature deployment of deep learning in computer vision and natural language processing, its development in molecular modeling and simulations is still at an early stage, largely because the inductive biases of molecules are completely different from those of images or texts. Footed on these differences, we first reviewed the limitations of traditional deep learning models from the perspective of molecular physics and wrapped up some relevant technical advancement at the interface between molecular modeling and deep learning. We do not focus merely on the ever more complex neural network models; instead, we introduce various useful concepts and ideas brought by modern deep learning. We hope that transacting these ideas into molecular modeling will create new opportunities. For this purpose, we summarized several representative applications, ranging from supervised to unsupervised and reinforcement learning, and discussed their connections with the emerging trends in deep learning. Finally, we give an outlook for promising directions which may help address the existing issues in the current framework of deep molecular modeling.},
    annotation = {WOS:000566759400001},
    author = {Zhang, Jun and Lei, Yao-Kun and {hZang}, Zhen and Chang, Junhan and Li, Maodong and Han, Xu and Yang, Lijiang and Yang, Yi Isaac and Gao, Yi Qin},
    date = {2020-08-27},
    doi = {10.1021/acs.jpca.0c04473},
    issn = {1089-5639},
    journaltitle = {Journal of Physical Chemistry A},
    langid = {english},
    location = {{Washington}},
    number = {34},
    pages = {6745--6763},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. A},
    title = {A {{Perspective}} on {{Deep Learning}} for {{Molecular Modeling}} and {{Simulations}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {124}
}

@article{zhaoLearningPhysicsPattern2020a,
    abstract = {Using a framework of partial differential equation-constrained optimization, we demonstrate that multiple constitutive relations can be extracted simultaneously from a small set of images of pattern formation. Examples include state-dependent properties in phase-field models, such as the diffusivity, kinetic prefactor, free energy, and direct correlation function, given only the general form of the Cahn-Hilliard equation, Allen-Cahn equation, or dynamical density functional theory (phase-field crystal model). Constraints can be added based on physical arguments to accelerate convergence and avoid spurious results. Reconstruction of the free energy functional, which contains nonlinear dependence on the state variable and differential or convolutional operators, opens the possibility of learning nonequilibrium thermodynamics from only a few snapshots of the dynamics.},
    annotation = {WOS:000513568400001},
    author = {Zhao, Hongbo and Storey, Brian D. and Braatz, Richard D. and Bazant, Martin Z.},
    date = {2020-02-14},
    doi = {10.1103/PhysRevLett.124.060201},
    issn = {0031-9007},
    journaltitle = {Physical Review Letters},
    langid = {english},
    location = {{College Pk}},
    number = {6},
    pages = {060201},
    publisher = {{Amer Physical Soc}},
    shortjournal = {Phys. Rev. Lett.},
    title = {Learning the {{Physics}} of {{Pattern Formation}} from {{Images}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {124}
}

@article{zhaoTheoreticalPredictionRedox2021,
    abstract = {Redox potential is a crucial electrochemical parameter in the electrorefining process of spent fuel reprocessing. Unfortunately, the harsh measurement environment of spent fuel makes it difficult to obtain. With the continuous updating of computing technology, people have developed the method for calculating the redox potentials using first-principles molecular dynamics (FPMD), but limited by the calculation cost, the simulation scale and time of FPMD are restricted. To make the calculation results more convincing, this work used deep potential (DP) to realize redox potential calculation on a larger time scale. We extracted datasets from FPMD calculations and used these to train and validate the DP, and compared energies, forces, and radial distribution functions that are evaluated using DFT and DP, to demonstrate that DP can achieve DFT accuracy. Using La3+/La as the reference electrode, the redox potentials of Ce3+/Ce, Pr3+/Pr, and Y3+/Y in the LiCl-KCl mixed molten salt system at 723 K were calculated. The results matched well with the FPMD results and experimental data. This work fully demonstrates the feasibility of the DP in calculating the redox potentials. Simultaneously, it provides new idea for obtaining accurate data in the process of spent fuel reprocessing.},
    annotation = {WOS:000628090300002},
    author = {Zhao, Jia and Liang, Wenshuo and Lu, Guimin},
    date = {2021-05},
    doi = {10.1007/s11581-021-03988-0},
    issn = {0947-7047},
    journaltitle = {Ionics},
    langid = {english},
    location = {{Heidelberg}},
    number = {5},
    pages = {2079--2088},
    publisher = {{Springer Heidelberg}},
    shortjournal = {Ionics},
    title = {Theoretical Prediction on the Redox Potentials of Rare-Earth Ions by Deep Potentials},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {27}
}

@article{zhengRetentionRecyclingDeuterium2021a,
    abstract = {Understanding the retention and recycling of hydrogen isotopes in liquid metal plasma-facing materials such as liquid Li, Sn, and Li-Sn are of fundamental importance in designing magnetically confined fusion reactors. We perform first-principles molecules dynamics simulations of liquid Li-Sn slab with inserted D atoms to provide microscopic insights into the interactions of D with Li-Sn liquid metal. We prepare two samples with low and high concentrations of D atoms. We observe evaporation of D molecules out of the Li-Sn slabs in both concentrations of D. With detailed analysis, we unveil a cooperative process of forming D-2 molecules in liquid Li-Sn, where Li atoms act as catalytic centers to trap a D atom before another D comes nearby to form a molecule, and the surplus charges are transferred from D-2 to nearby Sn atoms. Furthermore, we predict a temperature window in the low concentration case in which D-2 molecules can escape to vacuum, while LiD molecules cannot. The above findings deepen our understanding of interactions between hydrogen isotopes and Li-Sn liquid metal. (C) 2020 Elsevier B.V. All rights reserved.},
    annotation = {WOS:000600309100006},
    author = {Zheng, Daye and Shen, Zhen-Xiong and Chen, Mohan and Ren, Xinguo and He, Lixin},
    date = {2021-01},
    doi = {10.1016/j.jnucmat.2020.152542},
    issn = {0022-3115},
    journaltitle = {Journal of Nuclear Materials},
    langid = {english},
    location = {{Amsterdam}},
    pages = {152542},
    publisher = {{Elsevier}},
    shortjournal = {J. Nucl. Mater.},
    title = {Retention and Recycling of Deuterium in Liquid Lithium-Tin Slab Studied by First-Principles Molecular Dynamics},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/2},
    urldate = {2021-08-06},
    volume = {543}
}

@article{zhouAtomicstatedependentScreeningModel2021a,
    abstract = {An ion embedded in warm/hot dense plasmas will greatly alter its microscopic structure and dynamics, as well as the macroscopic radiation transport properties of the plasmas, due to complicated many-body interactions with surrounding particles. Accurate theoretically modeling of such kind of quantum many-body interactions is essential but very challenging. In this work, we propose an atomic-state-dependent screening model for treating the plasmas with a wide range of temperatures and densities, in which the contributions of three-body recombination processes are included. We show that the electron distributions around an ion are strongly correlated with the ionic state studied due to the contributions of three-body recombination processes. The feasibility and validation of the proposed model are demonstrated by reproducing the experimental result of the line-shift of hot-dense plasmas as well as the classical molecular dynamic simulations of moderately coupled ultra-cold neutral plasmas. Our work opens a promising way to treat the screening effect of hot and warm dense plasma, which is a bottleneck of those extensive studies in high-energy-density physics, such as atomic processes in plasma, plasma spectra and radiation transport properties, among others. Atoms embedded in dense hot plasmas are affected by complex many-body interactions which challenges our capacity to model high energy density plasma. The authors propose a solution to the effects of many-body interactions on ions in dense plasmas, with a particular focus on the threebody interaction.},
    annotation = {WOS:000668539100001},
    author = {Zhou, Fuyang and Qu, Yizhi and Gao, Junwen and Ma, Yulong and Wu, Yong and Wang, Jianguo},
    date = {2021-06-30},
    doi = {10.1038/s42005-021-00652-x},
    issn = {2399-3650},
    journaltitle = {Communications Physics},
    langid = {english},
    location = {{Berlin}},
    number = {1},
    pages = {148},
    publisher = {{Nature Research}},
    shortjournal = {Commun. Phys.},
    title = {Atomic-State-Dependent Screening Model for Hot and Warm Dense Plasmas},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/1},
    urldate = {2021-08-06},
    volume = {4}
}

@article{zhouStructureDynamicsSupercooled2021a,
    abstract = {Studies of supercooled liquid phase-change materials are important for the development of phase-change memory and neuromorphic computing devices. Herein, a machine-learning (ML)-based interatomic potential for Ge2Sb2Te5 (GST) to conduct large-scale molecular dynamics simulations of liquid and supercooled liquid GST is used. A pronounced effect of the thermostat parameters on the simulation results is demonstrated, and it is shown how using a Langevin thermostat with optimized damping values can lead to excellent agreement with reference ab initio molecular dynamics (AIMD) simulations. Structural and dynamical analyses are presented, including the studies of radial and angular distributions, homopolar bonds, and the temperature-dependent diffusivity. Herein, the usefulness of ML-driven molecular dynamics for further studies of supercooled liquid GST, with length and timescales far exceeding those that are accessible to AIMD is demonstrated.},
    annotation = {WOS:000582360600001},
    author = {Zhou, Yu-Xing and Zhang, Han-Yi and Deringer, Volker L. and Zhang, Wei},
    date = {2021-03},
    doi = {10.1002/pssr.202000403},
    issn = {1862-6254},
    journaltitle = {Physica Status Solidi-Rapid Research Letters},
    langid = {english},
    location = {{Weinheim}},
    number = {3},
    pages = {2000403},
    publisher = {{Wiley-V C H Verlag Gmbh}},
    shortjournal = {Phys. Status Solidi-Rapid Res. Lett.},
    title = {Structure and {{Dynamics}} of {{Supercooled Liquid Ge2Sb2Te5}} from {{Machine}}-{{Learning}}-{{Driven Simulations}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/3},
    urldate = {2021-08-06},
    volume = {15}
}

@article{zhuanDiscriminatingHighPressureWater2020,
    abstract = {Recent discoveries of dynamic ice VII and superionic ice highlight the importance of ionic diffusions in discriminating high-pressure (P) water phases. The rare event nature and the chemical bond breaking associated with these diffusions, however, make extensive simulations of these processes unpractical to ab initio and inappropriate for force field based methods. Using a first-principles neural network potential, we performed a theoretical study of water at 5-70 GPa and 300-3000 K. Long-time dynamics of protons and oxygens were found indispensable in discriminating several subtle states of water, characterized by proton's and oxygen ion's diffusion coefficients and the distribution of proton's displacements. Within dynamic ice VII, two types of proton transfer mechanisms, i.e., translational and rotational transfers, were identified to discriminate this region further into dynamic ice VII T and dynamic ice VII R. The triple point between ice VII, superionic ice (SI), and liquid exists because the loosening of the bcc oxygen skeleton is prevented by the decrease of interatomic distances at high P's. The melting of ice VII above similar to 40 GPa can be understood as a process of two individual steps: the melting of protons and the retarded melting of oxygens, responsible for the forming of SI. The boundary of the dynamic ice VII and SI lies on the continuation line ice VII's melting curve at low P's. Based on these, a detailed phase diagram is given, which may shed light on studies of water under P's in a wide range of interdisciplinary sciences.},
    annotation = {WOS:000532319600001},
    author = {Zhuan, Lin and Ye, Qijun and Pan, Ding and Li, Xin-Zheng},
    date = {2020-04-13},
    doi = {10.1088/0256-307X/37/4/043101},
    issn = {0256-307X},
    journaltitle = {Chinese Physics Letters},
    langid = {english},
    location = {{Bristol}},
    number = {4},
    pages = {043101},
    publisher = {{Iop Publishing Ltd}},
    shortjournal = {Chin. Phys. Lett.},
    title = {Discriminating {{High}}-{{Pressure Water Phases Using Rare}}-{{Event Determined Ionic Dynamical Properties}}*},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {37}
}

@article{zuoPerformanceCostAssessment2020,
    abstract = {Machine learning of the quantitative relationship between local environment descriptors and the potential energy surface of a system of atoms has emerged as a new frontier in the development of interatomic potentials (IAPs). Here, we present a comprehensive evaluation of machine learning IAPs (ML-IAPs) based on four local environment descriptors-atom-centered symmetry functions (ACSF), smooth overlap of atomic positions (SOAP), the spectral neighbor analysis potential (SNAP) bispectrum components, and moment tensors-using a diverse data set generated using high-throughput density functional theory (DFT) calculations. The data set comprising bcc (Li, Mo) and fcc (Cu, Ni) metals and diamond group IV semiconductors (Si, Ge) is chosen to span a range of crystal structures and bonding. All descriptors studied show excellent performance in predicting energies and forces far surpassing that of classical IAPs, as well as predicting properties such as elastic constants and phonon dispersion curves. We observe a general trade-off between accuracy and the degrees of freedom of each model and, consequently, computational cost. We will discuss these trade-offs in the context of model selection for molecular dynamics and other applications.},
    annotation = {WOS:000510531200016},
    author = {Zuo, Yunxing and Chen, Chi and Li, Xiangguo and Deng, Zhi and Chen, Yiming and Behler, Joerg and Csanyi, Gabor and Shapeev, Alexander and Thompson, Aidan P. and Wood, Mitchell A. and Ong, Shyue Ping},
    date = {2020-01-30},
    doi = {10.1021/acs.jpca.9b08723},
    issn = {1089-5639},
    journaltitle = {Journal of Physical Chemistry A},
    langid = {english},
    location = {{Washington}},
    number = {4},
    pages = {731--745},
    publisher = {{Amer Chemical Soc}},
    shortjournal = {J. Phys. Chem. A},
    title = {Performance and {{Cost Assessment}} of {{Machine Learning Interatomic Potentials}}},
    url = {https://www.webofscience.com/wos/alldb/summary/44a3404b-9775-4cfb-8e8f-3f1fab40830f-035e7f8d/date-descending/4},
    urldate = {2021-08-06},
    volume = {124}
}

